<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>human_id</th>
      <th>wikidata_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>The top 100 papers</td>
      <td> <a href=./notes/Q28250597.md> Q28250597 </a></td>
    </tr>
    <tr>
      <th>1</th>
      <td>The origin and evolution of cell types</td>
      <td> <a href=./notes/Q38798014.md> Q38798014 </a></td>
    </tr>
    <tr>
      <th>2</th>
      <td>Make code accessible with these cloud services</td>
      <td> <a href=./notes/Q91130546.md> Q91130546 </a></td>
    </tr>
    <tr>
      <th>3</th>
      <td>Genetic address book for retinal cell types.</td>
      <td> <a href=./notes/Q43299861.md> Q43299861 </a></td>
    </tr>
    <tr>
      <th>4</th>
      <td>A community-based transcriptomics classification and nomenclature of neocortical cell types</td>
      <td> <a href=./notes/Q98665291.md> Q98665291 </a></td>
    </tr>
    <tr>
      <th>5</th>
      <td>The network effect: studying COVID-19 pathology with the Human Cell Atlas</td>
      <td> <a href=./notes/Q96817154.md> Q96817154 </a></td>
    </tr>
    <tr>
      <th>6</th>
      <td>Comprehensive Classification of Retinal Bipolar Neurons by Single-Cell Transcriptomics.</td>
      <td> <a href=./notes/Q40967362.md> Q40967362 </a></td>
    </tr>
    <tr>
      <th>7</th>
      <td>Identifiers for the 21st century: How to design, provision, and reuse persistent identifiers to maximize utility and impact of life science data</td>
      <td> <a href=./notes/Q33037209.md> Q33037209 </a></td>
    </tr>
    <tr>
      <th>8</th>
      <td>The human disease network</td>
      <td> <a href=./notes/Q24678240.md> Q24678240 </a></td>
    </tr>
    <tr>
      <th>9</th>
      <td>The evolving concept of cell identity in the single cell era</td>
      <td> <a href=./notes/Q93086971.md> Q93086971 </a></td>
    </tr>
    <tr>
      <th>10</th>
      <td>PRINTS and its automatic supplement, prePRINTS.</td>
      <td> <a href=./notes/Q34169535.md> Q34169535 </a></td>
    </tr>
    <tr>
      <th>11</th>
      <td>The naming of neurons: applications of taxonomic theory to the study of cellular populations.</td>
      <td> <a href=./notes/Q34065481.md> Q34065481 </a></td>
    </tr>
    <tr>
      <th>12</th>
      <td>Making your database available through Wikipedia: the pros and cons</td>
      <td> <a href=./notes/Q28254676.md> Q28254676 </a></td>
    </tr>
    <tr>
      <th>13</th>
      <td>ezTag: tagging biomedical concepts via interactive learning.</td>
      <td> <a href=./notes/Q55483834.md> Q55483834 </a></td>
    </tr>
    <tr>
      <th>14</th>
      <td>GNormPlus: An Integrative Approach for Tagging Genes, Gene Families, and Protein Domains</td>
      <td> <a href=./notes/Q28200510.md> Q28200510 </a></td>
    </tr>
    <tr>
      <th>15</th>
      <td>tmVar 2.0: integrating genomic variant information from literature with dbSNP and ClinVar for precision medicine.</td>
      <td> <a href=./notes/Q47317210.md> Q47317210 </a></td>
    </tr>
    <tr>
      <th>16</th>
      <td>Assisting manual literature curation for protein-protein interactions using BioQRator</td>
      <td> <a href=./notes/Q33928865.md> Q33928865 </a></td>
    </tr>
    <tr>
      <th>17</th>
      <td>Neuronal diversity and convergence in a visual system developmental atlas</td>
      <td> <a href=./notes/Q101226729.md> Q101226729 </a></td>
    </tr>
    <tr>
      <th>18</th>
      <td>Enabling Open Science: Wikidata for Research (Wiki4R)</td>
      <td> <a href=./notes/Q26707522.md> Q26707522 </a></td>
    </tr>
    <tr>
      <th>19</th>
      <td>What Is it Like to Be a Bat?</td>
      <td> <a href=./notes/Q2703462.md> Q2703462 </a></td>
    </tr>
    <tr>
      <th>20</th>
      <td>Ten tips for a text-mining-ready article: How to improve automated discoverability and interpretability</td>
      <td> <a href=./notes/Q96032351.md> Q96032351 </a></td>
    </tr>
    <tr>
      <th>21</th>
      <td>Jointly defining cell types from multiple single-cell datasets using LIGER</td>
      <td> <a href=./notes/Q100526347.md> Q100526347 </a></td>
    </tr>
    <tr>
      <th>22</th>
      <td>Dovetailing biology and chemistry: integrating the Gene Ontology with the ChEBI chemical ontology.</td>
      <td> <a href=./notes/Q34878068.md> Q34878068 </a></td>
    </tr>
    <tr>
      <th>23</th>
      <td>Numerical taxonomy.</td>
      <td> <a href=./notes/Q50735370.md> Q50735370 </a></td>
    </tr>
    <tr>
      <th>24</th>
      <td>A Taxonomic Problem</td>
      <td> <a href=./notes/Q60137015.md> Q60137015 </a></td>
    </tr>
    <tr>
      <th>25</th>
      <td>Pathway information extracted from 25 years of pathway figures</td>
      <td> <a href=./notes/Q101473819.md> Q101473819 </a></td>
    </tr>
    <tr>
      <th>26</th>
      <td>The Development of Taxonomic Theory Since 1851</td>
      <td> <a href=./notes/Q59058260.md> Q59058260 </a></td>
    </tr>
    <tr>
      <th>27</th>
      <td>Semantic integration of gene expression analysis tools and data sources using software connectors.</td>
      <td> <a href=./notes/Q30570244.md> Q30570244 </a></td>
    </tr>
    <tr>
      <th>28</th>
      <td>The principles of science : a treatise on logic and scientific method</td>
      <td> <a href=./notes/Q51427139.md> Q51427139 </a></td>
    </tr>
    <tr>
      <th>29</th>
      <td>Some thoughts on bacterial classification</td>
      <td> <a href=./notes/Q74657187.md> Q74657187 </a></td>
    </tr>
    <tr>
      <th>30</th>
      <td>Cross-linking BioThings APIs through JSON-LD to facilitate knowledge exploration.</td>
      <td> <a href=./notes/Q49483648.md> Q49483648 </a></td>
    </tr>
    <tr>
      <th>31</th>
      <td>Knowledge-based biomedical Data Science</td>
      <td> <a href=./notes/Q57001498.md> Q57001498 </a></td>
    </tr>
    <tr>
      <th>32</th>
      <td>The Species Concept Among Viruses</td>
      <td> <a href=./notes/Q59063943.md> Q59063943 </a></td>
    </tr>
    <tr>
      <th>33</th>
      <td>The application of computers to taxonomy</td>
      <td> <a href=./notes/Q28181838.md> Q28181838 </a></td>
    </tr>
    <tr>
      <th>34</th>
      <td>Publishing DisGeNET as nanopublications</td>
      <td> <a href=./notes/Q31194033.md> Q31194033 </a></td>
    </tr>
    <tr>
      <th>35</th>
      <td>The moral status of animals</td>
      <td> <a href=./notes/Q79763269.md> Q79763269 </a></td>
    </tr>
    <tr>
      <th>36</th>
      <td>The use of models in bacterial classification</td>
      <td> <a href=./notes/Q78658697.md> Q78658697 </a></td>
    </tr>
    <tr>
      <th>37</th>
      <td>The types of retinal ganglion cells: current status and implications for neuronal classification.</td>
      <td> <a href=./notes/Q38432885.md> Q38432885 </a></td>
    </tr>
    <tr>
      <th>38</th>
      <td>Genuine semantic publishing</td>
      <td> <a href=./notes/Q46412887.md> Q46412887 </a></td>
    </tr>
    <tr>
      <th>39</th>
      <td>The anatomy of a nanopublication</td>
      <td> <a href=./notes/Q57011346.md> Q57011346 </a></td>
    </tr>
    <tr>
      <th>40</th>
      <td>Heritability enrichment of specifically expressed genes identifies disease-relevant tissues and cell types.</td>
      <td> <a href=./notes/Q52720499.md> Q52720499 </a></td>
    </tr>
    <tr>
      <th>41</th>
      <td>A gene expression atlas of the central nervous system based on bacterial artificial chromosomes</td>
      <td> <a href=./notes/Q29617753.md> Q29617753 </a></td>
    </tr>
    <tr>
      <th>42</th>
      <td>SourceData: a semantic platform for curating and searching figures.</td>
      <td> <a href=./notes/Q48094767.md> Q48094767 </a></td>
    </tr>
    <tr>
      <th>43</th>
      <td>Biological knowledge management: the emerging role of the Semantic Web technologies.</td>
      <td> <a href=./notes/Q37490532.md> Q37490532 </a></td>
    </tr>
    <tr>
      <th>44</th>
      <td>Systematic integration of biomedical knowledge prioritizes drugs for repurposing.</td>
      <td> <a href=./notes/Q42255083.md> Q42255083 </a></td>
    </tr>
    <tr>
      <th>45</th>
      <td>Utopia documents: linking scholarly literature with research data</td>
      <td> <a href=./notes/Q28292614.md> Q28292614 </a></td>
    </tr>
    <tr>
      <th>46</th>
      <td>Astrocyte layers in the mammalian cerebral cortex revealed by a single-cell in situ transcriptomic map</td>
      <td> <a href=./notes/Q90591926.md> Q90591926 </a></td>
    </tr>
    <tr>
      <th>47</th>
      <td>Web scraping technologies in an API world</td>
      <td> <a href=./notes/Q86694255.md> Q86694255 </a></td>
    </tr>
    <tr>
      <th>48</th>
      <td>How we learnt to stop worrying and love web scraping</td>
      <td> <a href=./notes/Q99234140.md> Q99234140 </a></td>
    </tr>
    <tr>
      <th>49</th>
      <td>A translational profiling approach for the molecular characterization of CNS cell types</td>
      <td> <a href=./notes/Q29615252.md> Q29615252 </a></td>
    </tr>
    <tr>
      <th>50</th>
      <td>Publishing on the semantic web</td>
      <td> <a href=./notes/Q28185770.md> Q28185770 </a></td>
    </tr>
    <tr>
      <th>51</th>
      <td>A periodic table of cell types</td>
      <td> <a href=./notes/Q93086975.md> Q93086975 </a></td>
    </tr>
    <tr>
      <th>52</th>
      <td>Universal cell type identifier based on number theory.</td>
      <td> <a href=./notes/Q52374237.md> Q52374237 </a></td>
    </tr>
    <tr>
      <th>53</th>
      <td>Neuronal cell types.</td>
      <td> <a href=./notes/Q35830509.md> Q35830509 </a></td>
    </tr>
    <tr>
      <th>54</th>
      <td>Heterogeneity of neutrophils</td>
      <td> <a href=./notes/Q91997728.md> Q91997728 </a></td>
    </tr>
    <tr>
      <th>55</th>
      <td>Natural killer cell immunotypes related to COVID-19 disease severity</td>
      <td> <a href=./notes/Q98622035.md> Q98622035 </a></td>
    </tr>
    <tr>
      <th>56</th>
      <td>Adventures in semantic publishing: exemplar semantic enhancements of a research article</td>
      <td> <a href=./notes/Q21092566.md> Q21092566 </a></td>
    </tr>
    <tr>
      <th>57</th>
      <td>Reproducibility of Flow Cytometry Through Standardization: Opportunities and Challenges</td>
      <td> <a href=./notes/Q90588777.md> Q90588777 </a></td>
    </tr>
    <tr>
      <th>58</th>
      <td>Application of a translational profiling approach for the comparative analysis of CNS cell types</td>
      <td> <a href=./notes/Q42585641.md> Q42585641 </a></td>
    </tr>
    <tr>
      <th>59</th>
      <td>Will a biological database be different from a biological journal?</td>
      <td> <a href=./notes/Q21092584.md> Q21092584 </a></td>
    </tr>
    <tr>
      <th>60</th>
      <td>The future of annotation/biocuration</td>
      <td> <a href=./notes/Q56880438.md> Q56880438 </a></td>
    </tr>
    <tr>
      <th>61</th>
      <td>Swiss-Prot: juggling between evolution and stability.</td>
      <td> <a href=./notes/Q47579749.md> Q47579749 </a></td>
    </tr>
    <tr>
      <th>62</th>
      <td>Retinal ganglion cells: properties, types, genera, pathways and trans-species comparisons.</td>
      <td> <a href=./notes/Q48839876.md> Q48839876 </a></td>
    </tr>
    <tr>
      <th>63</th>
      <td>WikiPathways: connecting communities</td>
      <td> <a href=./notes/Q102205677.md> Q102205677 </a></td>
    </tr>
    <tr>
      <th>64</th>
      <td>The neuron classification problem</td>
      <td> <a href=./notes/Q36854753.md> Q36854753 </a></td>
    </tr>
    <tr>
      <th>65</th>
      <td>Petilla terminology: nomenclature of features of GABAergic interneurons of the cerebral cortex</td>
      <td> <a href=./notes/Q24620160.md> Q24620160 </a></td>
    </tr>
    <tr>
      <th>66</th>
      <td>Protein ontology on the semantic web for knowledge discovery</td>
      <td> <a href=./notes/Q100525924.md> Q100525924 </a></td>
    </tr>
    <tr>
      <th>67</th>
      <td>Towards a classification of stem cells</td>
      <td> <a href=./notes/Q64097544.md> Q64097544 </a></td>
    </tr>
    <tr>
      <th>68</th>
      <td>Reader engagement with medical content on Wikipedia</td>
      <td> <a href=./notes/Q90109096.md> Q90109096 </a></td>
    </tr>
    <tr>
      <th>69</th>
      <td>How Single-Cell Genomics Is Changing Evolutionary and Developmental Biology.</td>
      <td> <a href=./notes/Q38622289.md> Q38622289 </a></td>
    </tr>
    <tr>
      <th>70</th>
      <td>The evolution of cell types in animals: emerging principles from molecular studies.</td>
      <td> <a href=./notes/Q37301036.md> Q37301036 </a></td>
    </tr>
    <tr>
      <th>71</th>
      <td>Key transitions during the evolution of animal phototransduction: novelty, "tree-thinking," co-option, and co-duplication.</td>
      <td> <a href=./notes/Q46221687.md> Q46221687 </a></td>
    </tr>
    <tr>
      <th>72</th>
      <td>Neuronal diversity: too many cell types for comfort?</td>
      <td> <a href=./notes/Q48373178.md> Q48373178 </a></td>
    </tr>
    <tr>
      <th>73</th>
      <td>The BridgeDb framework: standardized access to gene, protein and metabolite identifier mapping services</td>
      <td> <a href=./notes/Q28842753.md> Q28842753 </a></td>
    </tr>
    <tr>
      <th>74</th>
      <td>Definitions</td>
      <td> <a href=./notes/Q102232524.md> Q102232524 </a></td>
    </tr>
    <tr>
      <th>75</th>
      <td>A recipe book for cell types in the human brain</td>
      <td> <a href=./notes/Q93087299.md> Q93087299 </a></td>
    </tr>
    <tr>
      <th>76</th>
      <td>Single-cell sequencing paints diverse pictures of the brain</td>
      <td> <a href=./notes/Q58084012.md> Q58084012 </a></td>
    </tr>
    <tr>
      <th>77</th>
      <td>Neuron Names: A Gene- and Property-Based Name Format, With Special Reference to Cortical Neurons</td>
      <td> <a href=./notes/Q64065346.md> Q64065346 </a></td>
    </tr>
    <tr>
      <th>78</th>
      <td>The BRAIN Initiative Cell Census Consortium: Lessons Learned toward Generating a Comprehensive Brain Cell Atlas.</td>
      <td> <a href=./notes/Q42778383.md> Q42778383 </a></td>
    </tr>
    <tr>
      <th>79</th>
      <td>Transcriptional landscape of the prenatal human brain</td>
      <td> <a href=./notes/Q33926910.md> Q33926910 </a></td>
    </tr>
    <tr>
      <th>80</th>
      <td>Nanopublications: A Growing Resource of Provenance-Centric Scientific Linked Data</td>
      <td> <a href=./notes/Q62924270.md> Q62924270 </a></td>
    </tr>
    <tr>
      <th>81</th>
      <td>Transcriptomic and morphophysiological evidence for a specialized human cortical GABAergic cell type</td>
      <td> <a href=./notes/Q60017992.md> Q60017992 </a></td>
    </tr>
    <tr>
      <th>82</th>
      <td>DisGeNET-RDF: harnessing the innovative power of the Semantic Web to explore the genetic basis of diseases</td>
      <td> <a href=./notes/Q28596617.md> Q28596617 </a></td>
    </tr>
    <tr>
      <th>83</th>
      <td>Cell-type-specific consequences of Reelin deficiency in the mouse neocortex, hippocampus, and amygdala</td>
      <td> <a href=./notes/Q28595082.md> Q28595082 </a></td>
    </tr>
    <tr>
      <th>84</th>
      <td>The Semanticscience Integrated Ontology (SIO) for biomedical research and knowledge discovery</td>
      <td> <a href=./notes/Q19671659.md> Q19671659 </a></td>
    </tr>
    <tr>
      <th>85</th>
      <td>Ontology Design Patterns for Semantic Web Content</td>
      <td> <a href=./notes/Q57774232.md> Q57774232 </a></td>
    </tr>
    <tr>
      <th>86</th>
      <td>Single-nucleus and single-cell transcriptomes compared in matched cortical cell types</td>
      <td> <a href=./notes/Q60933812.md> Q60933812 </a></td>
    </tr>
    <tr>
      <th>87</th>
      <td>How should access to a COVID-19 vaccine be regulated?</td>
      <td> <a href=./notes/Q102542608.md> Q102542608 </a></td>
    </tr>
    <tr>
      <th>88</th>
      <td>Shifting the paradigm: new approaches for characterizing and classifying neurons.</td>
      <td> <a href=./notes/Q37629171.md> Q37629171 </a></td>
    </tr>
    <tr>
      <th>89</th>
      <td>OWL 2 Web Ontology Language Document Overview (Second Edition)</td>
      <td> <a href=./notes/Q102926772.md> Q102926772 </a></td>
    </tr>
    <tr>
      <th>90</th>
      <td>OWL 2 Web Ontology Language Primer (Second Edition)</td>
      <td> <a href=./notes/Q102929379.md> Q102929379 </a></td>
    </tr>
    <tr>
      <th>91</th>
      <td>A genetic and computational approach to structurally classify neuronal types.</td>
      <td> <a href=./notes/Q34182035.md> Q34182035 </a></td>
    </tr>
    <tr>
      <th>92</th>
      <td>Adult mouse cortical cell taxonomy revealed by single cell transcriptomics</td>
      <td> <a href=./notes/Q27311570.md> Q27311570 </a></td>
    </tr>
    <tr>
      <th>93</th>
      <td>Identifiers.org - Compact Identifier Services in the Cloud</td>
      <td> <a href=./notes/Q100500142.md> Q100500142 </a></td>
    </tr>
    <tr>
      <th>94</th>
      <td>Immune-centric network of cytokines and cells in disease context identified by computational mining of PubMed.</td>
      <td> <a href=./notes/Q64962407.md> Q64962407 </a></td>
    </tr>
    <tr>
      <th>95</th>
      <td>Single-nucleus RNA-seq identifies Huntington disease astrocyte states</td>
      <td> <a href=./notes/Q89763874.md> Q89763874 </a></td>
    </tr>
    <tr>
      <th>96</th>
      <td>Converting neXtProt into Linked Data and nanopublications</td>
      <td> <a href=./notes/Q103834013.md> Q103834013 </a></td>
    </tr>
    <tr>
      <th>97</th>
      <td>Innovations present in the primate interneuron repertoire</td>
      <td> <a href=./notes/Q100395427.md> Q100395427 </a></td>
    </tr>
    <tr>
      <th>98</th>
      <td>A curated database reveals trends in single-cell transcriptomics</td>
      <td> <a href=./notes/Q103034964.md> Q103034964 </a></td>
    </tr>
    <tr>
      <th>99</th>
      <td>Species and cell-type properties of classically defined human and rodent neurons and glia</td>
      <td> <a href=./notes/Q57462271.md> Q57462271 </a></td>
    </tr>
    <tr>
      <th>100</th>
      <td>Single cell transcriptomics in neuroscience: cell classification and beyond</td>
      <td> <a href=./notes/Q88605805.md> Q88605805 </a></td>
    </tr>
    <tr>
      <th>101</th>
      <td>Towards the collaborative curation of the registry underlying Identifiers.org</td>
      <td> <a href=./notes/Q28709067.md> Q28709067 </a></td>
    </tr>
    <tr>
      <th>102</th>
      <td>The Covid-19 Conundrum</td>
      <td> <a href=./notes/Q103964322.md> Q103964322 </a></td>
    </tr>
    <tr>
      <th>103</th>
      <td>Towards the automatic classification of neurons</td>
      <td> <a href=./notes/Q27010194.md> Q27010194 </a></td>
    </tr>
    <tr>
      <th>104</th>
      <td>Brain structure. Cell types in the mouse cortex and hippocampus revealed by single-cell RNA-seq.</td>
      <td> <a href=./notes/Q42776227.md> Q42776227 </a></td>
    </tr>
    <tr>
      <th>105</th>
      <td>Adding a little reality to building ontologies for biology</td>
      <td> <a href=./notes/Q21136172.md> Q21136172 </a></td>
    </tr>
    <tr>
      <th>106</th>
      <td>Data curation + process curation=data integration + science</td>
      <td> <a href=./notes/Q28303099.md> Q28303099 </a></td>
    </tr>
    <tr>
      <th>107</th>
      <td>Implementing a "publish, then review" model of publishing</td>
      <td> <a href=./notes/Q103811380.md> Q103811380 </a></td>
    </tr>
    <tr>
      <th>108</th>
      <td>Interneuron cell types are fit to function</td>
      <td> <a href=./notes/Q27021885.md> Q27021885 </a></td>
    </tr>
    <tr>
      <th>109</th>
      <td>Getting started in text mining</td>
      <td> <a href=./notes/Q21145382.md> Q21145382 </a></td>
    </tr>
    <tr>
      <th>110</th>
      <td>Abstract Meaning Representation for Sembanking</td>
      <td> <a href=./notes/Q104092178.md> Q104092178 </a></td>
    </tr>
    <tr>
      <th>111</th>
      <td>Reconstructing lineage hierarchies of the distal lung epithelium using single-cell RNA-seq</td>
      <td> <a href=./notes/Q34094294.md> Q34094294 </a></td>
    </tr>
    <tr>
      <th>112</th>
      <td>The genomic landscapes of individual melanocytes from human skin</td>
      <td> <a href=./notes/Q100491078.md> Q100491078 </a></td>
    </tr>
    <tr>
      <th>113</th>
      <td>Immune-evasive human islet-like organoids ameliorate diabetes</td>
      <td> <a href=./notes/Q98568692.md> Q98568692 </a></td>
    </tr>
    <tr>
      <th>114</th>
      <td>A single-cell transcriptome atlas of marsupial embryogenesis and X inactivation</td>
      <td> <a href=./notes/Q98568691.md> Q98568691 </a></td>
    </tr>
    <tr>
      <th>115</th>
      <td>Nanopublications for exposing experimental data in the life-sciences: a Huntington's Disease case study</td>
      <td> <a href=./notes/Q28606409.md> Q28606409 </a></td>
    </tr>
    <tr>
      <th>116</th>
      <td>Querying neXtProt nanopublications and their value for insights on sequence variants and tissue expression</td>
      <td> <a href=./notes/Q54670808.md> Q54670808 </a></td>
    </tr>
    <tr>
      <th>117</th>
      <td>Northstar enables automatic classification of known and novel cell types from tumor samples</td>
      <td> <a href=./notes/Q99564826.md> Q99564826 </a></td>
    </tr>
    <tr>
      <th>118</th>
      <td>Molecular biology gets wikified</td>
      <td> <a href=./notes/Q104161380.md> Q104161380 </a></td>
    </tr>
    <tr>
      <th>119</th>
      <td>Capillary cell-type specialization in the alveolus</td>
      <td> <a href=./notes/Q100560109.md> Q100560109 </a></td>
    </tr>
    <tr>
      <th>120</th>
      <td>Naming human genes</td>
      <td> <a href=./notes/Q98196903.md> Q98196903 </a></td>
    </tr>
    <tr>
      <th>121</th>
      <td>Guidelines for human gene nomenclature</td>
      <td> <a href=./notes/Q98148417.md> Q98148417 </a></td>
    </tr>
    <tr>
      <th>122</th>
      <td>Knowledge networks in the age of the Semantic Web</td>
      <td> <a href=./notes/Q28842577.md> Q28842577 </a></td>
    </tr>
    <tr>
      <th>123</th>
      <td>Internet encyclopaedias go head to head</td>
      <td> <a href=./notes/Q24239902.md> Q24239902 </a></td>
    </tr>
    <tr>
      <th>124</th>
      <td>snRNA-seq reveals a subpopulation of adipocytes that regulates thermogenesis</td>
      <td> <a href=./notes/Q101051363.md> Q101051363 </a></td>
    </tr>
    <tr>
      <th>125</th>
      <td>Beyond the frontiers of neuronal types.</td>
      <td> <a href=./notes/Q42156351.md> Q42156351 </a></td>
    </tr>
    <tr>
      <th>126</th>
      <td>Ontology Design Patterns for bio-ontologies: a case study on the Cell Cycle Ontology</td>
      <td> <a href=./notes/Q21093639.md> Q21093639 </a></td>
    </tr>
    <tr>
      <th>127</th>
      <td>Name-calling in the hippocampus (and beyond): coming to terms with neuron types and properties.</td>
      <td> <a href=./notes/Q37656899.md> Q37656899 </a></td>
    </tr>
    <tr>
      <th>128</th>
      <td>Biocuration - mapping resources and needs</td>
      <td> <a href=./notes/Q101217428.md> Q101217428 </a></td>
    </tr>
    <tr>
      <th>129</th>
      <td>Computing Machinery and Intelligence</td>
      <td> <a href=./notes/Q772056.md> Q772056 </a></td>
    </tr>
    <tr>
      <th>130</th>
      <td>ScispaCy: Fast and Robust Models for Biomedical Natural Language Processing</td>
      <td> <a href=./notes/Q101248419.md> Q101248419 </a></td>
    </tr>
    <tr>
      <th>131</th>
      <td>Hippocampome.org: a knowledge base of neuron types in the rodent hippocampus</td>
      <td> <a href=./notes/Q26253817.md> Q26253817 </a></td>
    </tr>
    <tr>
      <th>132</th>
      <td>An ontological approach to describing neurons and their relationships</td>
      <td> <a href=./notes/Q28730328.md> Q28730328 </a></td>
    </tr>
    <tr>
      <th>133</th>
      <td>Extracting a cellular hierarchy from high-dimensional cytometry data with SPADE</td>
      <td> <a href=./notes/Q34037159.md> Q34037159 </a></td>
    </tr>
    <tr>
      <th>134</th>
      <td>NeuroLex.org: an online framework for neuroscience knowledge</td>
      <td> <a href=./notes/Q28681367.md> Q28681367 </a></td>
    </tr>
    <tr>
      <th>135</th>
      <td>New insights into the classification and nomenclature of cortical GABAergic interneurons</td>
      <td> <a href=./notes/Q36747163.md> Q36747163 </a></td>
    </tr>
    <tr>
      <th>136</th>
      <td>Revisiting enigmatic cortical calretinin-expressing interneurons</td>
      <td> <a href=./notes/Q33796214.md> Q33796214 </a></td>
    </tr>
    <tr>
      <th>137</th>
      <td>Concept Wikiﬁcation for COVID-19</td>
      <td> <a href=./notes/Q104392113.md> Q104392113 </a></td>
    </tr>
    <tr>
      <th>138</th>
      <td>Local and Global Algorithms for Disambiguation to Wikipedia</td>
      <td> <a href=./notes/Q104415799.md> Q104415799 </a></td>
    </tr>
    <tr>
      <th>139</th>
      <td>Beyond the frontiers of neuronal types: fuzzy classification of interneurons.</td>
      <td> <a href=./notes/Q46137497.md> Q46137497 </a></td>
    </tr>
    <tr>
      <th>140</th>
      <td>Probabilistic cell typing enables fine mapping of closely related cell types in situ</td>
      <td> <a href=./notes/Q91330127.md> Q91330127 </a></td>
    </tr>
    <tr>
      <th>141</th>
      <td>mRNA-Seq whole-transcriptome analysis of a single cell</td>
      <td> <a href=./notes/Q28240611.md> Q28240611 </a></td>
    </tr>
    <tr>
      <th>142</th>
      <td>Entity linking for biomedical literature</td>
      <td> <a href=./notes/Q28642941.md> Q28642941 </a></td>
    </tr>
    <tr>
      <th>143</th>
      <td>SciBERT: A Pretrained Language Model for Scientific Text</td>
      <td> <a href=./notes/Q104433073.md> Q104433073 </a></td>
    </tr>
    <tr>
      <th>144</th>
      <td>Text mining approaches for dealing with the rapidly expanding literature on COVID-19</td>
      <td> <a href=./notes/Q104079663.md> Q104079663 </a></td>
    </tr>
    <tr>
      <th>145</th>
      <td>Structural Scaffolds for Citation Intent Classification in Scientific Publications</td>
      <td> <a href=./notes/Q104450390.md> Q104450390 </a></td>
    </tr>
    <tr>
      <th>146</th>
      <td>Integration of electrophysiological recordings with single-cell RNA-seq data identifies neuronal subtypes</td>
      <td> <a href=./notes/Q31033187.md> Q31033187 </a></td>
    </tr>
    <tr>
      <th>147</th>
      <td>Integrated Single-Cell Atlases Reveal an Oral SARS-CoV-2 Infection and Transmission Axis</td>
      <td> <a href=./notes/Q101210731.md> Q101210731 </a></td>
    </tr>
    <tr>
      <th>148</th>
      <td>Bridging the Gap Between Research and Practice: Predicting What Will Work Locally</td>
      <td> <a href=./notes/Q104475823.md> Q104475823 </a></td>
    </tr>
    <tr>
      <th>149</th>
      <td>Identification and characterization of distinct brown adipocyte subtypes in C57BL/6J mice</td>
      <td> <a href=./notes/Q103800764.md> Q103800764 </a></td>
    </tr>
    <tr>
      <th>150</th>
      <td>Dental cell type atlas reveals stem and differentiated cell types in mouse and human teeth</td>
      <td> <a href=./notes/Q99616616.md> Q99616616 </a></td>
    </tr>
    <tr>
      <th>151</th>
      <td>Construction of the Literature Graph in Semantic Scholar</td>
      <td> <a href=./notes/Q69587220.md> Q69587220 </a></td>
    </tr>
    <tr>
      <th>152</th>
      <td>CellMarker: a manually curated resource of cell markers in human and mouse</td>
      <td> <a href=./notes/Q56984510.md> Q56984510 </a></td>
    </tr>
    <tr>
      <th>153</th>
      <td>A standard nomenclature for referencing and authentication of pluripotent stem cells.</td>
      <td> <a href=./notes/Q48503347.md> Q48503347 </a></td>
    </tr>
    <tr>
      <th>154</th>
      <td>A resource for cell line authentication, annotation and quality control.</td>
      <td> <a href=./notes/Q38886793.md> Q38886793 </a></td>
    </tr>
    <tr>
      <th>155</th>
      <td>GeneDB and Wikidata</td>
      <td> <a href=./notes/Q91273803.md> Q91273803 </a></td>
    </tr>
    <tr>
      <th>156</th>
      <td>The Cellosaurus, a cell-line knowledge resource.</td>
      <td> <a href=./notes/Q54370168.md> Q54370168 </a></td>
    </tr>
    <tr>
      <th>157</th>
      <td>A coupled autoencoder approach for multi-modal analysis of cell types</td>
      <td> <a href=./notes/Q76472348.md> Q76472348 </a></td>
    </tr>
  </tbody>
</table>