# Blogs to read
https://bioeticacomplexa.blogspot.com/2020/05/
https://bioeticacomplexa.blogspot.com/2020/07/
# Books to read
Huxley, J. S.: The new systematics (Oxford Univ. Press, Oxford 1940).

Popper, K. R.: The open society and its enemies; 4th ed., vol. 1 (Routledge & Kegan Paul, London 1962)
Antonio D. Descarte’s Error. Vintage Press, 2006; 104–105.

Lorhard, J. (1606). Diagraph of Metaphysic or Ontology.St. Gallen: Ogdoas scholastica Book 8. PublishedinSangalli.Trans.Sara L. Uckelman.

Popper, K. R. Objective Knowledge: An Evolutionary Approach. Oxford: Clarendon, 1975.


 Elton CS. 1927. Animal Ecology. https://doi.org/10.5962/bhl.title.7435 

Wagner GP. 2014. Homology, Genes, and Evolutionary Innovatoin.
Princeton, NJ: Princeton University Press

Describing Species - Judith Winston

# Papers to read
https://plato.stanford.edu/archives/win2019/entries/truth-revision/
https://seop.illc.uva.nl/entries/models-science/
# Cell types










Q47565472
Q47870911
Q49192555
Q64118290
Q34978215
Q57945379
Q37068535
Q28288068
Q36625916
Q30980445
Q28586183
Q38771724
Q35984619
## No QID
9. He, M. et al. Cell-type-based analysis of microRNA profles in the mouse brain. Neuron 73, 35–48 (2012).

Evolution of cellular diversity in primary motor cortex of human, marmoset monkey, and mouse
Yao Z,  2020b. A taxonomy of transcriptomic cell types across the isocortex and hippocampal formation. bioRxiv. doi:10.1101/2020.03.30.015214

Replacement of an Essentialistic Perspective on Taxonomic Definitions as Exemplified by the Definition of "Mammalia"
https://www.sciencedirect.com/science/article/abs/pii/S1369848601000036?via%3Dihub

## Other

Tools and Concepts for Interrogating and Defining Cellular Identity
Perspectives on defining cell types in the brain



### Discovery of new cell types:

JJordao MJC, Sankowski R, Brendecke SM, Sagar LG, Tai YH, Tay TL, Schramm E, Armbruster S, Hagemeyer N, et al. Science 2019;363(6425):eaat7554. https://doi.org/10.1126/science.aat7554.
23. Gan Y, Li N, Zou G, Xin Y, Guan J. Identification of cancer subtypes from single-cell RNA-seq data using a consensus clustering method. BMC Med Genet. 2018;11(Suppl 6):117.
24. Mrdjen D, Pavlovic A, Hartmann FJ, Schreiner B, Utz SG, Leung BP, Lelios I, Heppner FL, Kipnis J, Merkler D, et al. High-dimensional single-cell mapping of central nervous system immune cells reveals distinct myeloid subsets in health, aging, and disease. Immunity. 2018;48(2):380–95 e386.
25. Der E, Ranabothu S, Suryawanshi H, Akat KM, Clancy R, Morozov P, Kustagi M, Czuppa M, Izmirly P, Belmont HM, et al. Single cell RNA sequencing to dissect the molecular heterogeneity in lupus neph
    ritis. JCI Insight. 2017;2(9).

26. Villani AC, Satija R, Reynolds G, Sarkizova S, Shekhar K, Fletcher J, Griesbeck M, Butler A, Zheng S, Lazo S, et al. Single-cell RNA-seq reveals new types of human blood dendritic cells, monocytes, and progenitors. Science. 2017; 356(6335

Stevens, C.F. (1998). Neuronal diversity: Too many cell types for comfort

Mapping the physical network of cellular interactions
# Biocuration




Q55440752
Q36625511
Q91728644
Q91804991
Q92043520



## No QID

 
https://www.aaai.org/Papers/KDD/1996/KDD96-051.pdf

WikiUMLS: Aligning UMLS to Wikipedia via Cross-lingual Neural Ranking
Flach P, Kakas A, Ray O: Abduction, Induction, and the Logic of Scientific
Knowledge Development. In Workshop on Abduction and Induction in AI
and Scientific Modelling Citeseer; 2006. 

The Next Big Thing: From Hypermedia to Datuments
 Improving text mining with controlled natural language: a case study for protein interations.

DisGeNET RDF: A Gene-Disease Association Linked Open Data Resource

Querying biomedical Linked Data with natural language questions

A competency question-oriented approach for the transformation of semi-structured bioinformatics data into linked open data

Lmn-2 interacts with Elf-2. On the meaning of common statements in biomedical literature

Ontologies for the life sciences Steffen Schulze-Kremer and Barry Smith  http://ontology.buffalo.edu/smith/articles/Ontologies_life_sciencesEnc.pdf

http://bioportal.bioontology.org/annotatorplus

On the role of knowlege graphs in
explainable AI.
## Other


Extraction of relations between genes and diseases from text and large-scale data analysis: implications for translational research.

Noy, N. F. et al. Nucleic Acids Res. 37, W170–W173 (2009

Himmelstein,D. Integrating resources with disparate licensing into an open network. https://thinklab.com/discussion/integrat ing-resources-with-disparate-licensing-into-an-open-network/ 107#1 (accessed Sep 16, 2016).

--
Neumann EK, Quan D: Biodash: a semantic web dashboard for drug development. Pac Symp on Biocomput 2006, 11:176-187.
--

Dumontier,M., Callahan,A., Cruz-Toledo,J., Ansell,P., Emonet,V. and Belleau,F. (2014) Bio2RDF release 3: a larger connected

https://academic.oup.com/nar/article/48/D1/D704/5614574

https://www.nejm.org/doi/full/10.1056/NEJMra1615014

 Ratnam,J., Zdrazil,B., Digles,D. et al. (2014) The applica- tion of the open pharmacological concepts triple store (open PHACTS) to support drug discovery research. PLOS ONE, 9, e115460.
6.

Lysenko,A., Roznov˘a¸t,I.A., Saqi,M. et al. (2016) Representing and querying disease networks using graph databases. BioData Min., 9, 23

A behind‐the‐scenes tour of the IEDB curation process: an optimized process empirically integrating automation and human curation efforts

Funding knowledgebases: Towards a sustainable funding model for the UniProt use case

https://academic.oup.com/bioinformatics/article/23/13/i41/238103
https://academic.oup.com/database/article/doi/10.1093/database/baaa006/5809234



https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1626147/
https://academic.oup.com/database/article/doi/10.1093/database/bar064/430006?searchresult=1

https://academic.oup.com/database/article/doi/10.1093/database/bau039/2634247?searchresult=1

https://academic.oup.com/database/article/doi/10.1093/database/baz085/5608006?searchresult=1
https://academic.oup.com/database/article/doi/10.1093/databa/baaa022/5828001

He Y, Yu H, Yang X, Shao C, Zhou W, Zhu Y, Wang H, Liu Q, Xie J, Zheng J, et al. Ontology: foundation of biomedical big data and precision medicine research. Chin J Bioinformatics. 2018;16(1):7–

On expert curation and scalability: UniProtKB/Swiss-Prot as a case study.

Guralnick RP, Cellinese N, Deck J, Pyle RL, Kunze J, Penev L, et al. Community next steps for making globally unique identifiers work for biocollections data. Zookeys. 2015; 133–154. https://doi.org/10. 3897/zookeys.494.9352 PMID: 25901117


Leitner F, Krallinger M, Rodriguez-Penagos C. Introducing meta-services for biomedical information extraction. Genome Biol 2008;9:S6


Crowdsourcing biomedical research: leveraging communities as innovation engines

Nonetheless, some examples of systems that support the integration of text mining with manual annotation are Textpresso [65], PaperBrowser [11], GOAnnotator [70] and PreBIND [74]


Mu¨ller H-M, Kenny EE, Sternberg PW. Textpresso: an ontology-based information retrieval and extraction system for biological literature. PLoS Biol 2004;2:e309.


Tá aqui os papers dos dois:
https://arxiv.org/abs/1902.07669
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3960092/

Tem esse serviço que tenta automatizar o processo com machine learning, mas eu acho bem primitivo ainda https://scite.ai/ ... Uma ontologia que todo mundo usasse ia ser ótimo

Arman Cohan, Waleed Ammar, Madeleine van Zuylen, and Field Cady. 2019. Structural scaffolds for ci- tation intent classification in scientific publications. CoRR, abs/1904.01608.
Nigel

Grenon P, Smith B, Goldberg L: Biodynamic Ontology: Applying BFO in the Biomedical Domain.In Ontologies in Medicine. Edited by Pisanelli DM. Amsterdam: Ios Press; 2004:20–38.
75.

1. Hakala, K., Kaewphan, S., Salakoski, T. & Ginter, F. Syntactic analyses and named entity recognition for PubMed and PubMed Central—up-to-the-minute. In Proceedings of the 15th Workshop on Biomedical Natural Language Processing 102–107, https://doi. org/10.18653/v1/W16-2913 (2016).
2. Bell, L., Chowdhary, R., Liu, J. S., Niu, X. & Zhang, J. Integrated bio-entity network: a system for biological knowledge discovery. PLoS One 6, e21474 (2011).

Lee, J. et al. BioBERT: a pre-trained biomedical language representation model for biomedical text mining. Bioinformatics 36, 1234–1240, https://doi.org/10.1093/bioinformatics/btz682 (2019).



BioGPS: building your own mash-up of gene annotations and expression profiles 
BioGPS and MyGene.info: organizing online, gene-centric information
High-performance web services for querying gene and variant annotation


## Reviews

Text mining resources for the life sciences

Pressing needs of biomedical text mining in biocuration and beyond: opportunities and challenges

## Text mining

Integrating text mining into the MGI biocuration workflow
Accelerating literature curation with text-mining tools: a case study of using PubTator to curate genes in PubMed abstracts
Collaborative biocuration--text-mining development task for document prioritization for curation
Discovering biomedical semantic relations in PubMed queries for information retrieval and database curation
Automated assessment of biological database assertions using the scientific literature
LitSense: making sense of biomedical literature at sentence level


## Ontology

Semi-automated ontology generation within OBO-Edit

The Confidence Information Ontology: a step towards a standard for asserting confidence in annotations

## Other
Knowledge engineering tools for reasoning with scientific observations and interpretations: a neural connectivity use case
Making your database available through Wikipedia: the pros and cons
Building a biomedical semantic network in Wikipedia with Semantic Wiki Links
MyMiner: a web application for computer-assisted biocuration and text annotation
GEOMetaCuration: a web-based application for accurate manual curation of Gene Expression Omnibus metadata
Accelerating annotation of articles via automated approaches: evaluation of the neXtA5 curation-support tool by neXtProt

## Biocreative

BioCreative III interactive task: an overview
Biocuration workflows and text mining: overview of the BioCreative 2012 Workshop Track II
An overview of the BioCreative 2012 Workshop Track III: interactive text mining task
Overview of the interactive task in BioCreative V
BioCreative V BioC track overview: collaborative biocurator assistant task for BioGRID
The BEL information extraction workflow (BELIEF): evaluation in the BioCreative V BEL and IAT track

## Crowd
Crowdsourcing in biomedicine: challenges and opportunities
Crowdsourcing and curation: perspectives from biology and natural language processing
Involving community in genes and pathway curation


# Top 100 papers
DNA sequencing with chain-terminating inhibitors.

Clustal W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.

Nonparametric estimation from incomplete observations.

Basic local alignment search tool.

Fuzzy sets.Zadeh, L. A. Inform. Control 8, 338–353 (1965).

Improved patch-clamp techniques for high-resolution current recording from cells and cell-free membrane patches. Hamill, O. P., Marty, A., Neher, E., Sakmann, B. & Sigworth, F. J.  Pflug. Arch. Eur. J. Physiol. 391, 85–100 (1981).

A rating scale for depression. Hamilton, M. J. Neurol. Neurosurg. Psychiatry 23, 56–62 (1960).

Maximum likelihood from incomplete data via EM algorithm. Dempster, A. P., Laird, N. M. & Rubin, D. B. J. R. Stat. Soc., B 39, 1–38 (1977).

Controlling the false discovery rate: a practical and powerful approach to multiple testing. Benjamini, Y. & Hochberg, Y.J. R. Stat. Soc. B 57, 289–300 (1995).

Isolation of mononuclear cells and granulocytes from human blood. Böyum, A. Scand. J. Clin. Lab. Invest. 21, S77–S89 (1968).

A new and rapid colorimetric determination of acetylcholinesterase activity. Ellman, G. L., Courtney, K. D., Andres, V. Jr & Featherstone, R. M.Biochem. Pharmacol. 7, 88 (1961).

The Protein Data Bank. Berman, H. M. et al. Nucleic Acids Res. 28, 235–242 (2000).

# Random
Q92575623
Q90696426
Q33535717
