# Papers to read

## Cell types

White, J. G., Southgate, E., Thomson, J. N. & Brenner, S. Philos. Trans. R. Soc. Lond., B 314,.1–340 (1986)

Sanes JR, Masland RH. The types of retinal ganglion cells: current status and implications for neuronal classification. Annu Rev Neurosci. 2015; 38:221–246. [PubMed: 25897874]
Schubert
Tyner, C. F. The naming of neurons: applications of taxonomic theory to the study of cellular populations. Brain Behav. Evol. 12, 75–96 (1975). A prescient discussion of how taxonomic principles could be used to improve cell-type classification. Cast in terms of physiological properties (see also REFS 24, 28 and 29), it is even more applicable to modern molecular schemes.

Jointly defining cell types from multiple single-cell datasets using LIGER

Heritability enrichment of specifically expressed genes identifies disease-relevant tissues and cell types

A gene expression atlas of the central nervous system based on bacterial artificial chromosomes

Scala Naturae. Why there is no theory in com? parative psychology

Huxley, J. S.: The new systematics (Oxford Univ. Press, Oxford 1940).

Popper, K. R.: The open society and its enemies; 4th ed., vol. 1 (Routledge & Kegan Paul, London 1962)

Rodieck, R. W. & Brening, R. K. Retinal ganglion cells: properties, types, genera, pathways and trans-species comparisons. Brain Behav. Evol. 23, 121–164 (1983).
A genetic and computational approach to structurally classify neuronal types
Neuronal diversity and convergence in a visual system developmental atlas 
Huang, Z. J. & Zeng, H. Genetic approaches to neural circuits in the mouse. Annu. Rev. Neurosci. 36, 183–215 (2013). This article summarizes the progress enabled by obtaining genetic access to specific cell types, which relied on molecular classification.

Neuronal diversity and convergence in a visual system developmental atlas 

Sanes, J. R. & Masland, R. H. The types of retinal ganglion cells: current status and implications for neuronal classification. Annu. Rev. Neurosci. 38, 221–246 (2015). This article reviews the progress in classifying RGC types in mice. Remarkably, the estimated number has grown from ~30 to >50 in the two years since its publication


Bock, W. J.: Philosophical foundations of classical evolutionary classification. Syst. Zool. 22: 375-392 (1973).

Crick, F. The impact of molecular biology on neuroscience. Phil. Trans. R. Soc. Lond. B 354, 2021–2025 (1999).
The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells

A maximum-likelihood approach for building cell-type trees by lifting
Forcing cells to change lineages

http://celltypes.brain-map.org/

24. Chevrier, S. et al. An immune atlas of clear cell renal cell carcinoma. Cell 169, 736–749 (2017)

Schapiro, D. et al. histoCAT: analysis of cell phenotypes and interactions in multiplex image cytometry data. Nat. Methods 14, 873–876 (2017)

Tasic, B. et al. Adult mouse cortical cell taxonomy revealed by single cell transcriptomics. Nat. Neurosci. 19, 335–346 (2016)
Habib, N. et al. Div-Seq: Single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons. Science 353, 925–928 (2016).
Korem, Y. et al. Geometry of the gene expression space of individual cells. PLOS Comput. Biol. 11, e1004224 (2015).

 A semi-supervised approach for cell phenotypic and functional estimation in tissue microenvironmen

Petilla Interneuron Nomenclature Group, Ascoli, G.A., Alonso-Nanclares, L., Anderson, S.A., Barrionuevo, G., Benavides-Piccione, R., Burkhalter, A., Buz- sa´ ki, G., Cauli, B., Defelipe, J., Faire´ n, A., et al. (2008). Petilla terminology: nomenclature of features of GABAergic interneurons of the cerebral cortex. Nat. Rev. Neurosci. 9, 557–568.


Kurtz A, Seltmann S, Bairoch A, Bittner MS, Bruce K, Capes-Davis A, Clarke L, Crook JM, Daheron L, Dewender J, Faulconbridge A, Fujibuchi W, Gutteridge A, Hei DJ, Kim YO, Kim JH, Kokocinski AK, Lekschas F, Lomax GP, Loring JF, Ludwig T, Mah N, Matsui T, Müller R, Parkinson H, Sheldon M, Smith K, Stachelscheid H, Stacey G, Streeter I, Veiga A, Xu RH.
A Standard Nomenclature for Referencing and Authentication of Pluripotent Stem Cells.

SFujibuchi,W., Kiseleva,L., Taniguchi,T. et al. (2005) Development of cell knowledge base and prediction of cell types and characteristics by gene expression profiles. IPSJ SIG Technical Reports 2005-BIO-2, 33–37 (in Japanese).
14.

Evolution of Cellular Differentiation: From Hypotheses to Models

Theory of cell fate Michael J. Casey  Patrick S. Stumpf  Ben D. MacArthur

CellMarker: a manually curated resource of cell markers in human and mouse 
Xinxin Zhang, Yujia Lan, Jinyuan Xu, Fei Quan, Erjie Zhao, Chunyu Deng, Tao Luo, Liwen Xu, Gaoming Liao, Min Yan, Yanyan Ping, Feng Li, Aiai Shi, Jing Bai, Tingting Zhao, Xia Li, Yun Xiao

Implications of Epigenetic Variability within a Cell Population for “Cell Type” Classification

Reproducibility of Flow Cytometry Through Standardization: Opportunities and Challenges Tomas Kalina

Models of global gene expression define major domains of cell type and tissue identity 

Genomic and molecular control of cell type and cell type conversions

SC2disease: a manually curated database of single-cell transcriptome for human diseases

Phylogenetic analysis of cell types using histone modifications

Study of cell differentiation by phylogenetic analysis using histone modification data

Identifying cell types to interpret scRNA-seq data: how, why and more possibilities

Integrating resident memory into T cell differentiation models

Visual cue-related activity of cells in the medial entorhinal cortex during navigation in virtual reality

Inferring cell type innovations by phylogenetic methods-concepts, methods, and limitations Koryu Kin 1 2

Genomic and molecular control of cell type and cell type conversions

Cell Atlas of The Human Fovea and Peripheral Retina

Geometry of the Gene Expression Space of Individual Cells

Cell-type deconvolution from DNA methylation: a review of recent applications

Cell-type-specific programs for activity-regulated gene expression

Circular RNA expression in human hematopoietic cells is widespread and cell-type specific

scReClassify: post hoc cell type classification of single-cell rNA-seq data

TOAST: improving reference-free cell composition estimation by cross-cell type differential analysis

Cell-type deconvolution in epigenome-wide association studies: a review and recommendations

The selection and function of cell type-specific enhancers.

Cell-type-resolved alternative splicing patterns in mouse liver

https://pubmed.ncbi.nlm.nih.gov/6574104/

J Cell Biol. 1969 Oct 1; 43(1): 32–39.
doi: 10.1083/jcb.43.1.32
PMCID: PMC2107835
PMID: 5824067
A SIMPLE MODEL OF A STEADY STATE DIFFERENTIATING CELL SYSTEM
S. I. Rubinow

Discovering Pathway and Cell Type Signatures in Transcriptomic Compendia with Machine Learning

Computational methods for the integrative analysis of single-cell data 

Evolution of cellular diversity in primary motor cortex of human, marmoset monkey, and mouse. bioRxiv. doi:10.1101/2020.03.31.016972

Batiuk MY. 2020. Identification of region-specific astrocyte subtypes at single cell resolution. Nat Commun 11:1220. doi:10.1038/s41467-019-14198-8

Fuzik J, Z. 2016. Integration of electrophysiological recordings with single-cell RNA-seq data identifies neuronal subtypes. Nat Biotechnol 34:175–183. doi:10.1038/nbt.3443

Gala R, 2019. A coupled autoencoder approach for multi-modal analysis of cell types In: Wallach H, Larochelle H, Beygelzimer A, d\textquotesingle Alché-Buc F, Fox E, Garnett R, editors. Advances in Neural Information Processing Systems 32. Curran Associates, Inc. pp. 9267–9276.

Geirsdottir L, Cross-Species Single-Cell Analysis Reveals Divergence of the Primate Microglia Program. Cell 179:1609–1622.e16. doi:10.1016/j.cell.2019.11.010

Haendel MA,  2014. Unification of multi-species vertebrate anatomy ontologies for comparative biology in Uberon. J Biomed Semantics 5:21. doi:10.1186/2041-1480-5-21

Hodge RD, 2019. Conserved cell types with divergent features in human versus mouse cortex. Nature 573:61–68. doi:10.1038/s41586-019-1506-7


Yao Z,  2020b. A taxonomy of transcriptomic cell types across the isocortex and hippocampal formation. bioRxiv. doi:10.1101/2020.03.30.015214
A community-based transcriptomics classification and nomenclature of neocortical cell types. arXiv [q-bioGN].

Zeng H, Sanes JR. 2017. Neuronal cell-type classification: challenges, opportunities and the path forward. Nat Rev Neurosci 18:530–546. doi:10.1038/nrn.2017.85


# Articles to read 

## Describing Species - Judith Winston
[@doi:10.1017/CBO9781107415324.004]

https://portal.brain-map.org/explore/classes

https://github.com/AllenInstitute/nomenclature

This is a collection of resources to be read and parsed.


Neves M, Damaschun A, Mah N, Lekschas F, Seltmann S, Stachelscheid H, Fontaine JF, Kurtz A, Leser U. Preliminary evaluation of the CellFinder literature curation pipeline for gene expression in kidney cells and anatomical parts. Database 2013, 2013:bat020. doi:10.1093/database/bat020

https://bmcbioinformatics.biomedcentral.com/articles/supplements/volume-18-supplement-17

Mapping the transcriptional diversity of genetically and anatomically defined cell populations in the mouse brain

In 1957, C.H. Waddington proposed a powerful metaphor for how differentiated cell types emerge from a single, totipotent cell (Waddington 1957). T

https://www.nature.com/scitable/blog/bio2.0/discovering_new_cell_types_one/

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3844401/

https://github.com/kevinrue/Hancock

DE QUEIROZ, K.  1994.  Replacement of an essentialistic perspective on taxonomicdefinitions as exemplified by the definition of "Mammalia."  Syst. Biol. 43: 497-510
 
  Replacement of an Essentialistic Perspective on Taxonomic Definitions as Exemplified by the Definition of "Mammalia"
https://www.sciencedirect.com/science/article/abs/pii/S1369848601000036?via%3Dihub

https://www.pnas.org/content/105/51/20067

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2662812/

## Molecular taxonomy of major neuronal classes in the adult mouse forebrain 
\cite{sugino2006molecular

/Genes and homology in nervous system evolution: Comparing gene functions, expression patterns, and cell type molecular fingerprints   \cite{arendt2005genes

Musser, J. M. & Wagner, G. P. Character trees from transcriptome data: origin and individuation of morphological characters and the so-called ‘species signal’. J. Exp. Zool. 324, 588–604 (2015).
Introduces the phenomenon of concerted transcriptome evolution and shows how it influences the comparison of cell type transcriptomes

Mullen, A. C. et al. Master transcription factors determine cell-type-specific responses to TGF-ß signaling. Cell 147, 565–576 (2011). Demonstrates empirically that cell type terminal selectors are responsible for mediating the cell type-specific response to a common cellular signal.

Hobert, O. Terminal selectors of neuronal identity. Curr. Top. Dev. Biol. 116, 455–475 (2016). Outlines the important role of terminal selector genes in establishing and maintaining cell type identity.


Stefanakis, N., Carrera, I. & Hobert, O. Regulatory logic of pan-neuronal gene expression in C. elegans. Neuron 87, 733–750 (2015). A systematic investigation of how different neuronal cell types use both cell type-specific and more general regulatory information to express pan-neuronal genes.

Arendt, D. Genes and homology in nervous system evolution: comparing gene functions, expression patterns, and cell type molecular fingerprints. Theory Biosci. 124, 185-197 (2005). A conceptual review that discusses the different levels of homology in evolutionary developmental biology research; it introduces the concept of molecular fingerprints for cell type comparisons.
Up next: 

% https://www.the-scientist.com/cover-story/advancing-techniques-reveal-the-brains-impressive-diversity-30197
% https://www.sciencedirect.com/topics/medicine-and-dentistry/cell-type
% http://www.diva-portal.org/smash/get/diva2:942241/FULLTEXT01.pdf
% https://sciencing.com/specialized-cells-definition-types-examples-13718073.html
% https://www.nature.com/scitable/blog/bio2.0/discovering_new_cell_types_one/

https://senselab.med.yale.edu/NeuronDB

https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3000691

# Discovery of new cell types:

JJordao MJC, Sankowski R, Brendecke SM, Sagar LG, Tai YH, Tay TL, Schramm E, Armbruster S, Hagemeyer N, et al. Single-cell profiling identifies myeloid cell subsets with distinct fates during neuroinflammation. Science 2019;363(6425):eaat7554. https://doi.org/10.1126/science.aat7554.
23. Gan Y, Li N, Zou G, Xin Y, Guan J. Identification of cancer subtypes from single-cell RNA-seq data using a consensus clustering method. BMC Med Genet. 2018;11(Suppl 6):117.
24. Mrdjen D, Pavlovic A, Hartmann FJ, Schreiner B, Utz SG, Leung BP, Lelios I, Heppner FL, Kipnis J, Merkler D, et al. High-dimensional single-cell mapping of central nervous system immune cells reveals distinct myeloid subsets in health, aging, and disease. Immunity. 2018;48(2):380–95 e386.
25. Der E, Ranabothu S, Suryawanshi H, Akat KM, Clancy R, Morozov P, Kustagi M, Czuppa M, Izmirly P, Belmont HM, et al. Single cell RNA sequencing to dissect the molecular heterogeneity in lupus nephritis. JCI Insight. 2017;2(9).

26. Villani AC, Satija R, Reynolds G, Sarkizova S, Shekhar K, Fletcher J, Griesbeck M, Butler A, Zheng S, Lazo S, et al. Single-cell RNA-seq reveals new types of human blood dendritic cells, monocytes, and progenitors. Science. 2017; 356(6335

Stevens, C.F. (1998). Neuronal diversity: Too many cell types for comfort

## Biocuration

11. Guralnick RP, Cellinese N, Deck J, Pyle RL, Kunze J, Penev L, et al. Community next steps for making globally unique identifiers work for biocollections data. Zookeys. 2015; 133–154. https://doi.org/10. 3897/zookeys.494.9352 PMID: 25901117

## Top 100 papers
DNA sequencing with chain-terminating inhibitors.

Clustal W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.

Nonparametric estimation from incomplete observations.

Basic local alignment search tool.

Fuzzy sets.Zadeh, L. A. Inform. Control 8, 338–353 (1965).

Improved patch-clamp techniques for high-resolution current recording from cells and cell-free membrane patches. Hamill, O. P., Marty, A., Neher, E., Sakmann, B. & Sigworth, F. J.  Pflug. Arch. Eur. J. Physiol. 391, 85–100 (1981).

A rating scale for depression. Hamilton, M. J. Neurol. Neurosurg. Psychiatry 23, 56–62 (1960).

Maximum likelihood from incomplete data via EM algorithm. Dempster, A. P., Laird, N. M. & Rubin, D. B. J. R. Stat. Soc., B 39, 1–38 (1977).

Controlling the false discovery rate: a practical and powerful approach to multiple testing. Benjamini, Y. & Hochberg, Y.J. R. Stat. Soc. B 57, 289–300 (1995).

Isolation of mononuclear cells and granulocytes from human blood. Böyum, A. Scand. J. Clin. Lab. Invest. 21, S77–S89 (1968).

A new and rapid colorimetric determination of acetylcholinesterase activity. Ellman, G. L., Courtney, K. D., Andres, V. Jr & Featherstone, R. M.Biochem. Pharmacol. 7, 88 (1961).

The Protein Data Bank. Berman, H. M. et al. Nucleic Acids Res. 28, 235–242 (2000).