# Papers to read
https://plato.stanford.edu/archives/win2019/entries/truth-revision/
https://seop.illc.uva.nl/entries/models-science/


# Blogs to read
https://bioeticacomplexa.blogspot.com/2020/05/
https://bioeticacomplexa.blogspot.com/2020/07/
https://addshore.com/2021/05/wbstack-setting-changes-federated-properties-wikidata-entity-mapping-more/
https://www.kmjn.org/notes/academic_ancestors_wikidata.html
https://neo4j.com/blog/ontologies-in-neo4j-semantics-and-knowledge-graphs/
# Books to read
Huxley, J. S.: The new systematics (Oxford Univ. Press, Oxford 1940).

Lorhard, J. (1606). Diagraph of Metaphysic or Ontology.St. Gallen: Ogdoas scholastica Book 8. PublishedinSangalli.Trans.Sara L. Uckelman.

Popper, K. R. Objective Knowledge: An Evolutionary Approach. Oxford: Clarendon, 1975.

 Elton CS. 1927. Animal Ecology. https://doi.org/10.5962/bhl.title.7435 

Wagner GP. 2014. Homology, Genes, and Evolutionary Innovatoin.
Princeton, NJ: Princeton University Press

Graph Databases
Specs

By Ian Robinson, Jim Webber and Emil Eifrem
https://neo4j.com/graph-databases-book/?ref=blog

13.  Robinson  PN,  Bauer  S.  Introduction  to  biol-ontologies.  Boca  Raton,  FL:    CRC  Press, 2011
# Cell types
Q60128972
Q42774504
Q92382884
Q35185544
Q104794656
Q56550183
Q42774833
Q37510512
Q38862522
Q34404570
Q42615577
Q91739840
Q50001722
Q46852287
Q52560248
Q37510607
Q52718178
Q37056065
Q47320490
Q92888602
Q91560057
Q92669541
Q59090070
Q60468467
Q93012764
Q102053616
Q37288678
Q91595285
Q91595293
Q42144472
Q91479798
Q96136775
Q92052916
Q89686934
Q92615426
Q64981158
Q92571628
Q75715838
Q34494714
Q34360246
Q91676982
Q64903100
Q40198119
Q101226747
Q92691104
Q92365687
Q92154155
Q91865409
Q92154185
Q93132926
Q59071038
Q63979660
Q46257233
Q52315063
Q57318688
Q59066334
Q59770869
Q91969373
Q41667192
Q42776089
Q91523279
Q91987032
Q57039955
Q92185170
Q35556351
Q52576606
Q91734742
Q90284285
Q92932854
Q57173237
Q89686928
Q92548431
Q39281660
Q44169496
Q67653684
Q91892543
Q60341043
Q90754750
Q29617430
Q90288393
Q94441043
Q52431092
Q29615604
Q92090793
Q37573627
Q88561926
Q60468470
Q90658651
Q91898650
Q55407585
Q92483956
Q52353816
Q58084076
Q64008489
## No QID

## Other
https://azimuth.hubmapconsortium.org/references/#Human%20-%20Pancreas
Terminologia histologicainternational terms for human cytology and histologyFederative International Committee on Anatomical Terminology (FICAT)
C H A P TER 1 WHAT WAS LIFE? Answers from Three Limit Biologies (Helmreich)
Meta-analytic markers reveal a generalizable 1description of cortical cell types
Barcoded viral tracing of single-cell interactions in central nervous system inflammation

# Biocuration
Q58587002
Q101222862
Q49826285
Q58611639
Q47555736
Q56992710
Q37060118
Q93337500
Q58611634
Q100525924
Q28646868
## No QID

https://www.biogateway.eu/ssb/
Himmelstein,D. . https://thinklab.com/discussion/integrat ing-resources-with-disparate-licensing-into-an-open-network/ 107#1 (accessed Sep 16, 2016).
Lmn-2 interacts with Elf-2. On the meaning of common statements in biomedical literature
The Next Big Thing: From Hypermedia to Datuments
https://www.semanticscholar.org/paper/Ontology-%3A-Foundation-of-biomedical-big-data-and-Jie-Yongqun/e94c4a60a8e58daaf39952e06266d197a81ca24f

https://scite.ai/ 

Comments to the first nomenclature of human cytology: the description of cells and their ultrastructure in the Terminologia Histologica. Which important medical and biological terms are disputable or missing?

## Other

[PDF] Towards a Definition of Knowledge Graphs.



https://www.ebi.ac.uk/spot/zooma/


COELHO, KatiaCardoso. Aquisição de conhecimento especializado para construção de ontologias: um estudo no domínio das ciências da vida. 2012.174f. Dissertação (Mestradoem Ciência da Informação). Escola de Ciência da Informação, Universidade Federal de Minas Gerais, Belo Horizonte, 2012. Disponível em: http://www.bibliotecadigital.ufmg.br/dspace/bitstream/handle/1843/ECID-8XLQS4/disserta__o_katia_cardoso_coelho.pdf?sequence=2. Acesso em: ago.2014

Bio4j: a high-performance cloud-enabled graph-based data platform.
Search, access, and explore life sciencenanopublications on the Web
Micropublication: incentivizing community curation and placing unpublished data into the public domain

SCHRIML, Lynn M.; et al. Disease Ontology: a backbone for disease semantic integration. Nucleic Acids Research. v.40. 2012. Disponível em: http://nar.oxfordjournals.org/content/40/D1/D940.full.pdf?keytype=ref&ijkey=oOOVAj4zMzRpayq. Acesso em:21 set 2014

USCHOLD, Mike.Building ontologies: Towards a Unified Methodology (1996). Disponível em:http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.39.9075. Acesso em:15 fev 2013

The role of trust in knowledge

https://doi.org/10.1093/nar/gks1193

 Colin N. Dewey. 2017. “MetaSRA: Normalized HumanSample-Specific Metadata for the Sequence Read Archive.”Bioinformatics (Oxford, England)33 (18): 2914–23. https://doi.org/10.1093/bioinformatics/btx334.

 https://www.zotero.org/google-docs/?OzuVZW

“Minimum Information about a Microarray Experiment(MIAME)—toward Standards for Microarray Data.”Nature Genetics29 (4): 365–71.https://doi.org/10.1038/ng1201-365.

Galeota, Eugenia, and Mattia Pelizzola. 2017. “Ontology-Based Annotations and SemanticRelations in Large-Scale (Epi)Genomics Data.”Briefings in Bioinformatics18 (3): 403–12.https://doi.org/10.1093/bib/bbw036.

“ALE: Automated LabelExtraction from GEO Metadata.”BMC Bioinformatics18 (14): 509.https://doi.org/10.1186/s12859-017-1888-1.Hadley, Dexter, James Pan, Osama El-Sayed,

Jihad Aljabban, Imad Aljabban, Tej D. Azad,Mohamad O. Hadied, et al. 2017. “Precision Annotation of Digital Samples in NCBI’s GeneExpression Omnibus.”Scientific Data4 (1): 170125. https://doi.org/10.1038/sdata.2017.125.


“Ontology-AwareClassification of Tissue and Cell-Type Signals in Gene Expression Profiles across Platforms andTechnologies.”Bioinformatics29 (23): 3036–44. https://doi.org/10.1093/bioinformatics/btt529.


“Real-Time Tagging of Biomedical Entities.”BioRxiv, September, 078469. https://doi.org/10.1101/078469.Panahiazar, Maryam, Michel Dumontier, and Olivier Gevaert. 2017. 

“Predicting BiomedicalMetadata in CEDAR: A Study of Gene Expression Omnibus (GEO).”Journal of BiomedicalInformatics72 (August): 132–39. https://doi.org/10.1016/j.jbi.2017.06.017.


“Extraction and Analysis ofSignatures from the Gene Expression Omnibus by the Crowd.”Nature Communications7 (1):12846. https://doi.org/10.1038/ncomms12846.


# Top 100 papers
# Random
Q92575623
Q90696426
Q33535717
Q92575623

