
Besca, a single-cell transcriptomics analysis toolkit to accelerate translational research
==========================================================================================
  
  [@wikidata:Q109755007]  
  
Publication date : 04 of October, 2021  

# Highlights
Here, we present the toolkit Besca, which streamlines scRNA-seq analyses and their use to deconvolute bulk RNA-seq data according to current best practices. Beyond a standard workflow covering quality control, filtering, and clustering, two complementary Besca modules, utilizing hierarchical cell signatures and supervised machine learning, automate cell annotation and provide harmonized nomenclatures.

Besca is an open-source Python library that is compatible with and extends Scanpy (18)

the Besca proportions estimate (Bescape) module integrates SCDC (20) and MuSiC (21), allowing to directly apply Besca-generated results and cell annotations to deconvolute bulk transcriptomics data (Figure 1E)

Cell type annotations were harmonized and can be reused across studies, also allowing to cross-validate the discovery of new cell types from multiple studies such as inflammatory fibroblasts in colitis described below

Analysis results are exported into interoperable file formats to allow FAIR data management of analysis results. This includes the Matrix Market exchange format (https://math.nist.gov/MatrixMarket/formats.html) for sparse count matrices, GCT (https://software.broadinstitute.org/software/igv/GCT) for dense count matrices, and simple tab-separated or comma-separated values formats for metadata and as interface for the cell deconvolution package Bescape, respectively.

The annotation process has three components:

a nomenclature table with long and short names, according to Cell Ontology (42), see Supplementary Table S3 and https://github.com/bedapub/besca/blob/master/besca/datasets/nomenclature/CellTypes_v1.tsv

--> - 1.3.1.4. Cell Ontology (and CELDA)


Finally, the obtained annotation will all be in line with the nomenclature table (component 1) and thus the Cell Ontology (42), greatly facilitating comparisons between studies and analyses, as also exemplified here.

 As the development of suitable scRNA-seq integration data increases, a key challenge in single-cell data analysis workflows is the accurate dissemination of this knowledge and the appropriate reuse of the information gathered. In particular, it is of utmost importance to be able to re-apply gene signatures extracted from individual studies across studies and within analyses.


In contrast to other single cell analysis toolkits (18,32,33,52), Besca is focusing on the standardization and reusability of analyses, and therefore we connected Besca to the Geneset Management System (GeMS) (https://github.com/bedapub/GeMS).

Once GeMS is deployed, Besca allows the export of gene signatures to the GeMS database (for example a geneset of marker genes from distinct populations) and the retrieval of user-defined signatures (Figure 1A)

To guarantee consistency across studies and communities, beyond scRNA-seq, the proposed cell type annotation schemas are based on the Cell Ontology (42), which is accessible via the Experimental Factor Ontology (67). The controlled vocabularies at different cell type hierarchies are summarized in a nomenclature sheet (Supplementary Table S3) and can be easily extended with further cell types. Newly generated cell type annotations in this manuscript provide the most fine-grained annotation as DBlabel assignment, which follows the Cell Ontology whenever possible, as well as higher level annotations according to the nomenclature sheet distributed with Besca.

In addition to the signature-based annotation approach, Besca provides the Auto-annot module (Figure 1D), a supervised machine learning workflow for automated cell type annotation based on well annotated training datasets

# Comments

## Tags
- 1.4.2.2. Computational analysis of scRNA-seq data

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q109755007)  
 * [Wikidata](https://www.wikidata.org/wiki/Q109755007)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q109755007&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1093/NARGAB/LQAB102)  
