
Promoting coherent minimum reporting guidelines for biological and biomedical investigations: the MIBBI project
===============================================================================================================
  
  [@wikidata:Q29301289]  
  
Publication date : 01 of August, 2008  

# Highlights
The Minimum Information for Biological and Biomedical Investigations (MIBBI) project provides a resource for those exploring the range of extant minimum information checklists and fosters coordinated development of such checklists.

 Many journals and funding agencies now require that authors reporting  microarray-based  transcriptom-ics  experiments  comply  with  the  Minimum Information about a Microarray Experiment (MIAME) checklist3 as a prerequisite for pub-lication4–7

 In  summary,  we  present the  MIBBI  project  (http://www.mibbi.org/), which maintains a web-based, freely accessi-ble resource for checklist projects, providing straightforward access to extant checklists (and to  complementary  data  formats,  controlled vocabularies,  tools  and  databases

It is clear that checklists should be developed through  close  consultation  with  their  sponsoring  practitioner  communities,  but  such checklists should also, we believe, be designed to anticipate ‘cross-domain’ integrative activi-ties.


MIRIAM (http://biomodels.net/miriam)The Minimum Information Requested In the Annotation of biochemical Models (MIRIAM) checklist offers formal requirements for describing theoretical models of biochemical systems

MIAPA (http://www.mibbi.org/index.php/projects/MIAPA)The Minimum Information About a Phylogenetic Analysis (MIAPA) checklist relates to the use of software to align biological sequences, and the subsequent use of algorithms to construct phylogenies orcladograms and to draw inferences from

--> Not about experiment

 develop (separa-ble) community-level extensions to shared core standards such as MIAME to better describe domain-specific studies (for example, in envi-ronmental biology13) are clearly the order of the day.

MIBBI  has  two  key  parts.  The  first  is  the ‘Portal’, which exists simply to raise awareness of, and afford more straightforward access to, a wide range of checklists by providing research-ers, journal editors, reviewers, funders and the wider community of checklist developers with a quick and simple way to discover (whether there  is)  a  checklist  addressing  a  particular area  and  to  establish  the  scope  and  progress of the underlying project.

The   second   key   part   of   MIBBI   is   the ‘Foundry’. Communities can, if motivated, sign up with the Foundry to jointly examine ways to refactor the checklists over which they have control and then to develop a suite of self-con-sistent, clearly bounded, orthogonal, integrable checklist modules. 

he concepts have been designed to capture the content of a checklist in an intuitive but compact manner, which means that some concepts represent a large body of methods and technologies (for example,‘nucleic acid sequencing’). However, where a component of such a broad concept was found to have an analog in another project’s checklist, that component was factored out to form a new standalone concept (for example,‘detection/tagging/staining’ is a concept common to workflows involving microarraying, gel electrophoresis and mass spectrometry.



# Comments

They are mostly about experiments/assay/data, different from a new theoretical class.


## Tags
--> general description of MIBBI 3.10 Minimal Information Abount New Cell Classes

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q29301289)  
 * [Wikidata](https://www.wikidata.org/wiki/Q29301289)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q29301289/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q29301289&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1038/NBT.1411)  
