
The Cell Ontology 2016: enhanced content, modularization, and ontology interoperability.
========================================================================================
  
  [@wikidata:Q36067763]  
  
Publication date : 04 of July, 2016  

# Highlights

The content in the CL is populated via gradual and en
masse class additions, most notably through several rounds
of improvements to representation of hematopoietic cells
in the ontology [2–4].

The CL is built according to the principles established
by the OBO Foundry [5] and is the designated candidate
ontology for metazoan cell types within the Foundry.
The domain and content of CL is intended to be orthogonal to other Foundry ontologies to allow for the construction of compositional classes via logical definitions,
as exemplified by the Gene Ontology (GO) [3, 6–8].

The CL is maintained primarily by a small group of editors
(ADD, YB, MH, DOS, CVS, NV, CJM), working in conjunction with interested parties from the ontology community

To ensure that CL classes are applicable to nonmammalian vertebrates two courses of action have been
necessary: 1) add non-mammalian classes to the CL; 2)
ensure that general cell type definitions do not unintentionally exclude certain organisms. Examples of nonmammalian cell types that have been recently added to
the CL include the pigmented cells ‘iridoblast’
(CL:0005001) and ‘xanthoblast’ (CL:0005002) [23], and
the ‘Kolmer-Agduhr neuron’ (CL:0005007) [24]. 


While the CL continues to represent a number of high
level insect cell types, the Drosophila Anatomy Ontology
(FBbt) contains cell types for many insect cell types not
represented in CL, particularly insect neurons [33–35].
Similarly, the Zebrafish Anatomy Ontology (ZFA) also
contains neuron types not represented in CL [36]. Going
forward, the general approach is that non-mammalian
species-specific cell types will be represented as is_a
children of the appropriate CL parent in the speciesspecific anatomy ontology when such an ontology exists.
The CL will continue to maintain general cell types for
representation of non-mammalian cells where no separate resource or ontology exists and will remain the principal ontology for the representation of mammalian cell
types.


We have also implemented a TermGenie [47] instance,
available at cl.termgenie.org
--> Never used that

Use of the CL as Metadata in ENCODE and FANTOM5
Projects
Two major projects studying gene expression have utilized the CL as part of their data analysis pipelines. The
Encyclopedia of DNA Elements (ENCODE) Consortium,
which is funded and organized by National Human Genome Research Institute (NHGRI), aims to discover and
define the functional elements encoded in the human
genome [59].


The CL is being used as metadata in a variety of nonontology projects, such as The Cell: An Image Library
[66], CELLPEDIA [67], Phenoscape [13], LINCS [68],
the Human Immunology Project Consortium (HIPC)
[11], and ImmPort [10].

We invite
community feedback and participation to continue the
improvements to the CL.


# Comments

## Tags
- 1.3.1.4. Cell Ontology (and CELDA)

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q36067763)  
 * [Wikidata](https://www.wikidata.org/wiki/Q36067763)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q36067763&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1186/S13326-016-0088-7)  
