
Single-Cell Multi-omic Integration Compares and Contrasts Features of Brain Cell Identity
=========================================================================================
  
  [@wikidata:Q92614159]  
  
Publication date : 06 of June, 2019  

# Highlights
A platform called LIGER allows for the
integration of gene expression,
epigenetic regulation, and spatial
relationships across single-cell datasets.

LIGER reveals inter-individual differences in bed nucleus and
substantia nigra cells

First, we defined region-specific and sexually dimorphic gene expression in the mouse bed nucleus of
the stria terminalis. Second, we analyzed expression
in the human substantia nigra, comparing cell states
in specific donors and relating cell types to those in
the mouse

(A) t-SNE visualization of 74,910 bed nucleus neurons analyzed by LIGER, colored by cluster, and labeled by an exclusive marker.

Initial
clustering identified 106,728 neurons, of which 70.2% were
localized to BNST by examination of marker expression in the
Allen Brain Atlas (ABA)

ough, to our
knowledge, NG cells have not been described in the BNST
before, cells with NG-like anatomy and physiology have been
observed within the amygdala (Manko et al., 2012  ), a structure
with related functional roles

A credible definition of cell type requires distinguishing the
invariant properties of cell identity from the dispensable across
a myriad of settings and measurements. LIGER promises to be
a broadly useful analytical tool for such efforts because of
several key technical advantages. 


# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q92614159)  
 * [Wikidata](https://www.wikidata.org/wiki/Q92614159)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q92614159&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1016/J.CELL.2019.05.006)  
