
Jointly defining cell types from multiple single-cell datasets using LIGER
==========================================================================
  
  [@wikidata:Q100526347]  

# Highlights

Here, we provide a step-by-step protocol for using LIGER to jointly define cell types from multiple single-cell datasets.

when leveraging such datasets to define cell identity, we want to capture both discrete
cell types and continuous variation such as cell states

--> Interesting that continuous variation is sometimes used as a synonym of cell states

Once cell clusters have been identified, these are usually interpreted in terms of known and novel cell types by manually annotating clusters using known marker genes or applying statistical models to link clusters to cell types

bed nucleus of the stria terminalis (BNST). Through the analysis of scRNA-seq data from BNST, we found notable sexual dimorphism in the gene expression patterns of multiple cell types.

Single-cell transcriptomic, methylation, and chromatin accessibility data have all been used to jointly define cell types.

The code is freely available at https://github.com/MacoskoLab/liger. The code is also available through an assigned DOI at https://doi.org/10.5281/zenodo.3765403.
# Comments

Not defining, they are identifying.