
SIMPLEs: a single-cell RNA sequencing imputation strategy preserving gene modules and cell clusters variation
=============================================================================================================
  
  [@wikidata:Q100493043]  
  
Publication date : 28 of September, 2020  

# Highlights

 We propose a statistical model-based approach, called SIMPLEs (SIngle-cell RNA-seq iMPutation and celL clustErings), which iteratively identifies correlated gene modules and cell clusters and imputes dropouts customized for individual gene module and cell type. Simultaneously, it quantifies the uncertainty of imputation and cell clustering via multiple imputations

 Given the log-normalized data (e.g. logarithm of one plus RPKM, FPKM, or TPM), we model the gene expression within a cell type by a zero-inflated censored multivariate Gaussian distribution, denoted as ZCN+ (Equation 2). If the dataset contains multiple cell types, we assume that the gene expression level across all the cells follows a mixture of ZCN+ distributions.

 s://s3.amazonaws.com/czbiohub-tabula-muris/TM_facs_mat.rds. The cell annotation was obtained from https://raw.githubusercontent.com/czbiohub/tabulamuris-vignettes/master/data/TM_facs_metadata.csv, which contains the cell ontology class for each cell. The cell ontology data were downloaded from https://raw.githubusercontent.com/obophenotype/cell-ontology/master/cl-basic.obo.
# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q100493043)  
 * [Wikidata](https://www.wikidata.org/wiki/Q100493043)  
 * [Author Disambiguator](https://author-
disambiguator.toolforge.org/work_item_oauth.php?id=Q100493043&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1093/NARGAB/LQAA077)  
