
Learning for single-cell assignment
===================================
  
  [@wikidata:Q101132241]  
  
Publication date : 30 of October, 2020  

# Highlights
Efficient single-cell assignment without prior marker gene annotations is essential for single-cell sequencing data
analysis. Current methods, however, have limited effectiveness for distinct single-cell assignment

To this end, we present scLearn, a learning-based
framework that automatically infers quantitative measurement/similarity and threshold that can be used for
different single-cell assignment tasks

--> aka distance metric + clustering

e. Efficiently assigning cells into
proper cell types or states presents to be one of the grand challenges
in single-cell data analysis (7).

--> - 1.4.2.3. Cell label identification

Recently, cell type assignment strategies without prior marker
gene annotations have emerged to solve these issues (8â€“15). Basically,
these strategies use a large amount of labeled datasets as a reference
to automatically assign or categorize cell types for the new query
cells

A comprehensive study of the existing single-cell assignment methods indicated that several issues remain: 
(i) the assignment performance of existing methods is greatly influenced by different cell types and
datasets for both positive control scenario and negative control scenario, failing to achieve a well-generalized and robust assignment performance in different cell types, and 
(ii) the current methods are biased toward one specific test scenario. 

he most likely situation is that some of the
query cell types are in the reference (positive controls), while the
others are not (negative controls). The former needs to be accurately
assigned with correct labels as many as possible, while the latter should
be labeled as unassigned as completely as possibl

Extending scLearn to multilabel single-cell assignment

Temporal information is also essential for single-cell data analysis
in certain application scenarios, especially for cell development and
lineage analysis (20, 21). In such cases, individual cell commonly
requires temporal annotations besides cell type categorization.
Every cell can be labeled in two aspects, i.e., cell type and time point.
Therefore, assigning or labeling single cells with proper time status during the development besides categorizing the cell type is highly
desired. This problem can be formulated as a multilabel single-cell
assignment task


The temporal information is also important for many single-cell
studies such as cell development and lineage analysis (20, 21).
Therefore, an efficient multilabel single-cell assignment is highly
desired. The multilabel single-cell assignment paradigm presented
in scLearn serves as a novel contribution to single-cell data analysis
community.


# Comments

## Tags
- 1.4.2.3. Cell label identification
- 
# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q101132241)  
 * [Wikidata](https://www.wikidata.org/wiki/Q101132241)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q101132241/P921%3BP4510)  
