
Integrated Single-Cell Atlases Reveal an Oral SARS-CoV-2 Infection and Transmission Axis
========================================================================================
  
  [@wikidata:Q101210731]  

# Highlights

We hypothesize that the diverse epithelia of the oral cavity, from mucosa to SGs, can be infected by SARS-CoV-2.

To test this hypothesis, we generated the first integrated human oral single cell RNA sequencing atlases to predict cellular sources of susceptibilities to SARS-CoV-2 infection. In situ hybridization (ISH) was used to confirm ACE2 and TMPRSS expression to epithelia of the SGs and the oral and oropharyngeal mucosae.

human oral single cell RNA sequencing (scRNAseq) datasets from the SGs (n=5, labial minor) and oral mucosae (n=4, gingiva and hard palate mucosae) were aggregated and analyzed to create a single cell reference atlas of the oral cavity (Figure 2)

Overlaid UMAPs delineated by color reveal similar and unique populations based on sample type. (b, Right) These 50 populations were jointly annotated before integration into 33 cell clusters for the first human pan-oral cell atlas.

The salivary epithelial compartment of the scRNAseq atlas revealed six major cell cell types including serous acini (PRR4, ZG16B), mucous acini (MUC5B, BPIFB2), ducts (S100A2, KRT14), myoepithelia (ACTA2), glial cells (CHGA), and intriguingly ionocytes (CFTR, FOXI1, V-ATPase) which were found in higher pro-portion than in the lung (38).

--> Cool results! The "scRNAseq atlas _revealed_" nomenclature is a bit slippery to me, though.

The connective tissue compartment consisted of fibroblasts (DCN, LUM), endothelium (arterial (CELP); venules (FOXD3); capillaries (C4), smooth muscle (DES), and pericytes (PTH1R). Reflecting the consistent presence of immune cell, 8 distinct populations of immune cells were identified and was comprised of a high proportion of IgA+ plasma cells, B-lymphocytes (MS4A1), T helper (Th) cells (CD154), and two subsets of CTLs (CD8A), subtype 1 (EOMES, GZMA), and subtype 2 (SPRY, ITGA1). The myeloid-derived compartment included a subpopula-tion of conventional DC2 (CD1C) and two different types of macrophages: subtype 1 (CD163) and subtype 2 exhibited an activation signature (C3, FCGR3A, IFNGR1, CX3CR1).


--> Interesting set of "common knowledge" (no reference!) markers.

joint annotation of the SG and gingival scRNAseq atlases was performed. Thirty-three unique cell types were identified.

--> "clusters" and "cell types" are used interchangeably.

Collaborative efforts led by the Human Cell Atlas (HCA) utilized single cell RNA sequencing (scRNAseq) datasets from across the body to examine cell-specific SARS-CoV-2 tropism, leading to the COVID-19 Cell Atlas (https://www.covid19cellatlas.org/).

--> Many datasets in the website. Very cool.
# Comments

Nice drawings of the different compartments.

Discussions seem a bit hurried, I bet the final article will be more readable.

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q101210731)  
 * [Wikidata](https://www.wikidata.org/wiki/Q101210731)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q101210731/P921%3BP4510)  
