
KG-COVID-19: a framework to produce customized knowledge graphs for COVID-19 response
=====================================================================================
  
  [@wikidata:Q102043840]  
  
Publication date : 09 of November, 2020  

# Highlights

G-COVID-19 is a framework for producing customizedCOVID-19 knowledge graphsdOur knowledge graph and framework is free, open-source,and FAIR

 Evidence suggests thatremdesivir (DrugBank: DB14761), a broad-spectrum antiviralmedication, can shorten the time to recovery in adults hospital-ized with COVID-19 infection and pneumonia (though the effectis not statistically significant)4and more recent evidencesuggests that dexamethasone (DrugBank:DB01234), a cortico-steroid that suppresses inflammation, may reduce mortality inpatients with severe COVID-19.5However, currently no treat-ment is available to prevent progression of COVID-19 to severedisease, and our knowledge of the causes and optimal medicalmanagement of the symptoms and resulting clinical complica-tions of COVID-19 are limited


data sources are alsooften stored in different formats, requiring transformation or pre-processing in order to serve the task at hand. For example, toexamine the function of proteins targeted by Food and DrugAdministration (FDA)-approved antiviral drugs, one must down-load and integrate drug, drug target, and FDA approval statusdata from, for example, Drug Central in a custom-made TSVformat7and functional annotations from, for example, GeneOntology in GPAD format.8Furthermore, many datasets are up-dated periodically, which requires researchers to re-downloadand re-process data in order to perform their analysis on themost current data.

TherehavebeenafewparalleleffortstoconstructKGstointegrateCOVID-19 data, each integrating different data sources and con-structed for different purposes. Several efforts have constructedKGs by ingesting and transforming scientific literature10(https://lg-covid-19-hotp.cs.duke.edu/), some with a few additional typesof data also included, such as confirmed case and mortality data(https://github.com/covidgraph/); clinical information, drug trial,and    sequencing    data    (https://www.wikidata.org/wiki/Wikidata:WikiProject_COVID-19); drug, drug trial, and genomesequence  data  (https://ds-covid19.res.ibm.com/).

--> Nice that they mention the WP Covid19! 

Here, we present a comprehensive COVID-19 KG derivedfrom 13 knowledge sources and containing 377,482 nodes and21,433,063 edges. The KG is freely available for download athttps://kg-hub.berkeleybop.io/kg-covid-19/, and the frameworkto produce the KG is freely available athttps://github.com/Knowledge-Graph-Hub/kg-covid-19.  The  knowledge  graphwas constructed using modern ontology best practices wherebydifferent data sources were normalized and merged. KG-COVID-19 allows flexible remixing of component subgraphs for usersinterested in specific areas. We demonstrate several use casesincluding graph-based machine learning.

we also ingest data sourcesthat our user community requests by opening tickets on our proj-ect GitHub page (https://github.com/Knowledge-Graph-Hub/kg-covid-19)

While our framework offers flexibility in deciding how best totransform each data source, KG-COVID-19 follows some gen-eral design principles to maintain the quality of the resulting KG


We use a core set of standardized ontologies and the BiolinkModel (https://biolink.github.io/biolink-model), a biological datamodel for categorizing nodes and edges, to facilitate interopera-bility and data summarizatio

Preserve ProvenanceEach ingest adds a ‘‘provided_by’’column in the edge TSV file,

We ingest data about COVID-19 scientific publica-tions to identify instances of concepts such as GO terms, Uni-Prot Knowledgebase (UniProtKB) proteins, National Center forBiotechnology Information (NCBI) and HGNC genes, andChEMBL  IDs  via  SciBite  annotations


The idea behind a KG-Hub is to provide a platform for buildingand exchanging knowledge graphs by following a set of guide-lines  and  design  principles  (https://knowledge-graph-hub.github.io/) that facilitates interoperability and reproducibility.

Figure 5. Hypothesis-Based Querying of KG-COVID-19 Knowledge Graph for Using SPARQL Queries

--> Pretty much exactly what I've done via Wikidata.

UniProtuses PRO_0000338259 to identify the cleavage product nsp5,the 3C-like protease, if it is cleaved from replicase polyprotein1a, and PRO_0000449623 if it is cleaved from replicase polypro-tein 1ab. Protein Ontology, in contrast, uses PR_000050274, ir-respective of the polyprotein from which it was cleaved. Notethat the UniProt ‘‘PRO_’’ prefix is unrelated to the ProteinOntology namespace. 

--> So confusing! Also 2.5. The concept of "protein"

# Comments

## Tags


1.2.4. Wikidata and Knowledge Graphs
1.3. Knowledge Representation in biology
1.3.2.3. Bio2RDF and semantic databases

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q102043840)  
 * [Wikidata](https://www.wikidata.org/wiki/Q102043840)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q102043840/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q102043840&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+workhttps://tabernacle.toolforge.org/?#/tab/manual/Q102043840/P921%3BP4510)  
