
Single nucleus and in situ RNA sequencing reveals cell topographies in the human pancreas
=========================================================================================
  
  [@wikidata:Q102206138]  
  
Publication date : 16 of November, 2020  

# Highlights

WHAT YOU NEED TO KNOW

BACKGROUND AND CONTEXT
Acomprehensive  transcriptomic  characterization  ofhuman pancreatic cell types has been lacking, mostlydue to technical limitations.

NEW FINDINGS Novel protocols facilitated the generation of single-nucleus  RNA-seq  data  and  the  identification  ofunprecedented cell type heterogeneity in the healthyadult, neonatal and in the diseased human pancreas.

LIMITATIONS This study made use of snap-frozen biopsies of human pancreata; further perturbation studies are needed usinganimal orin vitromodels.

IMPACT These data lay the foundation for future studies and comparisons of cell composition, transcriptomic andspatial alterations in pancreatic diseases.

LAY SUMMARY This study generated thefirst comprehensive human pancreas cell atlas and revealed the existence of novelcell types and states in this organ

--> Very nice section

We created thefirstcomprehensive atlas of human pancreas cells including epithelialand nonepithelial constituents, and uncovered 3 distinct acinarcell types, with possible implications for homeostatic and inflammatory processes of the pancreas

(A) UMAP plots showing the expression levels of the endocrine cellmarkers GCG(alpha cells),INS(beta cells),PPY(gamma cells), andSST(delta cells). 
--> cool

In Situ Sequencing by CARTANA and H&E StainingSections (4mm) were cut from formalin-fixed, paraffin-embedded pancreatic tissue prepared as described for RNA-FISH.Four DNA probes for each target gene were designed andsupplied by CARTANA,

Acinar cells (70% of the nuclei) wereidentified based on the expression levels of digestive en-zymes such asCPA1/2andPRSS1and hallmark transcrip-tion factors (TFs)RBPJLorFOXP2(aBHLHA15/MIST1target)

nlikein prior studies,2,4,7,15we identified 2 distinct ductal sub-types (Figure 2A)


 identified pancreatic stellate cells (PSCs)and distinguished 2 states of PSCs called quiescent (qPSCs)and activated pancreatic stellate cells (aPSCs).
 
One population of acinar cells (acinar-REGþ)expressed higher levels of mRNAs encoding the regenerat-ing (REG) protein family members such asREG3A,REG3G,andREG1B(Figure 4Aand Supplementary Figure 4). Acinar-REGþcells were reported in a previous scRNA-seq study2and represent a population of cells likely linked to thedevelopment of pancreatic lesions such as acinar-to-ductalmetaplasia and pancreatic intraepithelial neoplasia.1

. The acinar cellsubtype with higher numbers of UMIs was characterized byhigher expression levels of 21 genes (Supplementary Ta-ble 2)
confirming previous reports estimating that the majority ofthe mRNA molecules in a pancreatic acinar cell encode forfewer than 30 proteins.19We named this subset secretory acinar cells (hereafter, acinar-s).

The other acinar cell type also expressed digestiveenzyme genes but at markedly lower levels compared toacinar-s  cells  (<4%  vs>50%)

Thus, these acinar cells appear less robust intheir protein secretion and, instead, enriched for respon-siveness to external stimuli, like islet signals, and activationof the endocytic pathway. We named this subset idlingacinar cells (hereafter, acinar-i).


Second,  we  applied  single-cell  regulatory  networkinference and clustering (SCENIC) to infer TF-target regu-latory networks (regulons) from single-cell gene expres-sio


Second,  we  applied  single-cell  regulatory  networkinference and clustering (SCENIC) to infer TF-target regu-latory networks (regulons) from single-cell gene expres-sio

In exocrine cells from neonates, we detected acinar-i andacinar-s cells (Figure 6A). Approximately 4% of the tran-scripts in acinar-i cells encode for digestive enzyme genescompared to 33% in acinar-s, similar to the levels found incognate adult cells (Figure 6A). However, we did not detectAMY2A,AMY2B, andPNLIPin neonatal acinar-s cells,consistent with priorfindings that little to no pancreaticamylase or lipase enzyme activity is detectable in newborns(Figure 6B).28Moreover, we did not detect a population ofacinar-REGþcells, suggesting a further layer of REG proteinregulation  during  postnatal  to  adult  development(Figure 5B).


Our studies showed that acinar-REGþcells were absentin the neonatal pancreas, suggesting that their functionmight be specific for the adult tissue.


--> newborn x adult pancreas; 2.4.7.3 Differentia for cell classes: age and life stage

Few immune cells were detected in the human adulthealthy pancreas, but in the neonatal pancreas, we iden-tified at least 3 immune populations, including follicular Bcells (CD19,CR2,CD22,FCER2), T lymphocytes (CD247/CD3Z,CD3G,IL7R),  and  macrophages  (CD14,CD86,CSF2RA)


Chronic pancreatitis (CP) is a progressive inflammatorysyndrome in which damage of the pancreatic tissue leads toacinar atrophy,fibrosis, and chronic inflammation.44Here, snap-frozen samples from 2 CP donors (Supplementary Table 5) wereprocessed for sNuc-seq, and despite the extensivefibrosis andreduced cellular content (Supplementary Figure 10A), we suc-cessfully profiled 2726 nuclei. Cell types were identified by usinggene markers established in the healthy tissue; moreover, wedetected a chemosensory (tuftcells) and 2 immune (mast cellsand  monocytes)  CP-specific  cell  types  (SupplementaryFigure 10BandC). 

--> Chronic pancreatitis, disease-specific cell types; 2.4.7.6 Differentia for cell classes: disease condition




 # Comments

 Very nice study, quite comprehensive. No bullshit, just clear description, a bit following the case report school, I think.

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q102206138)  
 * [Wikidata](https://www.wikidata.org/wiki/Q102206138)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q102206138/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q102206138&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1053/J.GASTRO.2020.11.010)  
