
CSEA-DB: an omnibus for human complex trait and cell type associations
======================================================================
  
  [@wikidata:Q102205746]  
  
Publication date : 19 of November, 2020  

# Highlights

Here, we present a reference database for GWAS trait-associated cell type-specificity, named Cell type-Specific Enrichment Analysis DataBase (CSEA-DB, available at https://bioinfo.uth.edu/CSEADB/). Specifically, we curated total of 5120 GWAS summary statistics data for a wide range of human traits and diseases followed by rigorous quality control.

<!-- super interesting stuff -->

. The tissues and cell types were annotated with Uberon and Cell Ontology. By applying our deTS algorithm, we conducted 10 250 480 times of trait-cell type associations, reporting a total of 598 (11.68%)

 Human bodies are composed of 11 major organ systems, ∼100 organs/tissues, and more than 100 unique cell types or thousands of sub-cell types (8). 

<!-- Interesing numbers. Reference is funny: Tortora G.J., Derrickson B.H. Introduction to the Human Body. 2017; John Wiley & Sons.  -->

In this work, we aim to systematically explore the genetic signals of complex traits and diseases underlying human cell types.

 GWAS Catalog (15) and GRASP (grasp.nhlbi.nih.gov/) updated until 18 June 2020. Briefly, We collected the GWAS summary statistics from three major collections: the multi-trait collection (MTC) panel, which is a fixed collection of curations by previous studies (16–18); the UK Biobank (UKBB) panel, which deposited the UKBiobank ‘GWAS round 2’ results preprocessed by Neale's lab (http://www.nealelab.is/uk-biobank) on 1 August 2018

 We conducted a deep literature-mining for human tissue- single-cell transcriptome data. We downloaded the datasets from three major sources, Human Cell Landscape (http://bis.zju.edu.cn/HCL/) (14), Single Cell Expression Atlas (https://www.ebi.ac.uk/gxa/sc/home), and extensive literature curation (9–13,20–22) (Supplementary file S1).


 We used a hierarchical structure to store the cell type in transcriptome data. Under the assumption that the cell types work collaboratively within each tissue, their tissue context should serve as another layer of information. We generated a unique tissue-cell type (TC) id for each of the cell types identified from the focal tissue single-cell data. This structure is displayed on the front page and the Browse function.

  Specifically, we manually queried the ‘tissue name’ + ‘inferred cell type name’ in Cell Ontology (https://www.ebi.ac.uk/ols/ontologies/cl) (24). We recorded the most relevant cell type name and id returned from the Cell Ontology

  Due to the wide-spreading connective tissue cell, including fixed cells (fibrocytes and adipocytes) and ‘wandering cells’ such as leukocytes (25,26), they might not have the cell type-specificity or directly related to the tissue function. Thus, we utilized an ontology-based method to check whether these tissue-cell types (TCs) were annotated as one cell type belongs to the corresponding tissue ontology. Specifically, for each of our 752 TCs, we overlap its Uberon id set described in the previous ‘Cell type ontology’ session with its ‘tissue Uberon id set’. If there is an overlapping of these two Uberon id sets, we would annotate this focal TC to tissue-specific or non-tissue-specific cell type, respectively (Figure 1B)

  The front page of CSEA-DB includes an overview of human 11 organ systems. We used a hierarchical structure to store the tissue single-cell data information.

  Statistics of trait and tissue-cell type association map and vignettes for database functions. (A) Trait and tissue-cell type map statistics; l



  We built a documentation page to briefly describe the data collection, preprocessing, and analysis (https://bioinfo.uth.edu/CSEADB/document.php). We also provide vignettes for website functions, including Browse, Search, Multi-trait comparison and scExpression functions.

  
# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q102205746)  
 * [Wikidata](https://www.wikidata.org/wiki/Q102205746)  
 * [Author Disambiguator](https://author-
disambiguator.toolforge.org/work_item_oauth.php?id=Q102205746&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1093/NAR/GKAA1064)  
