
Cell segmentation-free inference of cell types from in situ transcriptomics data
================================================================================
  
  [@wikidata:Q104450627]  
  
Publication date : 28 of November, 2020  

# Highlights

Here, we present a novel method called Spot-based Spatial cell-type Analysis by Multidimensional mRNA density estimation (SSAM), a robust cell segmentation-free computational framework for identifying cell-types and tissue domains in 2D and 3D. SSAM is applicable to a variety of in situ transcriptomics techniques and capable of integrating prior knowledge of cell type.

Here we introduce a novel computational framework named Spot-based Spatial cell-type 77 Analysis by Multidimensional mRNA density estimation (SSAM). 

We demonstrate 90 that SSAM 1) correctly idsentifies the spatial distribution of known cell types in regions missed in 91 the SSp by cell segmentation based methods for the osmFISH data ; 2) can analyze the POA 92 MERFISH 3D data using the same parameters as for the 2D SSp osmFISH data without any 93 extra adjustments of the settings; 3) identifies new and rare cell types in the VISp, multiplexed 94 smFISH data.

In the second step, SSAM identifies cell-type gene expression signatures by clustering (Fig. 1B). 110 Before running the clustering algorithm.

Next, we continued with completely de novo cell-type identification. The resulting 30 cell-type 164 signatures (Fig. 2A, B, Supplementary Fig. 7-10) were consistent with those identified in the 165 segmentation-based clustering and scRNA-seq based cell-type signatures6 (Supplementary 166 Fig. 4C, D), implicating the robustness of the de novo cell-type calling by SSAM

AM was also able to identify a rare cell type, Sst Chodl, which is known to be related to long-296 range projection and sleep-active neurons33â€“35

SSAM identified a rare cell type and 316 elucidated a suspected spatial heterogeneity of cell types in the cortex without segmenting a 317 single cell. Overall, the results show that SSAM is not only a robust tool to validate 318 segmentation-based methods, but also a reasonable alternative when segmentation is difficult 319 or DAPI or Poly-A images are lacking.

# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q104450627)  
 * [Wikidata](https://www.wikidata.org/wiki/Q104450627)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q104450627/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q104450627&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1101/800748)  
