
Besca, a single-cell transcriptomics analysis toolkit to accelerate translational research | bioRxiv
====================================================================================================
  
  [@wikidata:Q104450593]  
  

# Highlights

We developed Besca, a toolkit that streamlines scRNA-seq analyses according to current best practices. A standard workflow covers quality control, filtering, and clustering.

Besca isan open-source Python library that is compatible withand extendsScanpy[15], one of the most established and up-to-date single-cell analysis toolkits.


ble 1Dataset overview, including hematopoietic cells of peripheral blood and bone marrow (orange), intestine (blue) and pancreas (green) in health and disease

o this end, we connected Besca to the Geneset Management System (GeMS) (https://github.com/bedapub/GeMS). GeMS is a light web-based platform that enables the centralized management of genesets using structured formats and a local application programming interface for geneset information retrieval and organization. The application is built on top of the Flaskmicro-framework (https://flask.palletsprojects.com) using MongoDB(http://www.monogdb.com), an open-source, document-based database as its backend.Once GeMS is deployed, Besca allows the export of gene signatures to the GeMS database (for example a geneset of marker genes from distinct populations) and the retrieval of user-defined signatures(Fig. 1a)

--> Cool repositories, nice documentation.


Cell type identification in scRNA-seq poses great challenges, mainly related tothelack of a biological consensus of what a cell type actually represents and a patchy overview of existing cell types and their identity footprints on the transcriptomic level[29,8]

 To guarantee consistency across studies and communities, beyond scRNA-seq, the proposed cell type annotation schemas are based on the Cell Ontology[40], which is accessible via the Experimental Factor Ontology[41].

 --> Nice! Use of CL within single-cell analysis; - 1.3.1.4. Cell Ontology (and CELDA)

Newly generated cell type annotations in this manuscript provide the most fine-grained annotation as DBlabel .CC-BY-ND 4.0 International licenseavailable under a(which was not certified by peer review) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity. It is made The copyright holder for this preprintthis version posted September 1, 2020. ; https://doi.org/10.1101/2020.08.11.245795doi: bioRxiv preprint 
assignment, which follows the Cell Ontology whenever possible, as well as higher level annotations according to the annotation sheet

Sig-annotis Besca's streamlined version of the manual process of cluster attribution based on marker gene enrichment including ready-to-go annotation schemas for a broad range of cell types, with a particular focus on immune cells. The flexible, multi-level identification schemas are based on a configuration file containing the cell types and their relations as well as the corresponding cell type signatures (see Methods).

--> Maybe BESCA can be fed by - 3.1.3. WikidataMarkers: a website and an R packase for cell type gene sets

efault configuration files for human and mouse are provided, covering a large range of tissues and cell types (human: Supplementary Table S2, mouse: Supplementary Table S3). These files are easily customisable and users are free to provide additional schemas or annotations. The corresponding cell type signatures provided with Besca (Supplementary Table S4) are derived and adapted from various scRNA-seq experiments and publications, with subsequent manual curation

or convenience, we have implemented various functions to guide the annotation based on the Sig-annot framework, and also provide visualisation at individual steps. For instance, one can visualise the relation between the individual cell types as a graph(Fig. 2a), plot the enrichment of individual signatures across all clusters in the dataset as a heatmap (Fig.2b), directly generate annotations at distinct levels in the cell hierarchy and add these in bulk to the AnnData(https://anndata.readthedocs.io) metadata.

Overview of the cell type hierarchy provided with Bescaâ€™s Sig-annotmodule and employed for annotating the datasets in the current manuscript.

eSig-annotcell type attribution at level 1, consisting of major cell types such as T cells and myeloid cells. All detected populations are broadly consistent with the original annotation(g). fSig-annotcell type attribution at level 3, the highest resolution provided in Besca's cell annotation schema. The detected populations are consistent with the original Granja annotation(g), cover T cell subsets with higher granularity and attribute the previously unknown ("14_Unk" and "26_Unk") clusters as well.gOriginal cell type attribution as obtained from Granja et al. Annotated cell populations are highly consistent with clusters obtained from the reanalysis of the original data following the Besca standard workflow.

--> Super cool!


Fig. 3Sig-annotapplied to Kotliarov2020 data containing hematopoietic cells of multiple healthy donors from blood, probed by CITE-seq. aRNA signature-based cell type attribution at level 2, consisting of cell subtypes such as CD4+ T cells and classical monocytes. s

--> Very nice annotation

In sum, the core benefits of adopting Besca for scRNA-seq data analysis are automation, standardization, and reusability. This is achieved (1) by a generalized standard workflow, including CITE-seq data processing, (2) by the automationof the cell type annotation process with two complementary approaches, (3) by managing knowledge about cell type marker gene signatures in GeMS, (4) by informing deconvolution algorithms to make better use of bulk transcriptional data, and (5) by building upon the widely used Scanpy toolkit


# Comments

## Tags
--> Good annotation schemes for - 1.4.2.3. Cell label identification

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q104450593)  
 * [Wikidata](https://www.wikidata.org/wiki/Q104450593)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q104450593/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q104450593&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1101/2020.08.11.245795)  
