
Single cell profiling of immature human postnatal thymocytes resolves the complexity of intra-thymic lineage differentiation and thymus seeding precursors
==========================================================================================================================================================
  
  [@wikidata:Q104450621]  
  
Publication date : 08 of April, 2020  

# Highlights

Here, we employ single-cell RNA sequencing onapproximately 70.000 CD34+ thymocytes to unravel the heterogeneity of the human immature postnatalthymocytes

Besides T cell precursors, we discover branches of intrathymic developingdendritic cells with predominantly plasmacytoid DCs. Trough trajectory inference, we delineate thetranscriptional dynamics underlying early human T-lineage development from which we predicttranscription factor modules that drive stage-specific steps of human T cell development.

Immature human thymocytes are characterized by the human stem/progenitor cell marker CD34.

Multiple thymus seeding populations have been reported based on differential CD7expression and these have led to conflicting reports on whether CD34​+​CD7​-​(Hao et al., 2008; Six etal., 2007) or CD34​+​CD7​int​(Haddad et al., 2006) precursors reflect the thymus-seeding progenitors

 we performed scRNAseqand CITEseq on 70.000 CD34​+thymocytes

 To facilitate the annotation of our CD34​+thymocytes, we integrated our data with the human cell atlas(HCA) bone marrow (BM) dataset that provides a development framework to identify all the variouscell types within our dataset

 To further improve our understanding of the TFs that drive early human T cell development, weimplemented SCENIC​(Aibar et al., 2017) which combines gene correlations with binding motifanalysis within individual cells to infer regulons that correspond with co-expressed genes 

 This studythus clarifies the heterogeneity of early T cell precursors in human and provides clear insights into theprecursor cells and their transcriptional dynamics that drive human T cell development after birth.

 A major strength of our study, to resolve the nature of the thymus seeding precursor, is that wenot only performed scRNAseq on a large number of immature thymocytes, but that we alsoimplemented CITEseq to link the transcriptome with phenotypic properties and used integration withother datasets to demonstrate their migratory potential. Using these approaches, we identified twonon-proliferative populations which are present in the thymus, BM and bloo

. Very few NK cells were detected, and thesewere transcriptionally most similar to the TSP2 subset. We certainly did not see any branch of developing NK cells from ETPs or T-lineage specified cells, suggesting that the residual NK-lineagepotential in these cells results from the similar transcriptional networks that drive their development(De Obaldia and Bhandoola, 2015)​, rather than being an actual event that occurs in the thymus. Whilethe lack of NK cell development is surprising, this finding is however consistent with observations inSCID patients that reconstituted T cells but failed to develop NK cells following HSC transplantation,without adverse effects.



# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q104450621)  
 * [Wikidata](https://www.wikidata.org/wiki/Q104450621)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q104450621/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q104450621&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1101/2020.04.07.007237)  
