
Deep learning and alignment of spatially-resolved whole transcriptomes of single cells in the mouse brain with Tangram | bioRxiv
================================================================================================================================
  
  [@wikidata:Q104450595]  
  

# Highlights

. We devise Tangram, amethod that aligns snRNA-seq data tovarious forms of spatial data collectedfrom the same brain region, including MERFISH1, STARmap2, smFISH3, and Spatial Transcriptomics4(Visium),as well as histological images and public atlases


By doing so, Tangramreconstructsa genome-wide, anatomically-integrated,spatial map  of  the visual  and  somatomotor  area  with  ~30,000 genes  at  single-cell  resolution, revealing  spatial  gene  expression and  chromatin  accessibility patterning  beyond  current limitation of in-situtechnologies

However, deconvolution is hindered by spatial “dropouts”, where cells types defined by sparse or dim markers are not correctly detected27. In many cases, only histological data is directly available for the specimens collected as part of single  cell  atlases,  but those  can  serve  as  a  bridge  to  pre-existing  atlases,  with  measuredin  situhybridization (ISH) data, and rich anatomical annotations in the context of a Common Coordinate Framework, as in the case of the Allen Brain Atl


We developed Tangram, an algorithm that uses sc/snRNA-seqas “puzzle pieces” to align in space to  match  “the  shape”  of  the  spatial  data(Fig.  1a).The  input to  Tangram  is  sc/snRNA-seq  dataalong with spatial data from the same regionor tissue type, from any currently available spatial method(e.g.,  MERFISH,  smFISH,  STARmap,  ISH, or Visium),  requiring  only  that  the two modalities share at least some subset of common genes.Intuitively, Tangram first randomly placesthe sc/snRNA-seq profilesin space, then computes an objective function that mimics the spatial correlation  between  each  gene  in  the  sc/snRNA-seq  dataand  in  the  spatial  data.

--> Good description

Tangram doesnot contain any hyperparameters,mapsa hundred thousand cells in a few minutes (using a single P100 GPU), and is released as PyTorch module.

or deconvolution, we performeda  deterministic  mapping  of  each of  the cellswithin  each 50𝜇𝑚voxel,  to  obtain  cell  type  localization  prediction  at  single-cell  resolution  (Fig.4a). We trained Tangram with a subset of the >30,000 genes, by selecting 1,237 training genes as the union .CC-BY-NC 4.0 International licenseavailable under a(which was not certified by peer review) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity. It is made The copyright holder for this preprintthis version posted September 24, 2020. ; https://doi.org/10.1101/2020.08.29.272831doi: bioRxiv preprint 
12of  the top  100  marker  genes  of  each  cell  type  in the MOp  snRNA-seq  data (usinga  standard pipeline2

Here, we developed and used Tangram to tackle various scenarios, all of which involved aligning snRNA-seq data onto a certain type of spatial data collected from the adult mouse brain cortex, from high resolution MERFISH and STARMap, through mid-resolution Spatial Transcriptomics, and to ISH associated with histological and anatomical coordinates

# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q104450595)  
 * [Wikidata](https://www.wikidata.org/wiki/Q104450595)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q104450595/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q104450595&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
