
Text mining approaches for dealing with the rapidly expanding literature on COVID-19
====================================================================================
  
  [@wikidata:Q104079663]  

# Highlights

More than 50 000 papers have been published about COVID-19 since the beginning of 2020 and several hundred new papers continue to be published every day.

We compile a list of 39 systems that provide functionality such as search, discovery, visualization and summarization over the COVID-19 literature. For each system, we provide a qualitative description and assessment of the system’s performance, unique data or user interface features and modeling decisions.

The process of corpus construction, data enrichment, model development,evaluation and eventual deployment can take months or years, which is unaccept- able during a public health crisis. In the current situation, public corpora help to remove the burden of corpus creation, while shared community annotations contribute to addressing the challenges of data enrichment and annotation

One of the earliest and largest literature corpora created to support COVID-19 text mining is the COVID-19 Open Research Dataset (CORD-19, https://www.semanticscholar.org/cord19) [98], a corpus of metadata and full text of COVID-19 publications and preprints released daily by Semantic Scholar at the Allen Institute for AI. PDFs are converted into machine- readable JSON using the S2ORC pipeline described in [54]and HTML representations of tables in papers are added using IBM Watson Discovery’s Global Table Extractor [115].

LitCovid is a curated set of open access COVID-19 papers
from PubMed [16], containing 52 000 papers at the time of writing and growing. Several text mining systems described in Section on “Text mining systems”use LitCovid as a source of data.

several publishers have compiled and released collections of their COVID-19 literature, such as Elsevier’s Novel Coronavirus Information Center (https://www.elsevier.com/ connect/coronavirus-information-center), Springer Nature’s Coronavirus Research Highlights (https://www.springernature. com/gp/researchers/campaigns/coronavirus), JAMA Network’s COVID-19 Collection (https://jamanetwork.com/journals/jama/ pages/coronavirus-alert) or Science’s COVID-19 Collection (https://www.sciencemag.org/collections/coronavirus).

Fig. 1. A typical workflow for creating a literature text mining system may consist of corpus construction, data enrichment, model development and evaluation

Annotations can be generated automatically (e.g. using pretrained models for named entity recognition and KB entity linking, with toolssuchasMetaMap Lite [23]orScispaCy[59]) or manually through expert annotation (e.g. asking a human to label spans describing population, intervention, comparator and outcome (PICO) elements in clinical trial papers). Several groups have pub- lished reusable annotations, either independently or through annotation sharing platforms such as PubTator(https://www. ncbi.nlm.nih.gov/research/pubtator/) or PubAnnotation (http:// pubannotation.org/). On PubAnnotation, for example, automat- ically generated annotations of terms from several ontologies and PICO elements are available for the CORD-19 and LitCovid corpora.

Also available is CORD-19-on-FHIR (https://github.com/fhircat/CORD-19-on-FHIR), a version of CORD-19 with semantic annotations to clinical entities in the categories of condition, medication and procedure.

--> Nice repo.

The CovidGraph(https:// covidgraph.org/) is perhaps the largest of these, combining literature, case statistics and genomic and molecular data. Another project, the Knowledge Graph Toolkit [36], integrates the CORD-19 corpus with gene, chemical, disease and taxonomic information from Wikidata (https://www.wikidata.org/

Variants of BERT models [24] fine-tuned on COVID-19 literature are available in the form of BioCovidBERT and ClinicalCovidBERT (https://github.com/ manueltonneau/covid-berts). Poerner

Numerous text mining systems for COVID-19 literature have been released in 2020 thus far. We compile a list of 39 systems in Table 2

Exploration-focused systems assist users with discovery and understanding of documents in a corpus. Such systems may not aim to satisfy a specific informational need but are rather used to help users understand the underlying data source; as such, their interfaces facilitate unfocused data exploration and repeated interactions.

Among the systems that leverage KBs, those that use curated domain-specific KBs tend to provide a better user experience, since the entities and relations in these KBs have been vetted by domain experts. IBM Watson’s COVID-19 Navigator (Row S14, https://covid-19-navigator.mybluemix.net/), perhaps the best example of this, allows users to perform boolean queries using UMLS concepts and semantic types [9].

QA systems over COVID-19 literature provide both search and QA function- alities, retrieving relevant documents and surfacing answering spans

Summarization systems aim to provide a condensed version of a longer piece of text. The motivation is to allow readers to derive the main points of a document without expending as much effort in reading or to provide a quick overview of a docu- ment for the reader to decide whether or not to investmore time.
 
Augmented reading systems attempt to improve upon the standard reading experience of papers by providing features such as entity highlighting.

--> Augmented reading is quite interesting.

Augmented reading systems attempt to improve upon the standard reading experience of papers by providing features such as entity highlighting.

Several shared tasks were introduced early during the COVID-19 pandemic to facilitate the evaluation of text min- ing systems. We briefly discuss the Kaggle CORD-19 Research Challenge, the TREC-COVID ad hoc retrieval challenge and the upcoming EPIC-QA challenge, a

Systematic review automation

many of the text mining tasks we discuss previously can be framed in the context of systematic review construction.

Intermediate infrastructure for sharing both automatically and manually produced data annotations, such as PubTator or PubAnnotation, increase the reach of annotation efforts. Annotations shared through these platforms can be reused by many downstream applications.

--> I missed mentions to SciLite.

# Comments
Very comprehensive review, these Brief. Bioinformatics articles are usually quite good.

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q104079663)  
 * [Wikidata](https://www.wikidata.org/wiki/Q104079663)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q104079663/P921%3BP4510)  
