
scConsensus: combining supervised and unsupervised clustering for cell type identification in single-cell RNA sequencing data
=============================================================================================================================
  
  [@wikidata:Q104450617]  
  
Publication date : 24 of April, 2020  

# Highlights


The clustering of single cells for annotation of cell types is a
major step in this analysis. There are two methodologies that
are commonly applied to cluster and annotate cell types: (i)
unsupervised clustering followed by cluster annotation using
marker genes (3) and (ii) supervised approaches that use reference data 
sets to either cluster cells (4) or to classify cells into cell types (5).


Unsupervised clustering methods have been especially useful
for the discovery of novel cell types. However, the markerbased annotation is a burden for researchers as it is a timeconsuming and labour-intensive task. Also, manual, markerbased annotation can be prone to noise and dropout effects. Furthermore, different research groups tend to use different
sets of marker genes to annotate clusters, rendering results to
be less comparable across different laboratories

In summary, despite the obvious importance of cell type identification in scRNA-seq data analysis, the single-cell community has yet to converge on one cell typing methodology (3).

(c) DE genes are computed between all pairs of consensus clusters. Those DE genes are used to re-cluster the data. The refined clusters thus obtained can be annotated with cell type labels.

--> Seems exceedingly complicated
# Comments
Â 1.4.2.3. Cell label identification

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q104450617)  
 * [Wikidata](https://www.wikidata.org/wiki/Q104450617)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q104450617/P921%3BP4510)  
