
Single-cell proteo-genomic reference maps of the hematopoietic system enable the purification and massive profiling of precisely defined cell states
====================================================================================================================================================
  
  [@wikidata:Q106143534]  
  
Publication date : 19 of March, 2021  

# Highlights

Here, we have generated high content single-cell 52 proteo-genomic reference maps of human blood and bone marrow that quantitatively link the expression 53 of up to 197 surface markers to cellular identities and biological processes across all hematopoietic cell 54 types in healthy aging and leukemia.

--> all cell types

the  precision  and  efficiency  of  commonly  used  cytometry  gating  schemes  are  largely 81 unknown, and the exact significance of many surface markers remains unclear. Together, these findings 82 highlight a disconnect between single-cell genomics-based molecular cell type maps and data generated 83 by widely used cytometry assays. 

The classical model of hematopoiesis is mainly based on 87 populations defined by flow cytometry 

For  this  purpose,  we  have  generated 107 proteo-genomic reference atlases encompassing 97-197 surface markers across 95,463 cells representing 108 all major populations of young, aged and leukemic human bone marrow and blood, as well as all states 109 of hematopoietic stem cell differentiation

Abseq experiments, in which mononuclear BM cells from hip 127 aspirates   were   labelled   with   97-197   oligo-tagged   antibodies,   followed   by   targeted   or   whole   128 transcriptome scRNAseq on the Rhapsody platform (Figure 1a)

Following data integration across experiments and measurement modalities (Methods), we identified 45 151 cell types and cell stages covering the vast majority of previously described hematopoietic cell types of 152 the  bone  marrow  and  peripheral  blood

The  Abseq-App  is  accessible  at:166 https://abseqapp.shiny.embl.de/.  

--> Usability is not great.

To quantitatively link surface marker expression with biological processes, we 175 assigned each cell in our data set to its respective cell type, and determined its differentiation stage, its 176 stemness  score,  its  cytotoxicity  score,  its  current  cell  cycle  phase  as  well  as  technical  covariates  (see  177 methods and below)

To validate and demonstrate this approach, we focused on determining novel gating strategies for rare 256 and poorly characterized BM cell types, such as cytotoxic CD4 T cells (Figure 5a) and mesenchymal 257 stem  or  stromal  cells  (MSCs)  (Figure  5g).  Cytotoxic  CD4  T  cells  represent  a  rare  T  cell  population  258 characterized by the expression of cytotoxicity genes typically observed in their well-characterized CD8 259 T cell counterparts (Szabo et al., 2019)

While this cell type has been suggested to be involved in several 260 physiological  and  pathophysiological  processes,  no  coherent  gating  strategy  for  their  prospective 261 isolation  exists (Takeuchi and Saito, 2017). Hypergate suggested cytotoxic CD4 T cells to display an 262 immunophenotype of CD4+CD28-, and differential expression analyses of surface markers revealed that 263 cytotoxic CD4 T cells express significantly lower levels of CD7, CD25, CD127 and CD197 if compared 264 to other CD4 T cell subsets (Figure 5a-d)

--> Hmm going to Wikidata https://www.wikidata.org/wiki/Q106143949

 even the most advanced gating schemes do not recapitulate the molecular and cellular 293 heterogeneity   observed   by   single-cell   genomics   approaches.

 --> Well, of course

 we obtained a gating 300 scheme that uses 12 surface markers to define 16 leaves representing molecularly defined cell states with 301 high precision (Figure 6a-c). The data-derived scheme outperformed a widely used scheme at resolving 302 the  different  cell  states  (Figure  6d; Doulatov  et  al.,  2010),  particularly  in  the  identification  of  lineage  303 committed progenitors, a major shortcoming of many current gating strategies (Notta et al., 2016; Paul 304 et al., 2015; PeriÃ© et al., 2015; Velten et al., 2017). Importantly, cell populations defined by the data-305 defined gating  scheme  were  transcriptionally  more  homogenous,  compared  to  the  literature-based 306 scheme (Figure 6e). 

 --> Cool. Implications for 3.5. Practical applications of a pragmatic cell type definition in 2.4.4

Moreover,  our  gating  scheme  permits  a  faithful  identification  and  317 prospective  isolation  of  molecularly  defined  progenitor  states  in  the  human  hematopoietic  hierarchy  318 using cost-effective flow cytometry.

Our dataset is carefully annotated 367 (Supplementary Note 5) and will serve as a key resource for hematology and immunology. 
--> Let's take a look at this supplementary note, shall we? 

While  we  have  demonstrated  the  implementation  of  data-driven  design  and  analysis  strategies  for  410 cytometry assays in the context of bone marrow, conceptually the approach presented here can be applied 411 to any organ of interest. Thereby, it has the potential to enable the precise isolation and routine profiling 412 of the myriad of cell types discovered by recent single-cell atlas projects

Datasets  including  raw  and  450 integrated gene expression data, cell type annotation, metadata and dimensionality reduction are available as 451 Seurat         v3         objects         through         figshare: https://figshare.com/projects/Single-cell_proteo-452 genomic_reference_maps_of_the_human_hematopoietic_system/94469

--> Noice

Supplementary Note 5: Cell type annotation138139For  cell  type  annotation,  we  used  both  information  from  mRNA  readouts  and  cell  surface  antibody 140readouts. Especially for inter-connected clusters within the hematopoietic differentiation hierarchy and 141their branches along myeloid and lymphoidtrajectories, fine gradients of surface marker expression and 142mRNA  expression  exist,  which  result  in  slight  expression  changes  from  one annotated  cluster  to  the 143next. SupplementaryTable S4 summarizes the surface markers and mRNAs used for annotation of the 14445  clusters.

--> 3.1. Cell-type markers in Wikidata

Supplementary Note 5: Cell type annotation

--> Very detailed annotation! I wonder how much of that can be converted to nanopublication format. 

# Comments

Many mentions of cell types. I wonder if it is worth it to extract them to Wikidata.

Tables with markers (3.1. Cell-type markers in Wikidata)): 
Table S4[supplements/435922_file08.xlsx]
## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q106143534)  
 * [Wikidata](https://www.wikidata.org/wiki/Q106143534)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q106143534/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q106143534&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+workhttps://tabernacle.toolforge.org/?#/tab/manual/Q106143534/P921%3BP4510)  
