
DISCO: a database of Deeply Integrated human Single-Cell Omics data
===================================================================
  
  [@wikidata:Q112637697 ]  
  
Publication date : 01 of January, 2022  

# Highlights

Here, we present DISCO (https://www.immunesinglecell.org/), a database of Deeply Integrated Single-Cell Omics data. The current release of DISCO integrates more than 18 million cells from 4593 samples, covering 107 tissues/cell lines/organoids, 158 diseases, and 20 platforms. We standardized the associated metadata with a controlled vocabulary and ontology system.

<!-- Nice! -->

https://www.immunesinglecell.org/ct_pub

<!-- Neat website, lots to explore/integrate -->

Employing these two tools on the assembled data, we constructed one global atlas and 27 sub-atlases for different tissues, diseases, and cell types.

Having integrated atlases for specific tissues or diseases are highly useful as consensus reference maps and for enhancing downstream analyses. Secondly, the associated metadata is not harmonized, with non-standard formatting and naming conventions. In particular, the cell type labels do not follow any standardized cell type ontology. Thirdly, they provide limited analysis functionalities and visualization capabilities. For example, none of them allow users to project their own data onto the hosted data.

. Leveraging on the large number of public cell type annotation, we developed CELLiD and applied it to annotate the cell types in an automatic and standardized way. 

y. Therefore, we first constructed a cell type specific gene expression reference by separately collecting 84 single-cell datasets that contain detailed cell type annotations. We extracted the original cell type names and manually standardized them for the same cell type. For compatibility with the studies used in atlas construction, we mapped the original cell type names to their closest Cell Ontology types (18). We also constructed a hierarchical tree that illustrates the relationships between cell types.

With this curated cell type reference, we developed CELLiD (CELL type iDentification) to annotate single-cell data at the cluster level (Figure 3A). CELLiD takes in the average gene expression of a cell cluster and performs two rounds of predictions. In the first round (coarse grain stage), it computes the Spearman correlation between each input cell cluster and each reference cell type with all overlapping genes. 

The front end of DISCO is implemented using React (https://reactjs.org/),

. In the UMAP panel, users can color individual cells by cell type, metadata, or gene expression values.

Detailed information on cell types and genes can be found on the cell type browser (https://www.immunesinglecell.org/ct_pub) 

ne. CELLiD (http://www.immunesinglecell.org/cellpredictor) is the same tool employed for annotating the DISCO datasets. From our experience, CELLiD is able to give accurate cell type predictions comparable to manual annotations. Users can upload their own data to CELLiD and export the cell type predictions in a tabular format. The second tool is CellMapper (http://www.immunesinglecell.org/ct_mapper) which enables projection of uploaded data onto the constructed atlases.
# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q112637697)  
 * [Wikidata](https://www.wikidata.org/wiki/Q112637697)  
 * [Author Disambiguator](https://author-
disambiguator.toolforge.org/work_item_oauth.php?id=Q112637697&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1093/NAR/GKAB1020)  
