
Polygenic enrichment distinguishes disease associations of individual cells in single-cell RNA-seq data
=======================================================================================================
  
  [@wikidata:Q116591826]  
  
Publication date : 01 of September, 2022  

# Highlights

We introduce single-cell disease relevance score (scDRS), an approach that links scRNA-seq with polygenic disease risk at single-cell resolution, independent of annotated cell types. scDRS identifies cells exhibiting excess expression across disease-associated genes implicated by genome-wide association studies (GWASs)

We applied scDRS to 74 diseases/traits and 1.3 million single-cell gene-expression profiles across 31 tissues/organs. Cell-type-level results broadly recapitulated known cell-type–disease associations. 


Previous studies integrating scRNA-seq with GWAS have largely focused on predefined cell-type annotations (for example, classical cell types defined using known marker genes), aggregating cells from the same cell type followed by evaluating overlap of the cell-type-level information with GWAS


Integrating information from scRNA-seq data at fine-grained resolution (for example, individual cells both within and across cell types) with polygenic signals from disease GWAS has considerable potential to produce new biological insights.

We applied scDRS to 74 diseases/traits (average GWAS N = 346K where “K” means thousands) and 16 scRNA-seq data sets (including the Tabula Muris Senis (TMS) mouse cell atlas17), assessing cell-type–disease associations and within-cell-type association heterogeneity, including heterogeneity of T cells in association with autoimmune diseases, neurons in association with brain-related diseases/traits and hepatocytes in association with metabolic traits

We analyzed GWAS data from 74 diseases/traits (average N = 346K; Supplementary Tables 1 and 11) in conjunction with the TMS FACS data with 120 cell types (cells from different tissues were combined for a given cell type; Supplementary Table 5). We first report scDRS cell-type-level results, aggregated for each cell type from the scDRS individual-cell-level results


irst, granulocyte monocyte progenitors were strongly associated with multiple sclerosis (MS), highlighting the role of myeloid cells in MS29. Second, oligodendrocytes and oligodendrocyte precursor cells (OPCs) were associated with multiple brain-related diseases/traits. These associations are less clear in existing genetic studies4,6,22,30 but are biologically plausible, consistent with the increasingly discussed role of oligodendrocyte lineage cells in brain diseases/traits

. In addition, the subpopulations of IBD-associated cells in clusters 4, 5 and 9 were labeled as “Th2/Treg-like”, “Th17-like” and “CD8+ effector-like”

We have introduced scDRS, a method that leverages polygenic GWAS signals to associate individual cells in scRNA-seq data with diseases and complex traits

Third, although we have primarily focused on the associations involving a single disease/trait, further investigation of differences between diseases/traits within the same category is an important future direction. Additional limitations are discussed in the Supplementary Note. Despite all these limitations, scDRS is a powerful method for distinguishing disease associations of individual cells in scRNA-seq data.

# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q116591826)  
 * [Wikidata](https://www.wikidata.org/wiki/Q116591826)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q116591826&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1038/S41588-022-01167-Z)  
