
A reference cell tree will serve science better than a reference cell atlas
===========================================================================
  
  [@wikidata:Q117229478]  
  
Publication date : 01 of March, 2023  

# Highlights

 In this perspective, we argue against an atlas or periodic table-like discretization as the right metaphor for a reference taxonomy of cell types. In its place, we advocate for a data-driven, tree-based nomenclature that is rooted in a “consensus ontogeny” spanning the life cycle of a given species. 

 <!-- You cannot argue that a metaphor is _right_ anyways. Also, you cannot divide the current views of cell diversity in a single-hierarchy (e.g. tree). Advocating for a tree-based nomenclature is negating the work of the Cell Ontology, and its strategy for organizing cell types. Though the Cell Ontology is not the sole source of truth, it is a very serious 19-year-old project. 
 
 For example B-cell can be understood both as a lymphocyte and as an antigen-presenting cell. It is a child of 2 parent nodes. In a tree, all nodes have exactly one parent node (https://en.wikipedia.org/wiki/Tree_(data_structure)). -->


 Analogous to the taxonomic classification of species, a consensus ontogeny would provide a universal, stable, and extendable framework for precise scientific communication, both contemporaneously and across the ages.

 <!-- Though a "consensus ontogeny" sounds interesting, it is not fundamented in a core biological principle. In the end, if you have 206 bone types, you have 206 developmental histories. But a classification of cells that creates a 206 osteocyte classes won't be wrong, but is probably too bloated for the practical purposes of undertanding physiology
 
Also, in "Cell types as species: exploring a metaphor" (Doyle, 2022), the author cites: 


" Slater (2013) noted that “Unlike species, cells do not fit into a single phylogenetic tree. Rather, development in each organism defines its own local tree,” but although this may cause philosophical concerns, the problem is minimized if, as here, the focus is on the cells of an individual in a single species. Rodieck and Brening (1983) suggested that “It might appear that the lineage of ganglion-cell types, expressed during development, would provide a causal agent as useful as the role genealogy plays in the classification of species.” However, they noted that despite the strong superficial resemblance between lineage trees of cells and organismal phylogenies, there are fundamental differences (Rodieck and Brening, 1983). With species, a lineage (however, defined) can be related ultimately to mutational variation at the genomic level. In contrast, all cells of an individual organism possess fundamentally the same genome (ignoring somatic mutation and phenomena such as endopolyploidy), and the process of differentiation that is analogous to speciation occurs through epigenetic mechanisms (Figure 1). The immediate ancestor of a species cannot produce a species that belongs, genetically, to a different clade, but a multipotent stem cell is theoretically capable of producing any mature cell type. The connection between phenotype (the mature cell’s transcriptome, morphology, etc.) and epigenotype for cells is thus looser than a whole organism’s phenotype to genotype relationship. Rodieck and Brening (1983) concluded that “if there proves to be little or no correspondence between the lineage map and its phenotypic expression, then it is unlikely that the lineage map will become a useful and widely accepted ordering of ganglion-cell types.”"

(...)
"The potential for discordance between phenotype (expression manifold) and cell mitotic lineage is what led both Zeng and Sanes (2017) and Arendt et al. (2016, 2019) to reject using cell lineage for defining mature cell types. For example, in blood development (hematopoiesis), different cell lineages can produce functionally similar cells (Yáñez et al., 2022)"
.  -->


 In the preface to The Order of Things,1 the French philosopher Michel Foucault illustrates the limitations of taxonomic classifications by quoting a fictional “certain Chinese encyclopedia” that states “animals are divided into: (a) belonging to the Emperor, (b) embalmed, (c) tame, (d) sucking pigs, (e) sirens, (f) fabulous, (g) stray dogs, (h) included in the present classification, (i) frenzied, (j) innumerable, (k) drawn with a very fine camel hair brush, (l) et cetera, (m) having just broken the water pitcher, (n) that from a long way off look like flies.” Foucault’s point in citing this absurd taxonomy is that we often assume that an accepted classification scheme represents an objective reality, but there are infinite alternatives. This matters, he argues, because our systems of classification both reflect and direct our thinking: how we apply labels, and how we relate those labels to one another, shapes our conception of the underlying phenomena.


 <!-- It is a great citation and relevant comment. The authors, however, should have acknowledge that the conception of this "Chinese encyclopedia" is actually from the argentinian writer Jorge Luis Borges. The Wikipedia page for the Celestial Emporium of Benevolent Knowledge discusses both Borges' creation and its influence on Foucalt's work: https://en.wikipedia.org/wiki/Celestial_Emporium_of_Benevolent_Knowledge -->


 For example, the European honey bee might be known as “Apis pubescens, thorace subgriseo, abdomine fusco, pedibus posticis glabris utrinque margine ciliatis,” which translates as “furry bee, grayish thorax, brownish abdomen, black legs smooth with hair on both sides”.3 These unwieldy descriptors, lacking a shared organizational framework or information on how classes relate to one another, bring to mind the contemporary practice of naming cell types by combinations of differentially expressed marker genes.


However, although there are excellent algorithms for organizing single-cell profiles into manageable numbers of “clusters”, researchers are largely left to their own devices with respect to naming clusters and relating them to other datasets.


<!-- True, good comment -->


This is unfortunate for several reasons. 

First, it leads to considerable repetition of effort, often in the form of time-intensive literature and web searches (e.g., the unsystematic practice of “googling” differentially expressed genes).

Second, it is the wild west out there, with no widely accepted standards around annotation quality or nomenclature. Although we are increasingly adept at integrating datasets and transferring labels computationally, this risks simply propagating potentially suboptimal descriptors. 
 
 
 Third, the resulting corpus is heavily biased toward the systems in which the data is being generated (a complex function of scientific interest, resource allocation, and technical factors), rather than being anchored in a natural distribution. 
 
 <!-- This is a very good point. That is also the case for the Cell Ontology and its description of cell types -->

Fourth, it represents a missed opportunity, as it doesn’t feel like we are moving toward any consensus or cohesion that mirrors Linnaeus’s index cards, where new information can simply be added to a stable backbone.

<!-- That is partially true, but partially false: new cell types can be added to the Cell Ontology, though it is underused -->


For example, cell “type”, “state”, and “identity” are often used interchangeably, as are cell “lineage” and “trajectory”.8 How we define these terms in the context of this perspective is summarized in Box 1.

Box 1
Terminology

Cell type: A recurring pattern of developmental origin and potential within and across cell lineage trees of individuals of a given species, generally reflected in shared molecular properties.

Cell state: Variations in molecular phenotypes within a cell type that do not impact its developmental potential (e.g., cell cycle, stochastic fluctuations).

Cell identity: An individual cell as characterized solely by its molecular phenotypes at a given moment in time.

Cell lineage: The relationships among cells of an individual organism as defined solely by the series of cell divisions that begins with a single zygote.

Cell trajectory: Ordering of cells’ developmental relationships inferred solely from similarity in molecular phenotypes, which might or might not recapitulate developmental cell lineage relationships.


<!-- Interesting definitions, though they seem to me a bit artificial. The definition of a cell type as a _recurring pattern_ and not as a class of individuals is a bit weird, and not very precise.  It also provides a definition for a single species, ignoring all notions of "cell type" that are plurispecific. The idea of "neuron" is not explained, just the idea of "human neuron". 

The notion of cell identity is good, though it is weird that the cell _is_ the identity -->

In our opinion, we should be pushing for a cell type nomenclature that meets some of the same key criteria as Linnaean taxonomy, as well as additional ones, including:
 (1) accommodating all cells arising during the life cycle of a given organism; 
 (2) accommodating inter-individual variation, both normal and disease-related; 
 (3) relating cell types to one another in a biologically meaningful way; 
 (4) being stable to the incorporation of new data or new data types; and 
 (5) being constructed in a largely, if not entirely, data-driven manner. 

<!-- It is very good that the authors are being explicit, that is brave. That opens up space for disagreement,though. 

First, two things are being mixed: the taxonomy and the naming system. Biocurators agree on criteria for good identifiers (https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2001414), and these are enough for computational systems. Of course, generating good names that humans can use is a different, related question.  

We definitely cannot host technically a taxonomy with 37 trillion terms (the number of adult human cells). I understand, though, that the authors mean "cell types", but ended up saying "cells". Maybe by "accomodating" they mean somehow representing the diversity, in which case, I agree.

Now, I have to disagree with the criterion 05. It should not be entirely data-driven. Any data dimension will miss others. Data should help us, but a final list of cell types will always depend on what cell types are useful for our theoretical models of reality.  -->

This mixing of organizing principles is also an explicit choice of the most serious effort to date to create a universal nomenclature. Cell Ontology is a curated resource, analogous to Gene Ontology, with over 2,000 cell type classes. No single organizing principle is prioritized, but rather the classes correspond to different cellular properties (e.g., functional, histological) or broad “lineage” classes (e.g., “blood cells”).9 In our view, we should be more consistent, choosing a “primary” principle, even if others are given secondary consideration. Given the known discrepancies between different organizing principles (e.g., two cells may have similar molecular profiles or functions but be developmentally unrelated), this is necessary to end up at an unambiguous, universal framework.

<!-- The Cell Ontology (CL) has evolved _a lot_ in the 18 years since the publication of the original article they cite. The fact that they don't cite other Cell Ontology papers and ignore the Cell Ontology's preferred citation, a 2016 article (see https://cell-ontology.github.io/index.html#cite) is a bit sad. The CL indeed not try and fix a "single organizing principles" but the "cell_by_histology", "cell_by_function", "cell_by_lineage" and similar organizer terms have long been obsoleted. 
Our primary principle in CL is evidence for real-world instances and relevance of the class for researchers and biocurators when representing their data. That is why https://cellxgene.cziscience.com/ and so many other resources use CL: because it is built to fit the needs of data organization.
-->

What is the “right” choice for this principle? Let us consider the more obvious options in turn: historical, morphological, functional, evolutionary, molecular, and developmental (Figure 1). We immediately reject historical footnotes, as there is no practical utility to naming cell types after specific researchers. 

<!-- Of course there is: These terms have been studied and learned by thousands, if not millions, of researchers. -->


An ideal principle would: arrange cells in a biologically meaningful way; be stable to the incorporation of new data; accommodate all cells arising during the organism’s natural life cycle; and be fully data-driven.

<!-- It is not clear to me why it should be so -->

This definition relies on the concept of an evolutionarily stable “core regulatory complex” (CoRC), composed of key transcription factors (TFs), miRNAs, RNA binding proteins, etc. Although undoubtedly useful for certain goals, we argue against an evolutionary perspective for our purposes.

First, identifying orthologous cell types across vast evolutionary distances is considerably more challenging for cell types than for gene sequences. Second, the CoRC definition focuses on terminally differentiated cell types, deemphasizing the way(s) that each cell type came to be within a developing organism.

 Third, the use of CoRCs as the distinguishing feature of each cell type is diffuse, as it is subjective which factors would be included or what level of difference would delineate CoRCs from one another. Fourth, the overwhelming majority of biomedical research is focused on humans and a few model organisms. Giving primacy to evolution comes at the expense of precision in defining relationships among cell types within this handful of intensely studied species.

<!-- Well, but the same is for identifying orthologous lineages, isn't it?  As your purpose is already limited to single species (by the definition), so the first objection doesn't apply.
The third is only partially true: the Arendt's CoRC theory is assuming that there exist a set of biologically meaningful stable CoRCs that evolve in each species. If they exist, the definition could be completely data driven.-->

As scRNA-seq has grown in use, the field increasingly relies on this approach to organize cells into “periodic tables” or hierarchically clustered dendrograms.12,13 Although laborious googling of marker genes is gradually being displaced by automated annotation via label transfer,14 these methods tend to rely on predefined marker genes or labeled reference datasets as input, which themselves do not follow any systematic nomenclature.

<!-- The "Periodic Table of Cell Types" article has acquired a lot of visibility using this flashy metaphor. However, no one is using this framework for organizing cell types in practice. The Cell Ontology, on the other hand, is widely used for annotation (e.g. in CELL x GENE), inside annotation algorithms (https://www.nature.com/articles/s41467-021-25725-x) and in data-driven applications that combine existing structure with cell type inference from single-cell transcriptomics data -->

A first concern is it is not necessarily stable to the incorporation of new data, be it from cancerous tissue, or a different developmental stage, or even from the same sample, as clustering is sensitive to the technology used, batch effects, depth of profiling, etc.

<!-- This is a good concern. Clusters are not cell types, and shouldn't be regarded as so. Recent theoretical work has mixed them a bit, but differentiated cell sets (the clusters themselves) from the provisional cell types ("Quantitatively derived data cluster defined within a taxonomy"). See https://elifesciences.org/articles/59928 for some of these considerations. -->


A second concern is that the discretization of cell types fails to incorporate continuous forms of molecular heterogeneity, e.g., spatial or temporal gradients in gene expression. This issue—of being unable to see a clear boundary between cell types and needing to rely on experts who might disagree on where to put the threshold—is an acknowledged challenge for automated label transfer.

<!-- This is the same as saying that the discretization of organisms intro species fails to incorporate  continuous forms of genetic heterogeneity. It is true, but it doesn't mean that discretizing individuals into species is not useful. The boundary problem is also present in species contexts, see the "Lineage separation and divergence (speciation) and species concepts" figure on https://academic.oup.com/sysbio/article/56/6/879/1653163-->

Finally, we come to developmental relationships. Lineage represents the ground truth of a cell’s history and future—where it comes from and what it will give rise to: clear, quantifiable relationships between cells as they arose within the context of one individual. The paradigmatic example is Caenorhabditis elegans, where an invariant lineage, painstakingly documented by visual observation, allows for each and every cell in this organism’s life history to be precisely named

<!-- That is coloring it a bit too much.  It may be a good framework for eutelic organisms (the ones with a fixed number of cells): https://en.wikipedia.org/wiki/Eutely Even them, however, there might be 2 cells with different developmental history, but a strong transcriptomic similarity. They will have 2 different names (because it is relevant for research), but for transcriptomicists, there will be no point in splitting them. -->

C. elegans also illustrates how purely lineage-based descriptors might obscure the functional homologies among cells with disparate lineage histories. For example, IL1 and IL2 neurons derive from different founder cells, yet are practically indistinguishable transcriptionally; the same is true of subsets of muscle cells

<!-- Oh, that is considered and exemplified, oops. Good!  -->

As an alternative, we and others have developed methods that explicitly record cell lineage via clonal tagging19 or evolving barcodes

<!--  Oh, there is a clear source of bias there. Not a problem, just a feature to notice.  -->

 As we would remain blind to the identities of inferred ancestors (not to mention “extinct” lineages, e.g., due to apoptosis), it is unclear how such a tree alone would yield a satisfactory cell type nomenclature. Finally, naming cells by each cell division is not necessarily useful for organisms consisting of trillions of cells, especially if those divisions do not even play out in an invariant manner.


<!-- Yes. This limitaion is not very clear from the abstract. -->

Two maturing technical paradigms are relevant here. The first involves “whole organism” scRNA-seq profiling of multiple developmental stages of model organisms, coupled to computational inference of transcriptional trajectories.21 Such studies can yield “pseudo-trees” encompassing the development of large, opaque model organisms, including fly, frog, zebrafish, and mouse.22,23,24,25 Although these pseudo-trees will undoubtedly continue to improve in resolution and completeness, they remain inherently inferential in terms of the cell identities l

<!-- Interesting context and opinion -->

Imagine that a series of progressively older embryos is subjected to a flavor of molecular recording that yields a comprehensive lineage tree, with rich molecular states for terminal nodes (e.g., scRNA-seq) but also information about the molecular identities of inferred ancestors. 

Such trees could be merged across individuals to yield a molecularly annotated consensus ontogeny of a given species (Figure 2).


Assuming it can be constructed, a molecularly annotated consensus ontogeny would provide an excellent framework for cell type classification

<!-- Well, maybe? It could be good for developmental biology researchers, but I echo my previous comment: do we really need to conceptualize 206 different types of osteocytes? -->

Second, both variation within (e.g., due to stochastic factors, genetic variation) or deviation from (due to disease) the consensus could be represented by “summary statistics” of individual branches or by alternative branches, respectively. 

<!-- The image is relatively clear, but the implementation is not; it does not "mirrors Linnaeus’s index cards". The Cell Ontology is a technical solution. The authors could have discussed how to implement their proposal in a computational ontology, or some other database structure or computational model. -->

Third, in contrast to current atlases which are biased toward specific systems by an amalgam of factors, a consensus ontogeny spanning the life cycle would include all “normal” cell states and relate them to one another. 

<!-- An ontogeny might cover all states regarding  -->

How would a consensus ontogeny be structured and represented? We recognize that the practical aspects of the proposed concept are likely still vague to most readers. Below we conceptualize a potential structure for a consensus ontogeny, address the integration of individuals and data types, and suggest a data-driven cell type nomenclature. We then provide a practical example of how we envision experimentally deriving and visually representing such a tree. Finally, we address outstanding challenges for the proposed framework, in particular for H. sapiens, and possible solutions.

<!-- Oh, ok, visual representations. Still not a computational scaffold, but it is something.  -->

Figure 3Parallels between modeling demographic and cell lineage histories

<!-- It is written "retinal projenitor cell", some mix of "projection" and "progenitor".  -->
# Comments 

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q117229478)  
 * [Wikidata](https://www.wikidata.org/wiki/Q117229478)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q117229478&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1016/J.CELL.2023.02.016)  
