
Significance analysis for clustering with single-cell RNA-sequencing data 
==========
  
  [@wikidata:Q121789756]  
  

# Highlights

However, the most widely used clustering algorithms are heuristic and do not formally account for statistical uncertainty. 

Here we extend a previous method, significance of hierarchical clustering, to propose a model-based hypothesis testing approach that incorporates significance analysis into the clustering algorithm and permits statistical evaluation of clusters as distinct cell populations

To show practical utility, we applied our approach to the Human Lung Cell Atlas and an atlas of the mouse cerebellar cortex, identifying several cases of over-clustering and recapitulating experimentally validated cell type definitions.

As an example, consider the Louvain and Leiden algorithms1 as implemented by the widely used Seurat toolkit2. A standard procedure is to (1) apply principal component analysis to the log-transformed and normalized counts, (2) compute the Euclidean distance between the first 30 principal components of each pair of cells, (3) find the 20 nearest neighbors for each cell, (4) specify a weight for each pair of cells based on the number of neighbors in common and use this to define weighted edges in a network, and (5) divide the network into clusters that maximize modularity

<!-- Good summary of the process -->

. As a result, if one does not account for this statistical reality, over-clustered output can appear to show convincing differences.





# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q121789756)  
 * [Wikidata](https://www.wikidata.org/wiki/Q121789756)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q121789756&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  

# Reading dates
  
 * 2023-08-23
