
Integration of electrophysiological recordings with single-cell RNA-seq data identifies neuronal subtypes
=========================================================================================================
  
  [@wikidata:Q31033187]  

# Highlights

Here, we present a method for obtaining full transcriptome data from single neocortical pyramidal cells and interneurons after whole-cell patch-clamp recordings in mouse brain slices

We show that Patch- seq reveals a close link between electrophysiological characteristics, responses to acute chemical challenges and RNA expression of neurotransmitter receptors and channels.

Through acute pharmacological probing of cortical interneurons, we established causality between RNA- seq–based predictions and experimentally observed neuronal responses ex vivo. 

--> Nice.

We focused on cholecystokinin (CCK)-containing(+) GABAergic interneurons (Fig. 1) because their morphological and molecular features are thought to form a quasi-continuum from axon-target- ing to dendrite-targeting interneurons in cortical areas.

As reliable histochemical detection of CCK+ interneurons is particularly challenging, we established a dual-labeled CCKBAC/dsRed::GAD67gfp/+ mouse reporter23,24 (Fig. 1a and Supplementary Fig. 1a), and sampled dual DsRed+/GFP+ interneurons in layers (L)1/2 of the somatosensory cortex.

Forty-five were interneurons (inhibitory types, “I-types”; Fig. 1b–f) and DsRed+/GFP+ with an additional 38 being pyramidal cells (excitatory types, “Exc-types”; Supplementary Fig. 1b–d)

Here, we first used patch-clamp electro- physiology to classify DsRed+/GFP+ interneurons in cortical L1 into five subclasses purely based on their electrophysiological properties.

L2/3 and L4 pyramidal cells (DsRed only) were categorized as nonbursting, accommodating and regular spiking.

After testing the neurons in a series of current-clamp and voltage-clamp protocols within 20–25 min, their entire somatic compartment was aspirated into the recording pipette.

To validate the quality of our RNA-seq data, we compared the results
from inhibitory and excitatory neurons. As expected, I-type neurons expressed Gad1 (the gene encoding GAD67; 41 of 45 cells) and Gad2 (the gene encoding GAD65; 39 of 45 cells), as well as Cck (44 of 45 cells).

None of the cells (0/45) Cell contained parvalbumin (Pvalb) mRNA transcripts. By contrast, Exc-type pyramidal cells only expressed Slc17a7, and to a lesser extent Slc17a6, but not Slc32a1 (0 of 38 cells), along with considerably lower copy numbers of Cck mRNA (Fig. 3e). Thus, our RNA-seq data accurately reflect the major functional distinction between inhibitory and excitatory neurons in the cer- ebral cortex


The quantitative expression of 24 genes
(Clic4, Clip3, Cacna1g, Kcnma1, Kcnj11, Kcnc1, Apba2, Cacna1g,
Cadps2, Exoc8, Gria1, Grin2b, Htr7, Kcnma1, Npy, Pak1, Pcdh8,
Slc32a1, Slc6a17, Sort1, Stx4a, Syt6, Syt7, Tac2) had significant corre-
lation with at least one electrophysiology parameter (

Cell type–specific quantitative expression of ion channel and receptor genes in CCK+ interneurons and pyramidal cells.

--> Interesting that the _definition_ is genetic, all other parameters are secondary classifications or just descriptions.

Many developmental biology studies utilize the serotonin (5-HT)
receptor 3a (Htr3a) subtype to detect and classify CCK interneurons4,41. In our Patch-seq data set, we observed no 5-HT3a expression in I-type 1 cells (Fig. 6a,c), low Htr3a mRNA expression in I-type 2 (Calb+/Npy+/Vip−) cells and high mRNA copy numbers in I-type 3 (Vip+/Npy−) and I-type 4 (Vip+/Npy+, Fig. 6b,c) and I-type 5 (Vip−/Npy+) interneurons. This suggests that Htr3a expression among CCK+ interneurons might be more restricted than initially thought, likely leading to the undersampling of CCK interneurons by Htr3a-driven genetic tags41

--> Another strong case for granular technotypes to avoid confusion.


e further showed that Patch-seq can identify a select number of molecular determinants that can be used to further subdivide CCK+ interneuron subclasses. Each of these examples is important because theyreconcile previously reported sets of data on receptors (5Ht3a, Cnr1) and channels (Hcn1) (refs. 3,6,7,39–41). We note that Patch-seq can discover sets of cellular markers independent of other classification systems or a priori knowledge of the cell type of interest.

--> Well, you did use genetically tagged lines (but I see that they are not strictly necessary).
# Comments

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q31033187)  
 * [Wikidata](https://www.wikidata.org/wiki/Q31033187)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q31033187/P921%3BP4510)  
