
Ontology-based annotations and semantic relations in large-scale (epi)genomics data
===================================================================================
  
  [@wikidata:Q31090008]  
  
Publication date : 03 of May, 2016  

# Highlights

In this study, we illustrate the semantic annotation of Gene Expression Omnibus samples metadata using concepts from biomedical ontologies, focusing on the association of thousands of chromatin immunoprecipitation followed by high-throughput sequencing (ChIP-seq) samples with a given target, tissue and disease state

We compared tools based on Unified Medical Language System with tools that use topic-specific ontologies, showing that the second approach outperforms the first both in the annotation process and in the computation of semantic similarity measures.

Crucial for this expansion process are (i) the ability of assigning formal semantic labels to public samples (semantic annotation; for example, determining the targeted protein, the investigated disease state and tissue type), and (ii) the possibility of relating different samples through these labels (semantic similarity), to identify semantically coherent experiments that are suitable to be associated to the initial data set.

 In particular, one of the main tools is the National Center for Biomedical Ontology (NCBO) Bioportal, which provides the ability to browse, search, visualize, annotate (using the NCBO annotator) and map text with concepts from more than 300 different ontologies [13, 14].

 In this article, we start by providing a brief description of the state-of-the art concepts and tools for recognizing and relating ontology concepts in biomedical texts. 

In this study, we considered, and compared, two main resources to perform the semantic annotation: UMLS and specific OBO ontologies
On the other hand, while ontologies belonging to the OBO foundry are also created by experts in the field and reflect human reasoning, and while they covered a more limited set of research fields, they have important advantages in terms of orthogonality, unambiguousity, scalability and topic specificity [10]. OBO ontologies considered within this study are the BRENDA Tissue Ontology (BTO) [18], which collects a hierarchy of tissues, cell types and cell lines for different organisms, and the Disease Ontology (DO) [19], which describes human diseases and phenotypes.

Metamap [20] is a Natural Language Processing (NLP) tool developed with the aim of annotating biomedical text with UMLS Metathesaurus concepts

Conceptmapper [21] is a Unstructured Information Management Architecture-based tool to annotate text with concepts from a dictionary. BO and DO OBO ontologies were converted into Conceptmapper dictionaries using the components developed at Colorado Computational Pharmacology and available at http://sourceforge.net/projects/bionlp-uima/.

In the case of GEO metadata, knowledge-based methods are expected to perform better than distributional-based ones, as the description of each sample is short and the frequency of each concept in the corpus is always low.

Surprisingly, while several thousands of experiments are already available for these regulatory factors in public databases, and while studies involving TFs and epigenetic marks would greatly benefit from this abundant and comprehensive information, the semantic information associated to this rich data resource is rarely exploited.

For example, the concept ‘MCF7-cells’ (human breast cancer cell line) always appears closer to the concept ‘K-562’ (myelogenous leukemia cell), than to the concept ‘Breast cancer cell’. Indeed, both ‘MCF7-cells’ and ‘K-562’ map to the ‘cell’ semantic type, while ‘Breast cancer cell’ belongs to the ‘bpoc’ semantic type. This would be problematic 

Future work might be dedicated to the development of user-friendly software applying these concepts, with emphasis on the field of epigenetics and transcriptional regulation, and possibly incorporating a database of semantically annotated GEO metadata.

--> I was expecting to see a database, like MetaSRA
# Comments

## Tags
- 3.12 Linked GEO and curation of transcriptomics datasets

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q31090008)  
 * [Wikidata](https://www.wikidata.org/wiki/Q31090008)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q31090008/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q31090008&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1093/BIB/BBW036)  
