
Variation in olfactory neuron repertoires is genetically controlled and environmentally modulated.
==================================================================================================
  
  [@wikidata:Q33603006]  
  
Publication date : 25 of April, 2017  

# Highlights

The mouse olfactory sensory neuron (OSN) repertoire is composed of 10 million cells and each expresses one olfactory receptor (OR) gene from a pool of over 1000. Thus, the nose is sub-stratified into more than a thousand OSN subtypes. 

We find that the OSN subtype distribution is stereotyped in genetically identical mice, but varies extensively between different strains. 

We find that the OSN subtype distribution is stereotyped in genetically identical mice, but varies extensively between different strains. Further, we identify cis-acting genetic variation as the greatest component influencing OSN composition and demonstrate independence from OR function.

 While this has been achieved in a simple invertebrate model containing hundreds of neurons (White et al., 1986), applying the same approach to mammalian brains that encompass many millions of neurons represents a significant challenge (Wichterle et al., 2013).

he mouse genome codes for over a thousand functional OR genes, but each mature OSN expresses only one abundantly, in a monoallelic fashion (Hanchate et al., 2015; Saraiva et al., 2016; Tan et al., 2015; Chess et al., 1994). 

It is also possible that relative OSN abundances could be altered by regulating the lifespan of each OSN subtype. Indeed activation extends a sensory neuronâ€™s life-span (Santoro and Dulac, 2012), suggesting that persistent exposure to particular odorants may, over time, increase the relative proportions of the OSNs responsive to them. 

We report that, while the repertoire of OSN subtypes is stable across individuals from the same strain, it reproducibly and extensively differs between genetically divergent strains of laboratory mice.

We next collected 63 single mature OSNs from WOM, and determined the OR gene most abundantly expressed in each using a single-cell RNAseq approach (Saraiva et al., 2016, unpublished data). 

One hundred and thirty-six of these are DE in all three pairwise comparisons (Figure 2G); for example, there are consistently different numbers of Olfr6-expressing OSNs in each strain (Figure 2H).

Genetic background instructs OSN diversity independent of odor environment


In a controlled environment, the influence of odortype on the development and maintenance of the MOE is minimal, perhaps restricted to only a few OSN subtypes.



# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q33603006)  
 * [Wikidata](https://www.wikidata.org/wiki/Q33603006)  
 * [Author Disambiguator](https://author-
disambiguator.toolforge.org/work_item_oauth.php?id=Q33603006&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.7554/ELIFE.21476)  
