
Reconstructing lineage hierarchies of the distal lung epithelium using single-cell RNA-seq
==========================================================================================
  
  [@wikidata:Q34094294]  

# Highlights
mechanisms that control the progression of lung progenitors along distinct lineages into mature alveolar cell types are still incompletely known

We empirically classified cells into distinct groups by using an unbiased genome-wide approach that did not require a priori knowledge of the underlying cell types or the previous puri- fication ofcell populations. The results confirmedthe basic outlines of the classical model of epithelial cell-type diversity in the distal lung and led to the discovery ofmany previously unknown cell-type markers

Using known marker genes within the different clusters, we were able to associate cells with four previ- ously reported epithelial cell types (Clara (Scgb1a1), ciliated (Foxj1), AT1 (Pdpn, Ager) and AT2 (Sftpc, Sftpb) cells)

Together, these results showthat single-cell RNA-seq enables the identification and molecular characterization of cell types and developmental intermediates retrospectively without the need to first purify populations of interest.

Among the numerous newly identified putative cell-type markers, several are of particular interest.

--> I find the idea of "putative cell markers" quite interesting. How can you confirm that some really is a bona-fide marker, and not only a putative one?? (My guess: you can't)

weobserved cells in intermediate maturation stages on the basis ofpartial co-expression ofAT1andAT2 marker genes. We used the newly identified genes specific for each mature alveolar cell type to subclassify these intermediates and thereby reconstruct the molecular pathway of differentiation ofBPs into AT1 andAT2 lineages, grouping the genes into earlyand late markers ofeither lineage 

--> "Maturation stages" is similar to "cell states" or "intermediary cell types"

c, Violin plots depicting the course ofexpression ofeach ofseven distinct gene groups across the six cell populations. Each violin plot shows the frequency distribution ofthe mean transcript level (log2-transformed FPKM) ofall genes per cell


We expect that a similar strategy to that pursued here can be applied to almost any tissue to empirically classify and characterize the full set ofdeveloping and mature cell types, explain the molecular regulation of these distinct populations, and explorehowtheyare disrupted in disease.


# Comments
Grr, the use of "Clara cells"!


# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q34094294)  
 * [Wikidata](https://www.wikidata.org/wiki/Q34094294)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q34094294/P921%3BP4510)  
