
Perturb-Seq: Dissecting Molecular Circuits with Scalable Single-Cell RNA Profiling of Pooled Genetic Screens
============================================================================================================
  
  [@wikidata:Q34547509]  
  
Publication date : 01 of December, 2016  

# Highlights
We demonstrate Perturb-seq by analyzing 200,000 cells in immune cells and cell lines, focusing on transcription factors regulating the response of dendritic cells to lipopolysaccharide (LPS). 

To address this challenge, we develop Perturb-seq, combining the modularity of clustered regularly interspaced short palindromic repeats (CRISPR)/Cas9 to perform multi-locus gene perturbation (Cong et al., 2013, Qi et al., 2013) with the scale of massively parallel single cell RNA sequencing (scRNA-seq) (Klein et al., 2015, Macosko et al., 2015) as a rich genomic readout.
Determining the fitness effects of each perturbation (STAR Methods), we found a strong proliferative advantage conferred by perturbing PTGER2, CABP7, and CIT, and a disadvantage when perturbing AURKA, TOR1AIP1, and RACGAP1 (Figure S6H). (Among K562 TFs, perturbing EGR1 had a disadvantage; Figure S6A.)
# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q34547509)  
 * [Wikidata](https://www.wikidata.org/wiki/Q34547509)  
 * [Author Disambiguator](https://author-
disambiguator.toolforge.org/work_item_oauth.php?id=Q34547509&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1016/J.CELL.2016.11.038)  
