
MS_HistoneDB, a manually curated resource for proteomic analysis of human and mouse histones.
=============================================================================================
  
  [@wikidata:Q34549515]  
  
Publication date : 10 of January, 2017  

# Highlights

nterpretation of mass spectrometry data relies on public databases which are either not exhaustive (Swiss-Prot) or contain many redundant entries (UniProtKB or NCBI). Currently, no protein database is ideally suited for the analysis of histones and the complex array of mammalian histone variants.

We propose two proteomics-oriented manually curated databases for mouse and human histone variants. We manually curated >1700 gene, transcript and protein entries to produce a non-redundant list of 83 mouse and 85 human histones.

Mouse and human histone entries were collected from different databases and subsequently curated to produce a non-redundant protein-centric resource, MS_HistoneDB

Altogether, histone variants have been described for H3, H2A, H2B and H1; H4 is the only histone for which no variant has been identified in mammals, but some organisms, such as the urochordate Oikopleura dioica, ciliates and trypanosomes, have evolved H4 variants [10, 27,28,29].

MS is a powerful technique to identify histones, their variants and their post-translational modifications but relies on databases with contradictory naming and excessive redundancy. Here we exhaustively collected histone sequences for mouse and human and used manual curation to establish a protein-centric list, MS_HistoneDB, dedicated to the proteomic study of mouse and human histones. 


# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q34549515)  
 * [Wikidata](https://www.wikidata.org/wiki/Q34549515)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q34549515&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1186/S13072-016-0109-X)  
