
Pressing needs of biomedical text mining in biocuration and beyond: opportunities and challenges
================================================================================================
  
  [@wikidata:Q36233671]  
  
Publication date : 26 of December, 2016  

# Highlights

We then describe related efforts within the text-mining community, with a special focus on the BioCreative series of challenge workshops. We believe that focusing on the near-term challenges identified in this work will amplify the opportunities afforded by the continued adoption of text-mining tools. Finally, in order to sustain the curation ecosystem and have text-mining systems adopted for practical benefits, we call for increased collaboration between text-mining researchers and various stakeholders, including researchers, publishers and biocurators.

In this article, we first discuss the two applications related to literature services, Europe PMC and SourceData, explaining both their value to the bioscience community and how text mining is essential for their continued progress. We next discuss two recent efforts supporting biological databases, BEL and VIROME, which curate information related to biological cause–effect relationships and microbiomes, respectively


Europe PMC is committed to enabling text mining. Currently, it provides features such as ‘Highlight terms’ to identify core biological entities such as genes/proteins and organisms within the article’s abstract view; the entities are also linked to relevant databases.
SourceData (http://sourcedata.embo.org/) (18) is building tools to allow researchers and publishers to generate machine-readable descriptions of data during the publication process and also to make this data searchable.


Biological Expression Language (BEL) is a knowledge representation developed by Selventa to capture biological cause-and-effect relationships from the scientific literature in a format suitable for computation.

--> Cool
Some online tools such as VIROME (32) and MG-RAST (33) do provide a route for the analysis results themselves to be made public. Recent work by VIROME (http://virome.dbi.udel.edu/) and others are working to ensure that such results are accompanied by standardized metadata to make them useful when considered in alternate contexts, but the ability to look for trends across projects remains very limited. 

--> Viral Informatics Resource for Metagenome Exploration - VIROME 
# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q36233671)  
 * [Wikidata](https://www.wikidata.org/wiki/Q36233671)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q36233671/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q36233671&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1093/DATABASE/BAW161)  
