
Single-cell transcriptome analyses reveal signals to activate dormant neural stem cells
=======================================================================================
  
  [@wikidata:Q36852027]  
  
Publication date : 01 of May, 2015  

# Highlights

Single-cell RNA-seq reveals a signature gene program in ependymal CD133+ cells

• VEGF is a mitogen for ependymal CD133+ cells and bFGF for subependymal B cells

• the fourth ventricle show NSC activity upon VEGF/bFGF treatment


Through single-cell transcriptome and weighted gene co-expression network analysis (WGCNA), we uncovered molecular properties of CD133+/GFAP− ependymal (E) cells in the adult mouse forebrain neurogenic zone


CD133+ ependymal neural stem cells (NSCs), lining not only the lateral but also the fourth ventricles and, together with basic fibroblast growth factor (bFGF), elicited subsequent neural lineage differentiation and migration. 


This region contains the previously described four cell types related to adult NSC activities: (1) ependymal E cells; (2) subependymal GFAP+ B cells, some of which are also referred to as mono-ciliated, CD133 (encoded by the prominin1 gene), and GFAP double-positive NSCs (Beckervordersandforth et al., 2010); (3) transit-amplifying C cells; and (4) neuroblast A cells (Figure S1).

Previously, we have demonstrated that during embryonic cortical development, immunoreactivity for CD133 labeled almost all cells in the germinal ventricular zone (VZ) lining the ventricular surface, which were considered NSCs (Coskun et al., 2008)

sing prominin1 gene-based lineage-tracing studies, we have shown that CD133+ cells give rise to neuroblast A cells in the rostral migratory stream (RMS) and interneurons in the olfactory bulb

Single-cell transcriptome analysis indicated that CD133+/GFAP− cells in the ependyma expressed vascular endothelial growth factor (VEGF) receptors (VEGFRs) but not basic fibroblast growth factor (bFGF) receptors (bFGFRs

Taken together, these studies demonstrated that single-cell transcriptome profiling provides new information that could be difficult to acquire using population-based analyses.

While single-cell analyses in combination with WGCNA revealed much information about the cells, modules that were shared by subsets of samples appeared to be quite effective in revealing common features or molecular signatures of specific groups of cells (ependymal CD133+ E cells, A cells, B cells, etc.)





# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q36852027)  
 * [Wikidata](https://www.wikidata.org/wiki/Q36852027)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q36852027&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1016/J.CELL.2015.04.001)  
