
Shifting the paradigm: new approaches for characterizing and classifying neurons.
=================================================================================
  
  [@wikidata:Q37629171]  

# Highlights

Cre recombinase-based approaches in particular allow the coupling of cell type specificity with a wide variety of genetic tools for visualization, molecular profiling, tract tracing and functional manipulation.

Although the concept of cell types has been thoroughly engrained in neuroscience since the time of Ramo´ny Cajal [1], there is remarkable disagreement about how literally the concept ofdiscrete cell types should be taken, and what the most salient features for defining cell types are.

While these features all describe the same underlying cells, each investigator typically measures one or several features at a time, creating difficulties linking work across different labora- tories, which is compounded by inconsistent nomencla- ture conventions for these features.

The outcome of the meeting, fittingly held in Petilla de Arago´n (the birthplace ofRamo´n y Cajal), was a simple standardized nomenclature for measurable features of cortical interneurons

While the methods for generation of large-scale, single modality data sets are growing, it is not easy to correlate different modalities without confidence that the different properties were measured in same cell classes.

The most promising advances toward improved cell classification schemes are emerging from the use of transgenics, by leveraging directed and combinatorial approaches to drive tissue-specific or cell-specific and temporal regulated gene expression.

Cre recombinase expression is driven by a cell type-specific promoter.

Data repositories are beginning to emerge for Cre-specific reagents (http://nagy.mshri.on.ca/cre/index.php).

--> Website is down.

A variety of Web-based research tools are available to promote organizational infrastructure, collect and mine data:

Allen Institute for Brain Science, Expression Atlas (AGEA) http://www.brain-map.org/ http://mouse.brain-map.org/agea/
Brain Architecture Knowledge Management System (BAMS) Neuroanatomical Ontology http://retzius.usc.edu/bkms/bams-ontology.html
Cell Centered Database (CCDB) http://ccdb.ucsd.edu/
Gene Expression Nervous System Atlas (GENSAT) http://www.gensat.org/
GenePaint  http://www.genepaint.org/
International Neuroinformatics Coordinating Facility (INCF)  http://www.incf.org/ 
NeuroMorpho http://neuromorpho.org
Neuroscience Information Framework (NIF), also part of the NIH’s NeuroGateway http://nif.nih.gov/ http://www. neurogateway.org/

Winden and colleagues applied weighted gene co-expression network analysis (WGCNA) to the Sugino data set and identified network modules that differentiate neuronal subtypes related.

ChR expressed under the control of a cell-specific promoter has been used to study a number of molecularly distinct cell types including parvalbumin-expressing cortical neurons [45] and Phox2b-expressing respiratory neurons [46].

The scaling of biological data generation and analytical techniques in modern neuroscience, coupled with the explosive growth of transgenic methods to reproducibly target and manipulate specific cell types, are elevating the development of neuronal taxonomies from a subjective esoteric endeavor to an objective, multimodal main- stream community effort. These taxonomies are likely to be built on a strong transcriptome underpinning, with stratification based not only on morphological and phys- iological features but also on developmental ontology, specific patterns of connectivity with other cell types, and in vivo functional contribution to circuits governing specific behaviors.

In this era of ‘omics’, with the tools discussed in hand and the emergence of more complex approaches on the horizon, the creation of a cellular inventory of the vertebrate brain is elevated from if to when.

4.Cahoy JD, Emery B, Kaushal A, Foo LC, Zamanian JL,Christopherson KS, Xing Y, Lubischer JL, Krieg PA, Krupenko SA et al.: A transcriptome database for astrocytes, neurons, and oligodendrocytes: a new resource for understanding brain development and function. J Neurosci 2008, 28:264-278.

The authors explore the molecular fingerprints of canonically ‘homoge- nous’ glial cell classes using FACS (fluorescent-activated cell sorting) to isolate transgenically labeled astrocytes across a range of early postnatal development in mouse, and then enrich for specific subclasses by using a ‘panning’ strategy based on cell surface-expressing variability within a S100b-positive cell population as compared with nonlabeled cells. This is a relatively straightforward but effective method that allowed the inves- tigators to fine tune molecular-based distinctions between neurons the less well-characterized variability seen within glia.

# Comments

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q37629171)  
 * [Wikidata](https://www.wikidata.org/wiki/Q37629171)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q37629171/P921%3BP4510)  
