## The origin and evolution of cell types
    [@wikidata:Q38798014]
    
### Highlights
Key to cell type identity are evolutionary changes in the ‘core regulatory complex’ (CoRC) of transcription factors, that make emergent sister cell types distinct, enable their independent evolution and regulate cell type-specific traits termed apomeres.

Typically, cellular functions require the cooperation of biomolecules that constitute  cellular modules. Consequently, a cell type has been envisaged as expressing an assemblage of cellular modules that perform discrete subfunctions.

From an evolutionary perspective, similarity may arise for different reasons, not all of which reflect bona-fide cell type homology due to inheritance from a common precursor.

For instance, different cell types can evolve similarity by cell phenotypic convergence. Alternatively, cell types might resemble each other because of concerted evolution, which occurs through mutations affecting genetic information used by multiple cell types.

New cell types have also appeared, or been lost, in many animal clades. 

Cell types must be rebuilt each generation and necessarily share some common developmental history.

Here, we define a cell type as ‘a set of cells in an organism that change in evolution together, partially independent of other cells, and are evolutionarily more closely related to each other than to other cells’.

Cell types are evolutionary units, which implies some genomic information exists is used only by the cells of a given type and not by other cells. 

We therefore propose that the formation of a new cell type identity requires the evolution of a unique cell type regulatory signature that includes a cell type specific core regulatory complex (CoRC).

Set of transcription factors and their cooperative inter actions that first enabled the evolution of independent gene expression, and thereby made the new cell type distinct from its evolutionary sister cell type

Cell type identity is defined by the regulatory mechanisms that enable and maintain the distinct gene expression program of a cell type within the organism. A homologous cell type remains recognizable across species, even when it acquires lineage specific phenotypic differences, owing to strong evolutionary conservation of these regulatory mechanisms.

We refer to the set of transcription factors that control cell type specific gene expression as terminal selectors(as defined by Hobert).

Forced expression of one or a few of these factors is sufficient to alter cell type identity.

The full set of terminal selectors driving differentiation, together with microRNAs
, splicing complexes and any
other regulatory mechanism that establishes and maintains the differential use of genomic information of a cell within the organism, represents the cell type ‘regulatory signature’ (as defined by Arendt) or ‘character identity network’ (as defined by Wagner17).

New cell type specific modules, called apomeres, to the diverging cell type lineages. These become synapomeres if descendant sister cell types undergo additional rounds of evolutionary diversification. 

In summary, the evolution of a new regulatory signature is necessary for establishing a novel cell type, which represents a new, individuated evolutionary unit within the organism.

Relatively few genes may be uniquely expressed in a specific cell type, which has important implications for cell type evolutionary interdependence. Effector genes are often used by different cell types for similar functions.

Module integration. Module integration leads to the emergence of apomeres by combining existing cellular machinery.

The two rounds of genome duplication during vertebrate evolution led to the duplication of almost all subunits in the ancestral pathway. This event facilitated the evolution of distinct rod and cone phototransduction apomeres.

Cell type evolutionary history reflects the diversification of sister cell types through the evolution of new genetic programmes, from the few cell types of metazoan ancestors to the many hundred cell types of most extant bilaterians. The hierarchical trees representing cell type evolutionary and developmental lineage histories may be incongruent.

A disconnect between cell type identity and developmental lineage was first indicated in late nineteenth century embryological experiments conducted by Vincenzo Colucci. In his research, Colucci showed that the lens in the eyes of urodeles (an order of amphibians) could be regenerated from the iris.

How then are we to understand the apparent disconnect between evolutionary and developmental histories? We propose the concept of serial sister cell types as a solution to this conundrum.

We propose that these regionalization events also led to the duplication and subsequent diversification of at least one of the cell types that populated that region.

serial sister cell types arise from different regions and are produced by different stem cells, despite sharing a close evolutionary relationship

Our evolutionary definition of a cell type will facilitate comparative cell biological and organismal research in a broad range of disciplines, from development to physiology or neurobiology, to name but a few.

The recognition that cell type identity and specific function depend on only a small set of genetic information, including CoRCs and apomeres, enables the efficient identification and categorization of cell types across the animal tree.

### Comments
Defines many interesting concepts:

Cellular modules: Protein complexes, pathways and molecular machines that make up cell structure and function.

Cell type homology: Cell types that trace back to the same cell type in a common ancestor.

Cell phenotypic convergence: Cell types that are
phenotypically similar due to independent changes occurring in separate evolutionary lineages.

Concerted evolution: Similar phenotypic changes that occur simultaneously across different cell types of the same species as a result of altering genetic information shared among the cell types.

Sister cell types: Cell types arising by the splitting of an ancestral cell type into two descendant cells via the process of individuation.

Assumes directly, from scratch, that cell types are restricted to single species.

Definition is not rigorously correct: by the first part each organism would have different cell types. And evolution does not apply for organisms! By organism, the intention was likely to be "organisms of the same evolutionary lineage"

By Arendt's definition "specialized Antigen Presenting Cells" is not a cell type.

The recognition that cell type identity and specific function depend on only a small set of genetic information, including CoRCs and apomeres, enables the efficient identification and categorization of cell types across the animal tree. --> I tend to disagre with this sentence.