
Cell-type deconvolution in epigenome-wide association studies: a review and recommendations.
============================================================================================
  
  [@wikidata:Q39314677]  
  
Publication date : 14 of March, 2017  

# Highlights
A major challenge faced by epigenome-wide association studies (EWAS) is cell-type
heterogeneity. 
. Here, we provide a critical review of the major existing
algorithms for correcting cell-type composition in the context of Illumina Infinium
Methylation Beadarrays, with the aim of providing useful recommendations to the
EWAS community

In the context of EWAS, the most relevant
epigenetic modification is DNA methylation


Reference-based
methods use an a priori defined set of DNAm reference
profiles for the cell types of interest, in other words,
those thought to be present in the tissue of interest.
These reference profiles are defined over a selected
set of CpGs, typically those which maximize the discriminative power of the different cell types

However, for certain phenotypes such as
age, it is fairly well accepted that specific DNAm
alterations (e.g., those that map to repressed chromatin
domains) [20], or those making up Horvath’s clock [63],
are not driven by cell type composition [64,65]. Indeed,
it would be hard to explain such tissue-independent
age-associated DNAm shifts purely in terms of alterations to the proportion of underlying cell types, since
these associations have even been observed in highly
diverse and purified cell populations, including stemcell populations [20,65–68]

the epigenomes, including DNA methylomes of all major and minor blood
cell subtypes are generated as part of Blueprint [71],
this resource will undoubtedly help generate more
complete reference DNAm databases, allowing more
refined higher resolution analyses of blood EWAS,
which may help resolve these outstanding issues. 


Generating more complete reference DNA methylomes for all major human cell types, as done by
IHEC, will also allow direct application of referencebased methods to many other complex tissues like
breast, brain, skin or cervix.

--> 1.4.4

Single-cell DNAm analysis also offers a means of
improving the resolution of reference-based methods.
While cell-to-cell variation in DNAm within a phenotypically homogeneous cell population has been
observed [77], it is yet unclear whether this intercellular epigenetic variation is of functional or phenotypic
importance.

# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q39314677)  
 * [Wikidata](https://www.wikidata.org/wiki/Q39314677)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q39314677/P921%3BP4510)  
 * [ScienceOpen](https://www.scienceopen.com/search#('v'~3_'id'~''_'isExactMatch'~true_'context'~null_'kind'~77_'order'~0_'orderLowestFirst'~false_'query'~'Cell-type%20deconvolution%20in%20epigenome-wide%20association%20studies%3A%20a%20review%20and%20recommendations.'_'filters'~!*_'hideOthers'~false))  
