
Single-cell spatial reconstruction reveals global division of labour in the mammalian liver.
============================================================================================
  
  [@wikidata:Q42320342]  
  
Publication date : 06 of February, 2017  

# Highlights

Here we measure the entire transcriptome of thousands of mouse liver cells and infer their lobule coordinates on the basis of a panel of zonated landmark genes, characterized with single-molecule fluorescence in situ hybridization7.

As a panel of landmark genes, we chose six highly expressed liver genes with diverse zonation patterns—the pericentral genes Glul and Cyp2e1 (ref. 9) and the periportal genes Ass1 (ref. 10), Asl10, Alb8 and Cyp2f2 (ref. 9) (Fig. 2a, Extended Data Fig. 1). 

All liver zonation profiles previously described were monotonically increasing or decreasing porto–centrally. Our high spatial resolution enabled identification of a new class of non-monotonic zonation profiles that peak at intermediate lobule layers (Figs. 3c, 4a, Extended Data Figs 5, 10). 

Liver gene expression is highly dynamic, with multiple genes showing circadian and metabolic variations30. Although our study focused on zonation profiles of fasted mice, similar reconstructions will decipher the dynamics of liver zonation profiles along these temporal axes. Another natural extension is the analysis of liver pathologies. Reconstruction of liver zonation profiles in response to liver intoxication will expose the plasticity of zonation and the extent to which different layers compensate for the functional loss of other layers.
# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q42320342)  
 * [Wikidata](https://www.wikidata.org/wiki/Q42320342)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q42320342&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1038/NATURE21065)  
