
Classifying Drosophila Olfactory Projection Neuron Subtypes by Single-Cell RNA Sequencing.
==========================================================================================
  
  [@wikidata:Q47391265]  
  
Publication date : 01 of November, 2017  

# Highlights
We established a single-cell RNA-seq protocol for neuronsand glia inDrosophiladTranscriptome identity corresponds with olfactoryprojection neuron subtypes

Neuronal cell types are specified by a combinatorialmolecular code

Drosophilaolfactory projection neurons (PNs) are among thebest-characterized neuronal types: different PN clas-ses target dendrites to distinct olfactory glomeruli,while PNs of the same class exhibit indistinguishableanatomical and physiological properties

heDrosophilaolfactory circuit offers an excellent systemto investigate the relationship between transcriptomes andneuronal cell types. 50 classes of olfactory receptor neurons(ORNs) form one-to-one connections with 50 classes of sec-ond-order projection neurons (PNs) in the antennal lobe indiscrete glomeruli, forming 50 parallel information-processingchannels (Figure 1A;Vosshall and Stocker, 2007; Wilson,2013)

--> Hmmm Wikidata.

We sequenced cells fromDrosophilapupalbrains that were labeled by the astrocyte driveralrm-GAL4(Doh-erty et al., 2009) and olfactory PNs labeled by theGH146-GAL4driver, which is expressed in 40 of 50 PN classes 

igure 5. Identification of New Lineage-Specific Transcription Factors Using Single-Cell RNA-Seq(A) Visualization ofGH146+ PNs using tSNE as inFigure 2E showing expression ofacj6,C15,knot, andunpg. adPNs are outlined (based onacj6expression) andremaining cells are lPN

--> Cool combination of scRNAseq and other techniques


How is cell-type identity encoded in the transcriptome? It ispossible that (1) each cell type expresses at least one uniquegene, or (2) each cell type expresses a unique subset of a sharedpool of genes. The strategy used for encoding cell-type identityin the nervous system remains an unresolved question

--> - 2.4.2. Modern perspectives of classification of cells. This is also very essentialist, as if cell types were determined by a god

We ranked genes by their information content and then selecteda minimal set of genes by greedy search, iteratively drafting thegene carrying the most non-redundant information about identityinto the set until 95% of the uncertainty of subtype identity wasexplained. The result of this search is a set of genes for whichknowledge of their expression states (ON/OFF) alone is sufficientto classify subtype identity with high accuracy. We first appliedthis strategy to the threeMz19+PN classes. Only two genes,C15andCG31676, were sufficient to distinguish these threesubtypes (Figure 7A), explaining 92% of the uncertainty of clas-sification of individualMz19+PN cells into subtypes

--> - 3.5. Practical applications of a pragmatic cell type definition in 2.4.4
--> - 3.5.1. Basic dictionary for cell type identification based on regular expression of cell type markers


address this challenge, we developed an unsuper-vised machine-learning algorithm, iterative clustering for identi-fying markers (ICIM), to identify genes that distinguish PNclasses. ICIM searches for genes having the highest expressionvariability within a cell population, partitions the cells into two1208Cell171, 1206â€“1220, November 16, 2017
subpopulations using clustering based on these genes, then iter-atively repeats the search on each subpopulation. I


We have developed a machine-learning algorithm called ICIMfor unbiased identification of genes that distinguish subtypes.Because this algorithm recursively examines finer-grained sub-populations, it is capable of detecting genes that distinguishsmall subpopulations. ICIM is conceptually similar to previouslydescribed iterative analysis methods (Usoskin et al., 2015; Zeiselet al., 2015; Gokce et al., 2016; Tasic et al., 2016). However, ICIMmay discriminate highly similar cell types with greater sensitivitythan methods based on PCA would because it reduces thefeature space to only those genes that are informative for distin-guishing cell types

--> - 3.5. Practical applications of a pragmatic cell type definition in 2.4.4
--> - 0.1 fcoex & FCBF

Our classification is limited by samplingdepth because 17 classes contain only 1 cell per hemisphere (Yuet al., 2010). Sequencing of many more cells may resolve theseclasses into distinct clusters, resulting in a more completedescription of PN transcriptome diversity.

The transcriptomes of identified PN classes can inform designof more precise experiments in which simultaneous manipula-tion of multiple genes through loss- and gain-of-function ap-proaches allows experimental testing of the combinatorial TFand CSM codes.


# Comments

Cool use of the word "class"

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q47391265)  
 * [Wikidata](https://www.wikidata.org/wiki/Q47391265)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q47391265/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q47391265&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+workhttps://tabernacle.toolforge.org/?#/tab/manual/Q47391265/P921%3BP4510)  
