
Cell type discovery and representation in the era of high-content single cell phenotyping.
==========================================================================================
  
  [@wikidata:Q47870911]  

# Highlights


New high-throughput, high-content cytometry and sequencing technologies. The resulting explosion in the number of distinct cell types being identified is challenging the current paradigm for cell type definition in the Cell Ontology.

The Cell Ontology (CL). Two core relations – is_a and develops_from – with is_a used to relate specific cell subtypes to a more general parent cell type, and develops_from used to represent developmental cell lineage relationships.

Masci et al. proposed a major revision to the CL using dendritic cells as the driving biological use case in 2008. Cells were often experimentally-defined based on the expression of specific marker proteins on the cell surface.

Logically coherent asserted framework for defining cell types based on the expression levels of marker proteins. Diehl et al. applied this approach first to cell types of the hematopoietic system and then later to the full CL.

We will provide examples of how data being generated by these high content experimental platforms are used to identify novel cell types in both blood and brain. 

Applying the cell type names used by the investigative team, the cell type definitions derived from the gating hierarchy would then be:

Population #18: Monocytes – a PBMC that expresses HLA-DR and CD14, and lacks CD19 and CD3

Population #19: Dendritic cell (DC) – a PBMC that expresses HLA-DR, and lacks CD14, CD19, and CD3

Monocyte - Morphology: Mononuclear cell, diameter, 14 to 20 μM, N/C ratio 2:1-1:1.
Nucleus may appear in variety of shapes: round, kidney, lobulated, or convoluted. Fine azurophilic granules present; markers: CD11b (shared with other myeloid cells), human: CD14, mouse: F4/80-mid, GR1-low; location: Blood, but can be recruited into tissues; role or process: immune & tissue remodeling; lineage: hematopoietic, myeloid. Myeloid mononuclear recirculating leukocyte that can act as a precursor of tissue macrophages, osteoclasts and some populations of tissue dendritic cells.

CD14-positive monocyte - This cell type is compatible with the HIPC Lyoplate markers for ‘monocyte’. Note that while CD14 is considered a reliable marker for human monocytes, it is only expressed on approximately 85\% of mouse monocytes. A monocyte that expresses CD14 and is negative for the lineage markers CD3, CD19, and CD20.

The CL monocyte definition includes information about cellular and nuclear morphology, for which we have no information from our flow analysis. 

The definition of the CD14-positive monocyte is very close to the monocyte cells identified in the flow cytometry experiment in that they are CD14+, CD3- and CD19-. However, since CD20 expression was not evaluated in the panel, we cannot be absolutely certain if the experimental cells represent an exact match to the CL counterpart.

Even with semantic assertions of marker protein expression used to formally define cell types (Fig. 2c), exact matching is not possible

Inconsistent use of experimentally reproducible criteria for cell type definition in CL. 

``A solution to the partial marker problem is sorely needed.`` 

A fundamental issue has also emerged in considering how to distinguish between discrete cell types and more fluid cell states.
cells are constantly responding and adapting to changes in their environment by subtly changing their phenotypic states.

Challenges:

Matching cell populations identified using assay platforms focused on molecular expression with cell types represented in the reference CL ontology that have been defined using other non-molecular characteristics;

--> impossible without species-specificity
--> impossible without mixing different classes in the same bag

matching cell populations identified using overlapping but non-identical marker panels.

--> technotype hierarchy

Adding new cell populations being rapidly identified with these high-throughput assay platforms to a reference ontology in a timely fashion.

--> open, easy, non burocratic addition, Wikipedia like. Systems for error correction, but freedom to change and make mistakes. Wikidata.

determining what kind of validation would be required to add a novel cell type to a reference ontology;

--> Very good question. Presence across members of a species by deductive reasoning. Confidence of its generalizability for species. Explanation of biological usefulness of such classification. 
determining if a standard naming and definition convention could be developed and adopted.

--> still open. The division between peritypes, conceptual cell types and archetypes

distinguishing between truly discrete cell types and responsive cell states.

--> distinction does not seem to be useful a priori.


Proposals:
- Establish a new working group
- Molecular phenotype-based definitions – The community should continue to focus cell type definitions in the CL on precisely describing the phenotype of the cells, molecular and otherwise, using a series of necessary and sufficient conditions expressed as logical axioms
- CL - If independent validation is required, the CL developers should consider the establishment of a “CL provisional ontology” that could be used to hold provisional cell type assignments.

--> An open system, like Wikidata, could store the data and cell types could be further classified into ontological levels.

- Incomplete overlapping of assessed phenotypes

--> Similar to the peritype proposal, but with the clarification of the species-specificity. Watchout that the use of the word "monocyte" in the definition adds a dangerous layer of vagueness.

- it will be critical to develop conventions for cell type naming and definition using data from transcriptional profiling experiments. For example, the 11 new cell types identified in Fig. 3 could be named by combining marker genes selectively expressed by the cells with the parent cell class and the context

--> Very good naming proposal for transcriptomics. A few points is that human readability is severely limited (not a problem for machines, though) and that this definition excludes cells from other contexts.

- 10.Necessary and sufficient conditions
--> Also, a rigorous definition.

- Use of negative assertions through “lacks expression of”.

--> Absence of property is also a property itself! Rigorous criteria, totally agree. 

- .Cell type matching. implementation and use of the Friedman-Rafsky statistical test in the FlowMap-FR tool for cross- sample cell population matching from flow cytometry data.
--> simpler approach: assignment to a technotype, and then matching between studies based on the ontological network that links both concepts.

- Cell types versus cell states - Our intuition is that there is a distinction between. The challenge is to come up with a coherent and consistent approach for making this distinction. We propose that the distinction relates to the stability and reversibility of the new cellular phenotype. 

--> If a cardiomiocyte does not contract when it is dividing, is it really a miocyte? Also, when you put reprogramming into account (as in the case of axolotls regeneration, for example), bona fide cell types lose their stability. 


# Comments
What does marker mean for a multispecies cell type?!

Is CD14 a marker of monocytes or of human monocytes?

Definition of CD14-monocytes depends on the definition of monocyte, which has way more things that can be measured in any given study.

Peritypes are a solution for the partial marker problem. 

Explosion in the number of new cell types being identified using these unbiased high- throughput/high-content experimental technologies.

Robustness accross same species individuals answers: if the change is robust and useful to understand the biology, than \textbf{yes it is a new cell type strictu sensu.

Precise definition of general cell types implies a degree of vagueness. Subdiving a CL archetype into more specific cell types might be beneficial.

# Tags
Core for 1.3.1.4. Cell Ontology (and CELDA)
Core for 2.4 The concept of "cell type"
# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q47870911)  
 * [Wikidata](https://www.wikidata.org/wiki/Q47870911)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q47870911/P921%3BP4510)  
