
Single-cell profiling of the developing mouse brain and spinal cord with split-pool barcoding.
==============================================================================================
  
  [@wikidata:Q52353816]  
  
Publication date : 15 of March, 2018  

# Highlights

s, we developed split-pool ligation-based transcriptome sequencing (SPLiT-seq), a single-cell RNA-seq (scRNA-seq) method that labels the cellular origin of RNA through combinatorial barcoding.

We used SPLiT-seq to profile nuclei from the
developing brain and spinal cord of postnatal
day 2 and 11 (P2 and P11) mice. The first round
of barcoding assigned identifiers for the P2 brain,
P2 spinal cord, P11 brain, and P11 spinal cord
samples (Fig. 2A and fig. S4). In total, four rounds
of barcoding (48 × 96 × 96 × 14) generated more
than 6 million distinct barcode combinations,
making it possible to process hundreds of thou-
sands of nuclei in a single experiment with min-
imal barcode collisions (2.5% expected collisions
for 150,000 nuclei).


Although some clusters corresponded to
abundant cell types, such as cerebellar granule
cells (CGCs), others mapped to rare and often
less-characterized cell types, such as mitral/tufted
cells.

In this work, we profiled hundreds of thousands
of cells using only basic laboratory equipment
with a library preparation cost of ~$0.01 per cell
(fig. S21 and table S11)


# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q52353816)  
 * [Wikidata](https://www.wikidata.org/wiki/Q52353816)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q52353816&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1126/SCIENCE.AAM8999)  
