
Improving Interpretation of Cardiac Phenotypes and Enhancing Discovery With Expanded Knowledge in the Gene Ontology
===================================================================================================================
  
  [@wikidata:Q53589942]  
  
Publication date : 01 of February, 2018  

# Highlights

In this study, we created a computational resource to facilitate genetic studies of cardiac physiology by integrating literature curation with attention to an improved and expanded ontological representation of heart processes in the Gene Ontology.

We determined that a combination of curation and ontology development for heart-specific genes and processes supports the identification and downstream analysis of genes responsible for the spread of the cardiac action potential through the heart.

Some examples of GO terms relevant to cardiac research are, respectively, voltage-gated sodium channel activity, Purkinje myocyte action potential, and Z disc.

MacArthur et al48 also describe selecting candidate genes based on a known function or role which is shared with other genes established as associated with the disease of interest or is consistent with their mutant phenotype. This approach can be easily undertaken using GO annotation data, and yet it is rarely included in the in silico investigations of GWAS.

oon et al3 profiled the transcriptome of human embryonic stem cells and compared this to the transcriptome of adult human VCMs, after filtering out the transcripts that were not significantly differentially expressed between the 2 cell types.
# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q53589942)  
 * [Wikidata](https://www.wikidata.org/wiki/Q53589942)  
 * [Author Disambiguator](https://author-
disambiguator.toolforge.org/work_item_oauth.php?id=Q53589942&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1161/CIRCGEN.117.001813)  
