
Bias, robustness and scalability in single-cell differential expression analysis
================================================================================
  
  [@wikidata:Q56879991]  
  
Publication date : 26 of February, 2018  

# Highlights

Many methods have been used to determine differential gene expression from single-cell RNA (scRNA)-seq data. We evaluated 36 approaches using experimental and synthetic data and found considerable differences in the number and characteristics of the genes that are called differentially expressed

Using the eight real null data sets, where no truly differential genes are expected, we evaluated the type I error control by recording the fraction of tested genes that were assigned a nominal P value of less than 0.05 (Fig. 1a). For unfiltered data sets, many methods struggled to correctly control the type I error, and the best performance was obtained by ROTS

--> That is bad. But the mainstream wones seem to have a nice result

# Comments

## Tags
1.4.2.2. Computational analysis of scRNA-seq data


# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q56879991)  
 * [Wikidata](https://www.wikidata.org/wiki/Q56879991)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q56879991/P921%3BP4510)  
 * [ScienceOpen](https://www.scienceopen.com/search#('v'~3_'id'~''_'isExactMatch'~true_'context'~null_'kind'~77_'order'~0_'orderLowestFirst'~false_'query'~'Bias%2C%20robustness%20and%20scalability%20in%20single-cell%20differential%20expression%20analysis'_'filters'~!*_'hideOthers'~false))  
