
Classification, Ontology, and Precision Medicine
================================================
  
  [@wikidata:Q57281359]  
  
Publication date : 11 of October, 2018  

# Highlights


 In this review, we describe ontologies and their use in compu-tational reasoning to support precise classification of patients for diagnosis, care management, and translational research.

 Table 1. Types of Terminologies Used for Computational Analysis.*

 --> Good  table with definitions for 1.2.2. Computational ontologies and their methods

Table 2. Standards, Terminologies, and Ontologies Widely Used in Clinical Medicine.*

--> Too many!

Some of the most important resources for infor-mation retrieval in the medical domain include Medical  Subject  Headings  (MeSH)  for  indexing  and searching PubMed; RxNorm,10 a terminology for generic and branded drugs; and the Unified Medical  Language  System  (UMLS),  which  inte-grates more than 100 clinical terminologies and coding  systems

Ontologies differ from terminologies in that ontologies define relationships between concepts in a way that allows computational logical rea-soning

Ontologies  can  also  support  integration  of  basic science data (e.g., data from animal models) and public knowledge (e.g., associations between genetic  variants  and  diseases),  enabling  patient  classification based on a corpus of data existing well beyond the EHR and permitting new clinical  insights.

he first modern medical classification that can be considered a true ontology of  diseases (nosol-ogy) was developed by Carl Linnaeus (1707–1778), who divided diseases into 11 classes, 37 orders and 325 species. His classifica-tion laid the foundation for work that eventually led to the first edition of the ICD, in 1893.

--> Good history for 1.3.1.3. MONDO and DO + 1.3.1.5. UMLS, NCiT and non-OBO resources


Currently, however, less than a quarter of  the  content  of  SNOMED  CT  is  logically  de-fined;  the  remainder  is  primitive  and  not  ame-nable to this method.

hown in this example are portions of the HPO profiles (green and brown) of two patients for whom clinical dysmorphologic analysis did not help establish the diagnosis, even though they were seen in the same clinic within weeks of each other. Clinical exome sequencing showed a mutation in KMT2A in both patients, which in combination with the phenotype comparisons led to a diagnosis of the Wiedemann–Steiner syndrome.42 Each set of HPO terms is compared with all other phenotypic profiles in the HPO database to find the best nonexact (“fuzzy”) match.41

 Monarch Initiative has extended the approach to human–mouse phenotype comparisons on the basis of a cross-species ontology of anatomy 45,46and the Mammalian Phenotype Ontology.47-49HPO-based phenotypic profile matching used in combination with genome sequencing has al-lowed  prioritization  of  candidate  genes  with  predicted pathogenic variants.

 
# Comments

## Tags


1.2. Formal representation of knowledge
1.3. Knowledge Representation in biology
1.3.1.3. MONDO and DO

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q57281359)  
 * [Wikidata](https://www.wikidata.org/wiki/Q57281359)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q57281359/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q57281359&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+workhttps://tabernacle.toolforge.org/?#/tab/manual/Q57281359/P921%3BP4510)  
