
Species and cell-type properties of classically defined human and rodent neurons and glia
=========================================================================================
  
  [@wikidata:Q57462271]  

# Highlights

Here, we report an efficient strategy for precise exploration of gene expression and epigenetic events in specific cell types in a range of species, including postmortem human brain.


It has been established, for example, that each anatomically distinct, classically defined cell type expresses a set of genes that is characteristic (Dougherty et al., 2010; Doyle et al., 2008), that these genes confer properties that are essential for specialized cellular functions (Kim et al., 2008; Nakajima et al., 2014), and that the expression of these genes is highly correlated with cell specific epigenetic states that organize nuclear function (Kriaucionis and Heintz, 2009; Melle´n et al., 2012)

--> Established is strong, however it is indeed solid. By "classically defined" I'd imagine that they mean histologically defined.

how these cell types vary between individuals or across species, whether the process of brain aging is equivalent between cell types, and why mutations in broadly expressed genes can have devastating consequences in one or a few select cell types.

We reasoned that we could expand the number of antibodies for purifying cell-type specific nuclei by targeting ER proteins in addition to nuclear regulatory proteins.

In this manuscript, we establish a workflow for purifying and molecular profiling of nuclei from
specific cell types in the mammalian nervous system without the need for transgenic animals

These profiles enabled us to perform comparative studies of gene expression at a deeper level than previously achieved, and revealed a surprising degree of evolutionary divergence of gene expression profiles in rodent and human brain, even in classically defined, highly conserved cerebellar cell types (D’Angelo, 2013; Eccles, 1967; Llinas, 1969).

We next produced gene exession profiles for each of the five cerebellar cell types by isolating RNA from the sorted nuclei and performing RNA-seq. 


An important reason for development of this approach was to enable cell-type specific profiling in species where transgenic strains are not readily available. As a first test of this property, we chose the rat because it is a well-established model organism for behavioral neuroscience.

Examination of known markers from each of these cell types established that relevant cell specific markers are enriched in the data for each cell type, and depleted from the other cell types.

Figure 3. (C) Browser view and heatmaps showing for each cell type, an example of a shared marker gene, a human-enriched gene, and a mouse-enriched gene.

We noted that although Lake et al. label two of their clusters as populations of Purkinje neu- rons, these clusters actually contain cerebellar GABAergic interneurons (no cerebellar interneurons were identified in the publication), as both clusters express cerebellar interneuron markers such as SLC6A1 and TFAP2B (Chiu et al., 2002; Zainolabidin et al., 2017), and neither express well charac- terized Purkinje markers such as CALB1 and CA8 (Jiao et al., 2005; Taniuchi et al., 2002; Tolosa de Talamoni et al., 1993; Whitney et al., 2008) (Figure 3—figure supplement 3A - left). Instead, the group labeled Purk1 express basket cell markers SORCS3 and LYPD6 (Doyle et al., 2008; Hermey et al., 2004; Schilling and Oberdick, 2009)(Figure 3—figure supplement 3A - left). These marker genes for Purkinje neurons, GABAergic interneurons, and basket neurons are correctly expressed in the mouse data from Saunders et al., although we note that while basket neuron sub- types 3 and 4 express Slc6a1 and Tfap2b, they do not express Sorcs3 and Lypd6 (Figure 3—figure supplement 3A - right). Thus, we have renamed the two populations from Lake et al. Basket and Interneuron type 2 (Int2). In keeping with the conventions of this paper, we also renamed their Cere- bellar astrocyte population Bergmann glia.

--> Lots of confusion! A rule-based system would never lead to such range of problems

First, gene expression in the brain is highly conserved in specific cell types across species. Accordingly, most abundant and cell-specific markers of well characterized cell types are shared between rodent and human brain. Second, despite this shared identity, there are a significant number of genes in each cell type whose expression is not conserved between species.

Although the consequences of these gene expression changes will have to be interro- gated in future studies, our data demonstrate that important differences in gene expression in specific CNS cell types occur between species and suggest that they may result in functionally important differences in the biochemical functions of even the most classically defined rodent and human cell types.

--> The "most classically defined"! Interesting wording

Gender To examine the effect of gender on our data, we compared gene expression in samples from males and females and identified genes that are differentially expressed. For example, we identi- fied Glycogenin 2 Pseudogene 1 (GYG2P1) as a male-enriched gene across all samples, although it is clearly more male-enriched in basket cells than granule cells and glia.

Although a consensus definition of cell type has yet to emerge for the mammalian CNS, a recent evolutionary model (Arendt, 2008; Arendt et al., 2016) is helpful for consideration of the data we have generated here. According to this model, the specific characteristics of homologous cell types can vary as long as they remain defined by a distinctive, shared regulatory apparatus.

--> Arendt is so hot

the fine-tuned biochemistry of homologous human and mouse CNS cell types is likely to differ signifi- cantly, and they argue strongly that differences in gene expression profiles cannot be used as the defining criterion for cell type identity between species.

--> Strongly put conclusion.

Finally, our data provide additional examples of the importance of epigenetic mapping in specific
cell types as a complement to gene expression analysis.
# Comments

Very complete and thorough article.

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q57462271)  
 * [Wikidata](https://www.wikidata.org/wiki/Q57462271)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q57462271/P921%3BP4510)  
