
Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris
=====================================================================
  
  [@wikidata:Q59070958]  

# Highlights

Direct and controlled comparison of gene expression in cell types that are shared between tissues, such as T lymphocytes and endothelial cells from different anatomical locations.

These data are by no means a complete representation of all mouse organs and cell types, they provide a first draft attempt to create an organism-wide representation of cellular diversity

Cluster-specific gene expression of known markers and genes that are differentially expressed between clusters to assign cell-type annotations to each cluster.

For each cluster, we provide annotations in the controlled vocabulary of the cell ontology to facilitate interexperiment comparisons

Cluster co-membership alone, however, is insufficient to conclude that two cell populations from different organs represent the same or similar cell types;

Ten thymus cells in cluster 3 (composed of 2,379 cells) are annotated as ‘leukocytes’, but they express Pecam1, which is an endothelial marker. This is a predictable artefact of the annotation scheme: because entire clusters, rather than individual cells, were annotated in each organ, a sufficiently rare cell type that was algorithmically grouped with a more populous cell type will be misannotated.

We investigated how transcription factors contribute to cell-type identity by clustering averaged gene-expression profiles for each cell type using only the 1,016 transcription factors expressed in our dataset.

136 transcription factors are needed to simultaneously define all cell types across all organs
--> bold statement

# Comments

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q59070958)  
 * [Wikidata](https://www.wikidata.org/wiki/Q59070958)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q59070958/P921%3BP4510)  
