
Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species
=============================================================================================================
  
  [@wikidata:Q64108372]  
  
Publication date : 01 of May, 2019  

# Highlights
scCoGAPS is an implementation of
NNMF that is specifically suited for large,
sparse scRNA-seq datasets. ProjectR
implements a transfer-learning
framework that rapidly projects new data
into learned latent spaces.


We have adapted a transfer-learning approach for high-throughput genomic data analysis with two new methods, scCoGAPS and projectR.

Adaptive Sparsity for Learning Factors from scRNA-Seq
(scCoGAPS): Theory

Figure 1. Mathematical Core of the scCoGAPS Algorithm
(A) scRNA-Seq data yields a data matrix that has each sample as a column and each observed gene expression value as a row. scCoGAPS decomposes the
preprocessed data matrix into two related matrices.

--> Nice way of presenting it


An open question in retinal development
is how progenitor cells can generate specific subtypes of
neuronal and glial cell types during specific intervals during
developmentâ€”a phenomenon known as progenitor compe-
tence (Bassett and Wallace, 2012; Javed and Cayouette,
2017). In an effort to identify genes associated with changes in
retinal progenitor cell (RPC) competence, we performed bulk
RNA-seq analysis on replicate populations of fluorescence-acti-
vated cell sorting (FACS)-isolated RPCs and post-mitotic cells,
which were isolated using the Chx10:GFP reporter (Rowan and
Cepko, 2004)

We identified a total of 80 scCoGAPS patterns across the full
developmental time course (Figure S2; Table S2). Pattern
weights were tested for significant differential cell-type repre-
sentations (Figure 3D) and predictive power (AUC) for each cell-type annotation (Figure S1F)


Figure 4. projectR Recovers Shared Cell Types in Independent Murine Retina scRNA-Seq Data

projectR Enables Latent Space Comparison across
Model Systems from the Developing Retina to the
Developing Brain

Using only patterns learned in the developing retina, we were
able to identify and annotate a variety of cellular features in these
data. Many progenitor-associated patterns project into adult
tissues with high levels of cell turnover, and specifically within
subsets of cells that are actively proliferating (Figure 7B).basal interme-
diate progenitor cells

By leveraging the structure generated by the co-regulation of
genes, we are able to find a reduced set of continuous factors
that describe cellular identity, state, and phenotype in a model
system where differential expression analysis and marker genes
are insufficient alone
# Comments


## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q64108372)  
 * [Wikidata](https://www.wikidata.org/wiki/Q64108372)  
 * [Author Disambiguator](https://author-
disambiguator.toolforge.org/work_item_oauth.php?id=Q64108372&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1016/J.CELS.2019.04.004)  
