
Transcriptional and Functional Programming of Decidual Innate Lymphoid Cells
============================================================================
  
  [@wikidata:Q89621143]  
  
Publication date : 01 of January, 2019  

# Highlights

. Dimensionality reduction and transcriptomic analysis revealed the presence of two novel CD56Bright decidual ILCs that express low T-bet and divergent Eomes levels. 

Vento-Tormo et al. (20) defined three decidual NK subtypes based on shared nearest-neighbor cluster analysis of single cell RNA-seq from first trimester decidua (dNK1-3). We obtained differentially expressed genes for each dNK population from Supplementary Table 7 with q < 0.05, and further defined “marker genes” for each cell type as described in the caption for Figure 4A (20). Marker genes for a cell population were positively expressed (log fold change >0), expressed in at least 10% of cells in the population, and expressed in fewer than 60% of other cells. Applying these rules yielded 135 genes specific to dNK1, 45 genes for dNK2, and 208 genes for dNK3.

 DensVM revealed 13 clusters in the merged data set t-SNE map (3 decidua basalis, 2 decidua parietalis, 3 PBMC; Figure 1A, top). CD56Dim/Bright NK cells, ILC3s, and LTi-like cells occupied unique regions of the generated t-SNE map (Figure 1A, bottom).

 
# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q89621143)  
 * [Wikidata](https://www.wikidata.org/wiki/Q89621143)  
 * [Author Disambiguator](https://author-
disambiguator.toolforge.org/work_item_oauth.php?id=Q89621143&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.3389/FIMMU.2019.03065)  
