
Supervised clustering for single-cell analysis
==============================================
  
  [@wikidata:Q90003298]  
  
Publication date : 01 of October, 2019  

# Highlights

With the increase in throughput and the improvement in analytical capabilities, two machine-learning-based tools CellAssign3 and Garnett4 allow automated cell-type annotation of scRNA-seq data

Even for an expert it can be time consuming, as it involves weighing quantitative evidence regarding the cluster with more qualitative information about the established roles of different genes.

--> 2.4.2.2.2. A posteriori identification in high-throughput metodologies

Considering these challenges in annotating clusters, this issue of Nature Methods presents a timely contribution in the form of two new elegant methods, CellAssign3 and Garnett4. Both methods are based on a well-known paradigm from machine-learning, supervised clustering, and they fill an important niche between unsupervised clustering methods and projection methods.

Unlike unsupervised methods, CellAssign and Garnett require the user to provide a list of marker genes for each cluster. At first, it may seem as if this requirement makes the methods less user-friendly than unsupervised methods. The upshot, however, is that a list of marker genes provides a natural and intuitive means of incorporating prior knowledge.



# Comments

## Tags

1.4.2.2.1. Clustering algorithms

2.4.2.2.2. A posteriori identification in high-throughput metodologies

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q90003298)  
 * [Wikidata](https://www.wikidata.org/wiki/Q90003298)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q90003298/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q90003298&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+workhttps://tabernacle.toolforge.org/?#/tab/manual/Q90003298/P921%3BP4510)  
