
Identifying cell types to interpret scRNA-seq data: how, why and more possibilities
===================================================================================
  
  [@wikidata:Q90784336]  
  
Publication date : 31 of March, 2020  

# Highlights
there are several means to transfer single-cell data into human understandable information, the most frequently used convention is to identify the constitution of cell types (and/or subtypes)
and their number/proportion in the sample, followed by some
differentially expressed genes (DEs) or ‘marker gene’ calls

This can be found in plenty of scRNA-seq-associated research papers
[9–15], in which the step ‘cell type identification’ is the
foundation for functional analysis. Thus, herein, we provide
an overview of methods that are used to identify cell types
from scRNA-seq data and the logic behind them, as well as
several specific tools that have been developed for this task,
and, in the end, we try to discuss the possibilities for scRNA-seq
development in the near future.

Although
labour-consuming and inevitably subjective and arbitrary, manual annotation must be mentioned here, because this primitive
slash-and-burn mode of research is still being adopted by many
laboratories and biotech companies and this present condition
calls for the development of annotation tools

--> Tough but fair

For classification, a set of reference cell types are also needed;
however, it differs from the former discussed ‘clustering + annotating’ method in terms of its resolution, in which it deals withthe transcriptome: no clustering is done with the target data;
instead, classification compares each cell in the target dataset
directly with its reference (Figure 1, right panel)


This does not mean that HCA excludes those traditional
means of cell type characterization; instead, HCA tries to
integrate them with newly developed profile-characterized
types by examining whether they are consistent with each
other and, fortunately, they are consistent, at least for readily
examined cell types, such as neuron [37] and spleen cells [38].

--> This consistency is disputable, though

Cell ontology

Cell ontology (CL) is one of the ontologies of The Open Biological and Biomedical Ontology (OBO) Foundry, which provides a
structured controlled vocabulary for cells. All the cell types in
CL are collected manually and are reviewed from articles with
PMID and books with ISBN according to the builder of CL. Since
CL is designed to be species neutral, it should be used together
with other species-specific ontologies, such as human ontology,
Drosophila anatomy ontology and so on [39].

--> Weird take on CL, but it is not innacurate. Tag 1.3.4 .

Intuitively, one should assume that values will perform much
better than binary data and that using only 0 and 1 may make
a mess across cell types. However, the supervised classification
tool SuperCT [25] has proven that using binary data as input
could clarify the cell types in scRNA-seq data, and it is sort of
analogous to flow cytometry, which also sorts cells into different
types or subtypes only by the existence of several known marker
genes in an even less-detailed way than do binarized sequencing data

--> !! This is so 3.5 and 2.4.4

However, this at
least reminds us that if it is merely for cell type identification,
0 and 1 s seem to be enough to get the work done, which makes
molecule-counting implements such as UMIs unnecessary

--> Disagree. There are many ways to assign 0s and 1s

Therefore, when choosing from binary or value, one could expect
that the seem-to-be course-grained binary data is more than
enough and that for some task such as classification, too many
details may be useless, due to the nature of the task itself.

--> Nice, old school comment


The artefacts of our interpretation of scRNA-seq data can have
multiple sources. Compared with annotations at the cluster
level, classification at the cellular level is thought to avoid
clustering artefacts due to parameter-dependency and insensitivity to cell types of small numbers [25]
which is based on
the assumption that the formerly defined cell types in the
reference data are of high accuracy because they come from
the accumulated knowledge of tons of thousands of biological
studies. However, in my opinion, although the existing clustering
algorithms are undoubtfully far from perfect, the existing cell
types are doubtful as well.

--> Nice!!


Other than the intrinsic structure of cells types, as discussed
above, the definition of ‘cell type’ itself might be a problem,
especially in regard to diseases such as cancer, which usually
cannot be identified to any known healthy cell type [18, 17].
Although the word ‘axiom’ frequently appears in the CL [42]
description, it may be misused to some extent though, denoting
the relatively arbitrary nature of the notion of cell type, as well as
the structure or the relationship among cell types in this manually reviewed ontology. However, sometimes this ontology is
used as a golden standard for cell type annotation [39]. Although
it has been partially proven in several studies [37, 38] that cell
types that are defined at various levels are consistent with one
another in healthy samples during stringently regulated biological processes, such as development, nobody knows what could
happen in the case of processes that have less predictability

--> Nicer! Tag 1.3.4 .

the mismatch
between our knowledge and the need for the level of hierarchy
is still unneglectable, in that newly discovered cell types are
usually at a lower level, such as subtypes of a formerly defined
cell type, thus making the superior level powerless in the task of
annotating its subtypes, leaving newly discovered subtypes to be
named alphabetically or numerically, without a human readable
name

Based on the discussion above, maybe it is better to ‘annotate
without classifying’, such as to interpret scRNA-seq results in a
structure-free or even cell-type-free way

--> this is key

Therefore, the development of possible ways to interpret scRNA-seq data in the
future might take two directions, either ignoring cell types and
trying to link phenotypes to profiles with other intermediates
or utilizing the readily well-accepted notion of the cell type as
an aid, but with a different approach, which means instead of
using it as a standard, maybe it is better to employ this concept
at a proper granularity and abandon it when the resolution is too
high and the cell type impedes our understanding of cells in the
sense of their function and relationships.

Problems such as the consistency among cell type
levels, the impairment of identifying subtypes and
even the notion of ‘cell type’ itself still exist in this
area, waiting to be solved in the future.


# Comments


## Tags

1.4.2.3


# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q90784336)  
 * [Wikidata](https://www.wikidata.org/wiki/Q90784336)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q90784336/P921%3BP4510)  
