
Expression Atlas update: from tissues to single cells
=====================================================
  
  [@wikidata:Q91007034]  
  
Publication date : 01 of January, 2020  

# Highlights

With the increased availability of single cell RNA-Seq datasets in the public archives, we have now extended Expression Atlas with a new added-value service to display gene expression in single cells. Single Cell Expression Atlas was launched in 2018 and currently includes 123 single cell RNA-Seq studies from 12 species.

In addition, Expression Atlas serves as a long term portal for transcriptomics data generated by large-scale genomics studies such as Gramene (5), the comparative resource for plants (http://www.gramene.org/) and the Pancancer Analysis of Whole Genomes (PCAWG, https://docs.icgc.org/pcawg/).

 Samples are annotated with cell types, where these have been submitted or made available by the primary authors of these datasets. Cell types are then linked with terms from the Cell Ontology.

Analysis pipelines for Single Cell Expression Atlas comprise two parts: gene expression quantification and downstream analysis. Whilst quantification is accomplished using technology-dependent pipelines, downstream analysis is currently performed identically for all studies.

All workflow components now use Bioconda (18) packages

The website can be searched by genes within or across species to reveal experiments, tissues and cell types where this gene is expressed or under which conditions it is a marker gene.

Each experiment in Single Cell Expression Atlas has its own experiment page that displays the results of the data analysis, provides links to the original archival source of the study

Expression Atlas remains committed to making available and visualizing large-scale datasets for the benefit of the wider scientific community. These include version 6 of Genotype-Tissue Expression (GTEx) (21), Arabidopsis Information Portal version 11 (ARAPORT11) (22), FANTOM5 (https://www.ebi.ac.uk/gxa/experiments?experimentSet=FANTOM5); ENCODE (https://www.ebi.ac.uk/gxa/experiments?experimentSet=ENCODE) and proteomics data via the Human Protein Atlas. Landmark single cell datasets include the SMART-seq2 part of the datasets from the Tabula muris project (https://www.ebi.ac.uk/gxa/sc/experiments/E-ENAD-15/) and the Malaria Cell Atlas (https://www.ebi.ac.uk/gxa/sc/experiments/E-CURD-2/).




# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q91007034)  
 * [Wikidata](https://www.wikidata.org/wiki/Q91007034)  
 * [Author Disambiguator](https://author-
disambiguator.toolforge.org/work_item_oauth.php?id=Q91007034&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1093/NAR/GKZ947)  
