
Probabilistic cell typing enables fine mapping of closely related cell types in situ
====================================================================================
  
  [@wikidata:Q91330127]  

# Highlights

Understanding the function of a tissue requires knowing the spatial organization of its constituent cell types

Here we introduce probabilistic cell typing by in situ sequencing (pciSeq), an approach that leverages previous scRNA-seq classification to identify cell types using multiplexed in situ RNA detection.

To validate that spatial transcriptomic analyses can genuinely
distinguish finely related cell types, it is essential to work in a sys- tem where ground truth is available from previous work with other methods9â€“11
. The interneurons of rodent hippocampal area CA1
provide a unique opportunity: several decades of work using meth- ods of anatomy, immunohistochemistry and electrophysiology have identified around 20 interneuron subtypes, which are arranged in a stereotyped spatial organization and differ in their computational function and expression of marker genes1

--> Too strong to say that there is a ground truth! The concept of the ground truth, as presented, is also very essentialistic.

To select a gene panel, we developed an algorithm that searches for a subset of genes that can together identify scRNA-seq cells to their original clusters, after downs- ampling expression levels to match the lower efficiency of in situ data (Methods).

--> Nice, this is basically feature selection.

After verifying that detected expression patterns matched in situ hybridization data from the Allen Mouse Brain Atlas19
, we
continued with two further experiments using the full 99-gene panel, on two and eight coronal sections, respectively. All 14 sec- tions were from one P25 male CD1 mouse and covered different parts of the dorsal hippocampus

--> All in one mouse!

 A fundamental challenge for in situ cell typing is assigning genes to cells, as boundaries between cells are difficult to obtain in 2D imaging

we developed a Bayesian algorithm which leverages scRNA-seq data to simultaneously esti- mate the probability of assigning each read to each cell and each cell to each class.

for all 28 cell-type maps, see Supplementary Results; online viewer at http://insitu.cortexlab.net)

--> Wow, super cool explorer.

The cell-type assignments of the algo- rithm conformed closely to known combinatorial patterns of gene expression in CA1 interneuron subtypes. Across all experiments, the patterns of both classical and new interneuron markers were consistent with scRNA-seq results, as well as the known biology of CA1 interneurons (Supplementary Discussion; Supplementary Fig. 13)

We took cell-type definitions from the scRNA-seq data pub- lished by Zeisel et al.8
, using all neuronal types that the authors
annotated to be present in those cortical regions found in the coro- nal section analyzed (isocortex, cingulate/retrosplenial and piri- form

--> No cell type _definitions_ on that article. Only "expert-based" identification/classification.

while the method has higher false-neg- ative rates than FISH-based approaches, classification of cell types can still confidently be performed by designing a panel of genes that are expressed strongly enough to ensure enough reads of each are present.


# Comments

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q91330127)  
 * [Wikidata](https://www.wikidata.org/wiki/Q91330127)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q91330127/P921%3BP4510)  
