
A single-cell transcriptome atlas of the adult human retina
===========================================================
  
  [@wikidata:Q92766837]  
  
Publication date : 22 of August, 2019  

# Highlights


, we identified18tran-scriptionally distinct cell populations representing all known neuralretinal  cells:  rod  photoreceptors,  cone  photoreceptors,  Müller  glia,bipolar  cells,  amacrine  cells,  retinal  ganglion  cells,  horizontal  cells,astrocytes, and microglia. Our data captured molecular profiles forhealthy  and  putative  early  degenerating  rod  photoreceptors,  andrevealed  the  loss  ofMALAT1expression  with  longer  post-mortemtime,  

Herein, we report the generation of a human neural retina tran-scriptome atlas using 20,009 single cells collected from three healthydonors.

n this study, we extracted theneural retina from 12 donor eyes (Appendix Table S1). We observedconsistent cell viability across retinal tissues retrieved within 15 hpost-mortem

Based on known markers (Blackshawet al, 2001; Imanishiet al,2002; Corboet al, 2007; Sotoet al, 2008; Klimovaet al, 2015;Macoskoet al, 2015; Shekharet al, 2016; Vecinoet al, 2016), wewere able to assign cell identities to 16 of the 18 clusters (Fig 1A–D), corresponding to rod photoreceptors (PDE6A, CNGA1, RHO),cone photoreceptors (ARR3, GNGT2, GUCA1C), Mu ̈ller glia (RLBP1/CRALBP), retinal astrocytes (GFAP), microglia (HLA-DPA1, HLA-DPB1, HLA-DRA), bipolar cells (VSX2, OTX2), retinal ganglion cells(NEFL, GAP43, SNCG), amacrine cells (GAD1, CALB1, CHAT), and horizontal cells (ONECUT1, ONECUT2). 


mportantly, many rod photoreceptorclusters consist of a majority of cells from a single donor (>90% forC0, C2, and C4 and>80% for C1 and C7; Fig 2B). It is possible thatthis observation is due to the systematic biases such as differencesin tissue retrieval time, age of donors, or other sample preparationvariation

--> Good to notice that 


In this regard, we compared the spontaneously immortalizedhuman Mu ̈ller glia cell line MIO-M1 (Limbet al, 2002; Lawrenceet al, 2007) to all the retinal cell types identified in our dataset. UsingscRNA-seq, we profiled 7,150 MIO-M1 cells with 23,987 reads percell


Together, our results highlight the similarities and dif-ferences in MIO-M1 to adult retinal glial cells in humans


--> Interesting comparison of cell line with cell type, maybe useful for 2.2.2. Bacteria, cell lines and the plurality of the concept of taxon

Another outcome of this study is the assessment of biomarkersthat allow identification of major retinal cell types and subtypes



# Comments

Many cell types to Wikidata :) 

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q92766837)  
 * [Wikidata](https://www.wikidata.org/wiki/Q92766837)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q92766837/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q92766837&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.15252/EMBJ.2018100811)  
