
CHETAH: a selective, hierarchical cell type identification method for single-cell RNA sequencing
================================================================================================
  
  [@wikidata:Q92937241]  

# Highlights

CHETAH (CHaracterization of cEll Types Aided by Hierarchical classification) is an accurate cell type identification algorithm.

Single cell RNA sequencing has paved the way for discovering previously uncharacterized cell types and this application too would greatly benefit from efficient identification of known cell types prior to focusing on new types

Analyses are becoming more complex given the fast-expanding catalogue of defined cell types.

For example, if the evidence for a very specific cell type assignment such as effector CD8 T-cell is not strong enough, assignment such as T cell may  be useful.

The correct T-cell subtype may be not part of the reference dataset, or even that there is in- sufficient read-depth for the more specific call to be made. CHETAHâ€™s classification is a stepwise process that traverses the tree and, depending on the available evidence, ends at one of the reference cell types or halts at the unassigned or one of the intermediate types.
 
It is firmly established that hierarchical trees derived from clustering gene expression data reflect many aspects of the underlying biology (52).

--> Strong affirmation

For the method presented here, the definition of cell type is pragmatic and can best be described as any group of cells annotated within a reference set as belonging together.

# Comments

argues that we need to identify what we know before we can efficiently focus on new types. 

explicitly assumes one tree

explicitly defines cell type! awesome!

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q92937241)  
 * [Wikidata](https://www.wikidata.org/wiki/Q92937241)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q92937241/P921%3BP4510)  
