
Highly parallel and efficient single cell mRNA sequencing with paired picoliter chambers
========================================================================================
  
  [@wikidata:Q94474158]  
  
Publication date : 30 of April, 2020  

# Highlights
Here, we present a scRNA-seq platform, named Paired-seq, with high cells/beads utilization efﬁciency, cell-free
RNAs removal capability, high gene detection ability and low cost.

2009, single cell mRNA sequencing (scRNA-seq) was ﬁrst introduced by Tang to analyze the whole transcriptome in single cells 4 .

raditional methods, including limiting dilution 26 ,
capillary picking 27 , and laser capture microdissection (LCM) 28 ,
suffer from time and labor consumption, cell damage, and low
throughput.

Schematic diagram of cell and
barcoded bead pairing on Paired-seq chip. Scale bar is 200 μm.
<!-- beautiful chip  -->


Among all the 768 harvested cell barcodes, 386 were identiﬁed as
human species and 376 as mouse, yielding less than 0.8% mixed-
species dots (while 2.4% mixed-species dots with 2000-unit
Paired-seq chip)(Fig. 4b and Supplementary Fig. 2).

Although conventional methods, including CEL-seq2/
C1 and SCRB-seq, have higher gene detection ability due to the
use of liquid barcoding primers, the large reagent consumption
greatly increases the cost and the complexity of manual
operations, thus it is unpractical for high-throughput single-cell
analysis


# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q94474158)  
 * [Wikidata](https://www.wikidata.org/wiki/Q94474158)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q94474158&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1038/S41467-020-15765-0)  
