
Computational methods for the integrative analysis of single-cell data
======================================================================
  
  [@wikidata:Q94503935]  
  
Publication date : 04 of May, 2020  

# Highlights

The integrative analysis of genomic data, collected at different molecular layers from diverse cell populations, holds promise to address the full-scale complexity of biological systems. However, the combination of different single-cell genomic signals is computationally challenging, as these data are intrinsically heterogeneous for experimental, technical and biological reasons. 

. Most approaches adopt a transformation-based integration, meaning that each data type is first transformed into an intermediate form (e.g. through dimensionality reduction) and then all transformations are merged to perform the downstream integrative analysis [1, 9, 10]. This strategy borrows elements from the standard analysis of individual scRNA-seq samples where the identification of a set of latent variables enables the algorithms to distinguish groups of cells sharing common genomic traits despite the intrinsic heterogeneity of the data.


The goal of this survey is to present an overview of computational methods for the integrative analysis of single-cell data.
Most of the tools reviewed here are also listed in the ‘integration’ and ‘classification’ categories of the scRNA tool database (scrna-tools.org; [11]).

--> Cool database

Table 1Methods for the integration of multiple scRNA-seq datasets

--> Nice table

 Merging multiple scRNA-seq datasets requires to first remove all variations caused by batch effects (as differences in sample processing, library preparation or sequencing technology) or biological factors (e.g. treatments, individuals), which cause cells to cluster by sample, dataset or experiment, rather than by type

 --> Type is considered sample-independent


The mnnCorrect and fastMNN functions of the batchelor R package determine anchors taking advantage of mutual nearest neighbor (MNN) cells, i.e. identifying pairs of cells that are mutually closest to each other across batches

, Seurat v3 uses canonical correlation analysis (CCA) to identify a low-dimensional space where the correlation between the canonical variates is maximized. Anchors are defined as MNN cells in this reduced low-dimensional representation, filtered according to the original high-dimensional expression values and scored based on the shared overlap of mutual neighborhoods


Conos [26], BBKNN [27] and scPopCorn [28] implement joint graph-based approaches. In Conos (Clustering on network of samples), the integration of different datasets is obtained generating a weighted graph representation where shared populations are identified by community detection methods. 

Table 2Methods for the automatic cell label transfer from reference scRNA-seq datasets

 An alternative approach for cell annotation is to take advantage of well-annotated scRNA-seq atlases and automatically transfer cell type classifications from a reference to a query dataset (Figure 1B; Table 2). Some of the tools previously described for the integration of scRNA-seq data contain routines for the automatic transfer of cell labels [20, 26]. In particular, Seurat v3 identifies the anchors between reference and query sets and uses the anchor weights to predict labels of the query cells [20]. Conos, once integrated a reference and a query scRNA-seq dataset in a joint graph, can propagate cell labels between graph vertices, thus annotating the unlabeled cells of the query set [26]. Notably, both Seurat v3 and Conos can transfer labels across multiple modalities, as for instance annotate cells from a scATAC-seq experiment using transcriptional profiles as reference. 

 --> 1.4.2.3. Cell label identification


 perCT [40], ACTINN [41] and LAmbDA [42] exploit the knowledge encoding of artificial neural networks to predict unknown cell types from binarized or digital transcriptional data

 --> Good examples for fcoex! 


. The majority of these methods exploit algorithms commonly adopted in multi-view machine learning and build on sophisticated techniques for dimensionality reduction, pattern recognition, graph analysis, maximum likelihood estimation and statistical modeling. In most applications, the integrative analysis is approached in an unsupervised manner to identify cell types shared by different datasets or previously unknown correlations between modalities. In some other cases, the integration is conducted supervising the transfer of information from one dataset to another (as in cell label transfer from well-annotated atlases) or across different layers (as in the joint analysis of transcriptional and spatial expression).

# Comments

Very good review.

## Tags
1.1.2. Interoperable data: dataset integration


# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q94503935)  
 * [Wikidata](https://www.wikidata.org/wiki/Q94503935)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q94503935/P921%3BP4510)  
