
Single-cell landscape of bronchoalveolar immune cells in patients with COVID-19
===============================================================================
  
  [@wikidata:Q94600823]  
  
Publication date : 12 of May, 2020  

# Highlights

We characterized bronchoalveolar lavage fluid
immune cells from patients with varying severity of COVID-19
and from healthy people by using single-cell RNA sequencing

Clustering analysis showed
31 distinct clusters composed of macrophages (CD68), neutrophils
(FCGR3B), myeloid dendritic cells (mDCs) (CD1C, CLEC9A), plasmacytoid dendritic cells (pDCs) (LILRA4), natural killer (NK) cells
(KLRD1), T cells (CD3D), B cells (MS4A1), plasma cells (IGHG4)
and epithelial cells (TPPP3, KRT18) (Fig. 1a), identified by signature
genes (Extended Data Fig. 1a,b).

--> "signature genes"

We re-clustered 20 macrophage subclusters to further dissect their
heterogeneity (Extended Data Fig. 2a). According to recent classification criteria4
 and typical markers (Extended Data Fig. 2b), the
macrophages were grouped by FCN1, SPP1 and FABP4 expression
patterns including the FCN1hi group 1, FCN1loSPP1+ group 2, SPP1+
group 3 and FABP4+ group 4 macrophages (Extended Data Fig. 2c)



These data suggest that lung macrophages in patients with severe
COVID-19 infection may contribute to local inflammation by
recruiting inflammatory monocytic cells and neutrophils though
CCR1 and CXCR2, whereas macrophages in patients with moderate
COVID-19 infection produce more T  cell-attracting chemokines
through engaging CXCR3 and CXCR6.

Fig. 2 | T cell clonal expansion in BALF from patients with COVID-19. a, The clustering of BALF-derived T cells in patients with COVID-19 (n=â€‰8, S3
excluded). The dotted lines partition the UMAP into CD4+ T cells (upper), CD8+ T cells (lower), innate T cells (left) and proliferating T cells (right
CCR7+ T
ZNF683+
 CD8+ T
GZMB+
 CD8+ T
Proliferating T
Innate T
PD-1+ T
Treg
Mixed T

--> types for Wikidata? 


Single-cell capturing and downstream library constructions were performed using
Chromium Single Cell 3g Reagent kits v.2 (10x Genomics; PN-120237, PN-120236
and PN-120262

Isolation of BALF cells. Approximately 20ml of BALF was obtained and placed
on ice

. The following criteria were then applied to each cell of all nine
patients and four healthy controls: gene number between 200 and 6,000, UMI
count > 1,000 and mitochondrial gene percentage < 0.1

--> I always wonder the bias introduced by the mito gene percentage cutoff

# Comments



## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q94600823)  
 * [Wikidata](https://www.wikidata.org/wiki/Q94600823)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q94600823/P921%3BP4510)  
 * [ScienceOpen](https://www.scienceopen.com/search#('v'~3_'id'~''_'isExactMatch'~true_'context'~null_'kind'~77_'order'~0_'orderLowestFirst'~false_'query'~'Single-cell%20landscape%20of%20bronchoalveolar%20immune%20cells%20in%20patients%20with%20COVID-19'_'filters'~!*_'hideOthers'~false))  
