
Aggregating transcript-level analyses for single-cell differential gene expression
==================================================================================
  
  [@wikidata:Q96127495]  
  
Publication date : 01 of June, 2020  

# Highlights

transcript-level analyses can identify differential expression events,
such as isoform-switching events, that a gene-level analysis may be
unable to reveal. However, we highlight several omissions and errors
that we believe unfairly penalized previously published approaches
in the report by Ntranos et al. We have performed our own analysis, which suggests that these approaches perform just as well if not
better than logistic regression, while being more flexible in terms
of modeling

In conclusion, we agree with Ntranos et  al. that using
transcript-level data to perform gene expression analyses in scRNAseq experiments is of use in revealing subtle differences across cell
populations that would otherwise be left undetected, as previously
published2,10. However, here we have shown that transcript-level
analysis with MAST, edgeR or limma followed by Šidák aggregation
leads to results that are comparable if not better on both experimental and simulated dataset

# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q96127495)  
 * [Wikidata](https://www.wikidata.org/wiki/Q96127495)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q96127495/P921%3BP4510)  
 * [ScienceOpen](https://www.scienceopen.com/search#('v'~3_'id'~''_'isExactMatch'~true_'context'~null_'kind'~77_'order'~0_'orderLowestFirst'~false_'query'~'Aggregating%20transcript-level%20analyses%20for%20single-cell%20differential%20gene%20expression'_'filters'~!*_'hideOthers'~false))  
