
Comparative transcriptomics of primary cells in vertebrates
===========================================================
  
  [@wikidata:Q97903250]  
  
Publication date : 27 of July, 2020  

# Highlights
Gene expression profiles in homologous tissues have been observed to be different between species, which may be due to differences between species in the gene expression program in each cell type, but may also reflect differences in cell type composition of each tissue in different species

Here, we compare expression profiles in matching primary cells in human, mouse, rat, dog, and chicken using Cap Analysis Gene Expression (CAGE) and short RNA (sRNA) sequencing data from FANTOM5.

An analysis of expression levels of mature miRNAs and of primary miRNAs identified by CAGE revealed that evolutionary old miRNAs are more likely to have conserved expression patterns than young miRNAs

Vertebrate organisms consist of hundreds of cell types, with more
than 400 cell types defined in human (Vickaryous and Hall 2006).

--> interesting ref, Vickaryous MK, Hall BK. 2006. Human cell type diversity, evolution, devel-
opment, and classification with special reference to cells derived from the neural crest. Biol Rev Camb Philos Soc 81: 425â€“455. doi:10.1017/
S1464793106007068

The confounding effects of cell type composition in tissue-
based studies can be avoided by comparing the transcriptome of
different species in homologous primary cells. Here, we present a
comparative analysis of genome-wide expression in vertebrate spe-
cies profiled in FANTOM5 (The FANTOM Consortium and the
RIKEN PMI and CLST (DGT) 2014; Lizio et al. 2017a,b) to elucidate
patterns of gene expression conservation during evolution.


The FANTOM5 collection contains Cap Analysis Gene Expression
(CAGE) data for three primary cell types in human, mouse, rat,
dog, and chicken, and for an additional 12 cell types in human
and mouse only (Supplemental Table S1).

In each species, we defined the dominant promoter for each
gene as the most highly expressed promoter associated with the
gene.


The correlation value decreased exponentially as a function of phylogenetic
distance between species and dropped off most rapidly for mesen-
chymal stem cells compared to aortic smooth muscle cells and he-
patocytes 

Next, we extended our comparative
analysis to the expression levels of
microRNAs (miRNAs). miRNAs are small
noncoding RNA (typically 22 nt) that si-
lence mRNA post-transcriptionally and
regulate biological processes such as cell
growth and differentiation by functional
effects on direct targets and regulatory
networks (Bracken et al. 2016)

The Human Cell Atlas aims to create
a comprehensive map of cell types in the
human body by profiling gene expres-
sion levels in single cells fromhealthy tis-
sues (Regev et al. 2017). Our comparative
analysis suggests that differences in the
regulatory signature (Arendt et al.
2016), rather than the overall gene ex-
pression patterns, are the key require-
ment for distinguishing cell types.

# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q97903250)  
 * [Wikidata](https://www.wikidata.org/wiki/Q97903250)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q97903250/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q97903250&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.1101/GR.255679.119)  
