
An era of single-cell genomics consortia
========================================
  
  [@wikidata:Q99418649]  
  
Date : 15 of September, 2020  

# Highlights

 Previous efforts to profile a wide range of human cell types
have been attempted by the FANTOM and GTEx consortia. Now, with the advancement in genomic technologies,
profiling the human body at single-cell resolution is possible and will generate an unprecedented wealth of data that
will accelerate basic and clinical research with tangible applications to future medicine


 Gene expression profiling of 400+ human cell types using CAGE
revealed gene-regulatory modules that define cell types
and states1,2
--> 2.4.3

 To circumvent this, the GTEx project has recently released a
unique strategy to infer cellular heterogeneity based on
gene signatures from different cell types known to be
present in a given tissue. The method relies on the Tabula
Muris dataset11 to deconvolute the cellular composition

Table 1 Single-cell profiling of major human organs and tissues.

--> Super useful table! For 3.4  and 1.4.2

 In parallel, tens and hundreds of thousands of single-cell
genomics data across various human tissues are leading to
the discovery of new cell types and states, fundamentally
changing the picture of human anatomy in multiple ways
(summary of large-scale single-cell genomics data across
human tissue in Table 1). Integrating our knowledge that
we gained through single-cell genomics shows tremendous
potential for translational discoveries and applications, and
impacting diagnostic and clinical practices.

Per into the complex nature of individual organ systems. Here, we summarize the evergrowing single-cell genomics consortia and describe their missions. We further showcase benefits from generating
single-cell data in a regional and coherent manner through the formation of single-cell consortia.

With membership reaching over 1600 and over 100 research institutes
represented, arriving at a single solution to create the atlas
can be logistically challenging to coordinate. At the same
time, the relatively flexible, community-based nature of
HCA attracts motivated scientists to freely explore the
field and collaboratively construct platforms that will set
new standards for building an international reference for
human cells.

Concurrent programs include The Cell Atlas, which represents 64 cell lines
with subcellular details, The Pathology Atlas representing 17 different
cancer types, The Brain Atlas of the human, pig, and
mouse, The Blood Atlas including the secretome, and The
Metabolic Atlas with over 120 curated molecular pathways

In the United States, the National Institutes of Health
(NIH) have recently reported to support the Human
Biomolecular Atlas Program (HuBMAP) for 7 years.

As one of the BRAIN initiative’s priority areas, the
consortium aims to characterize all cell types in the nervous system at single-cell resolution, and to develop
tools to record, mark, and manipulate neurons in the
living brain. The Brain Initiative. https://braininitiative.nih.gov/

--> The Cell Census Network (BICCN) is part of BRAIN: https://braininitiative.nih.gov/brain-programs/cell-census-network-biccn


Table 2 List of existing and nascent single-cell genomics consortiums
--> Core for 1.4.1

To further minimize possible technical and
biological variations, the consortium can provide SOPs
with clear instructions, and requires comprehensive
metadata (donor information, site, and time). The Human
Cell Atlas relies on a central repository for SOPs (protocols.io63) and systematic collection of metadata

--> Human Cell Atlas Method Development Community. https://www.protocols.io/groups/hca


Finally, when discussing cell
identity, one confounding factor that is especially relevant
to single-cell genomics consortia is how one differentiates
between cell type (stable features of a cell’s identity) and
cell state (transient aspects of a cell’s status). How to
firmly establish these concepts using data-driven and
generalizable approaches is a discussion that is necessary
within the single-cell consortium.
Once the cell types have been established, we can proceed to identify the gene signatures that are specific to
each type, and make inferences about the biology behind
them. The most common technique is to perform differential expression analysis among different populations
of cell types or states.

--> Core for 2.4! Also useful for 1.4.4

# Comments
1.4.1`
## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q99418649)  
 * [Wikidata](https://www.wikidata.org/wiki/Q99418649)  
 * [TABernacle](https://tabernacle.toolforge.org/?#/tab/manual/Q99418649/P921%3BP4510)  
 * [Author Disambiguator](https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q99418649&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)