
Genetic mapping of etiologic brain cell types for obesity
=========================================================
  
  [@wikidata:Q99587937]  
  
Publication date : 21 of September, 2020  

# Highlights

Developing a novel strategy for integrating scRNA-seq data with GWAS data, we identified 26, exclusively neuronal, cell types from the hypothalamus, subthalamus, midbrain, hippocampus, thalamus, cortex, pons, medulla, pallidum that were significantly enriched for BMI heritability (p<1.6×10−4)

In recent years, genome-wide association studies (GWAS) have identified about a thousand common (minor allele frequency, MAF ≥0.1) single-nucleotide polymorphisms (SNPs) that associate with body mass index

 and we recently leveraged mouse; single-cell RNA-sequencing (scRNA-seq) to implicate mediobasal hypothalamic cell types in obesity (Campbell et al., 2017). 

 Here, we developed two computational toolkits for human genetics-driven identification of cell types underlying disease and leveraged them to systematically identify cell types enriching for obesity susceptibility by combining publicly available BMI GWAS summary statistics from >457,000 individuals with scRNA-seq data spanning 380 cell types representing adult mouse organs especially the nervous system and 347 cell types from the adult mouse hypothalamus.

 <!-- 347 ontly in the adult mouse hypothalamus! -->

Here, we – due to its polygenic nature and well-controlled type I error rate – used CELLECT with S-LDSC as the genetic prioritization model to quantify the effects of cell type ES on BMI heritability. 

Overview of CELLECT and CELLEX and main datasets used (a) CELLECT quantifies the association between common polygenetic GWAS signal (heritability) and cell type expression specificity (ES) to prioritize relevant etiological cell types.

t BMI variants tend to colocalize with genes specifically expressed in neurons, while also demonstrating that CELLECT is able to prioritize relevant cell types across a number of complex traits.

The 22 BMI GWAS-enriched cell types mapped to eight brain regions, namely the subthalamus, midbrain, hippocampus, thalamus, cortex, pons, medulla and pallidum (Figure 4a). To assess the extent to which shared transcriptional signatures could explain the enrichments across the 22 cell types, we clustered all cell types based on their genes’ ESμ values. 


# Comments

## Tags

# Links
  
 * [Scholia Profile](https://scholia.toolforge.org/work/Q99587937)  
 * [Wikidata](https://www.wikidata.org/wiki/Q99587937)  
 * [Author Disambiguator](https://author-
disambiguator.toolforge.org/work_item_oauth.php?id=Q99587937&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work)  
 * [DOI](https://doi.org/10.7554/ELIFE.55851)  
