<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>human_id</th>
      <th>wikidata_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>The top 100 papers</td>
      <td> <a href=./notes/Q28250597.md> Q28250597 </a></td>
    </tr>
    <tr>
      <th>1</th>
      <td>The origin and evolution of cell types</td>
      <td> <a href=./notes/Q38798014.md> Q38798014 </a></td>
    </tr>
    <tr>
      <th>2</th>
      <td>Make code accessible with these cloud services</td>
      <td> <a href=./notes/Q91130546.md> Q91130546 </a></td>
    </tr>
    <tr>
      <th>3</th>
      <td>Genetic address book for retinal cell types.</td>
      <td> <a href=./notes/Q43299861.md> Q43299861 </a></td>
    </tr>
    <tr>
      <th>4</th>
      <td>A community-based transcriptomics classification and nomenclature of neocortical cell types</td>
      <td> <a href=./notes/Q98665291.md> Q98665291 </a></td>
    </tr>
    <tr>
      <th>5</th>
      <td>The network effect: studying COVID-19 pathology with the Human Cell Atlas</td>
      <td> <a href=./notes/Q96817154.md> Q96817154 </a></td>
    </tr>
    <tr>
      <th>6</th>
      <td>Comprehensive Classification of Retinal Bipolar Neurons by Single-Cell Transcriptomics.</td>
      <td> <a href=./notes/Q40967362.md> Q40967362 </a></td>
    </tr>
    <tr>
      <th>7</th>
      <td>Identifiers for the 21st century: How to design, provision, and reuse persistent identifiers to maximize utility and impact of life science data</td>
      <td> <a href=./notes/Q33037209.md> Q33037209 </a></td>
    </tr>
    <tr>
      <th>8</th>
      <td>The human disease network</td>
      <td> <a href=./notes/Q24678240.md> Q24678240 </a></td>
    </tr>
    <tr>
      <th>9</th>
      <td>The evolving concept of cell identity in the single cell era</td>
      <td> <a href=./notes/Q93086971.md> Q93086971 </a></td>
    </tr>
    <tr>
      <th>10</th>
      <td>PRINTS and its automatic supplement, prePRINTS.</td>
      <td> <a href=./notes/Q34169535.md> Q34169535 </a></td>
    </tr>
    <tr>
      <th>11</th>
      <td>The naming of neurons: applications of taxonomic theory to the study of cellular populations.</td>
      <td> <a href=./notes/Q34065481.md> Q34065481 </a></td>
    </tr>
    <tr>
      <th>12</th>
      <td>Making your database available through Wikipedia: the pros and cons</td>
      <td> <a href=./notes/Q28254676.md> Q28254676 </a></td>
    </tr>
    <tr>
      <th>13</th>
      <td>ezTag: tagging biomedical concepts via interactive learning.</td>
      <td> <a href=./notes/Q55483834.md> Q55483834 </a></td>
    </tr>
    <tr>
      <th>14</th>
      <td>GNormPlus: An Integrative Approach for Tagging Genes, Gene Families, and Protein Domains</td>
      <td> <a href=./notes/Q28200510.md> Q28200510 </a></td>
    </tr>
    <tr>
      <th>15</th>
      <td>tmVar 2.0: integrating genomic variant information from literature with dbSNP and ClinVar for precision medicine.</td>
      <td> <a href=./notes/Q47317210.md> Q47317210 </a></td>
    </tr>
    <tr>
      <th>16</th>
      <td>Assisting manual literature curation for protein-protein interactions using BioQRator</td>
      <td> <a href=./notes/Q33928865.md> Q33928865 </a></td>
    </tr>
    <tr>
      <th>17</th>
      <td>Neuronal diversity and convergence in a visual system developmental atlas</td>
      <td> <a href=./notes/Q101226729.md> Q101226729 </a></td>
    </tr>
    <tr>
      <th>18</th>
      <td>Enabling Open Science: Wikidata for Research (Wiki4R)</td>
      <td> <a href=./notes/Q26707522.md> Q26707522 </a></td>
    </tr>
    <tr>
      <th>19</th>
      <td>What Is it Like to Be a Bat?</td>
      <td> <a href=./notes/Q2703462.md> Q2703462 </a></td>
    </tr>
    <tr>
      <th>20</th>
      <td>Ten tips for a text-mining-ready article: How to improve automated discoverability and interpretability</td>
      <td> <a href=./notes/Q96032351.md> Q96032351 </a></td>
    </tr>
    <tr>
      <th>21</th>
      <td>Jointly defining cell types from multiple single-cell datasets using LIGER</td>
      <td> <a href=./notes/Q100526347.md> Q100526347 </a></td>
    </tr>
    <tr>
      <th>22</th>
      <td>Dovetailing biology and chemistry: integrating the Gene Ontology with the ChEBI chemical ontology.</td>
      <td> <a href=./notes/Q34878068.md> Q34878068 </a></td>
    </tr>
    <tr>
      <th>23</th>
      <td>Numerical taxonomy.</td>
      <td> <a href=./notes/Q50735370.md> Q50735370 </a></td>
    </tr>
    <tr>
      <th>24</th>
      <td>A Taxonomic Problem</td>
      <td> <a href=./notes/Q60137015.md> Q60137015 </a></td>
    </tr>
    <tr>
      <th>25</th>
      <td>Pathway information extracted from 25 years of pathway figures</td>
      <td> <a href=./notes/Q101473819.md> Q101473819 </a></td>
    </tr>
    <tr>
      <th>26</th>
      <td>The Development of Taxonomic Theory Since 1851</td>
      <td> <a href=./notes/Q59058260.md> Q59058260 </a></td>
    </tr>
    <tr>
      <th>27</th>
      <td>Semantic integration of gene expression analysis tools and data sources using software connectors.</td>
      <td> <a href=./notes/Q30570244.md> Q30570244 </a></td>
    </tr>
    <tr>
      <th>28</th>
      <td>The principles of science : a treatise on logic and scientific method</td>
      <td> <a href=./notes/Q51427139.md> Q51427139 </a></td>
    </tr>
    <tr>
      <th>29</th>
      <td>Some thoughts on bacterial classification</td>
      <td> <a href=./notes/Q74657187.md> Q74657187 </a></td>
    </tr>
    <tr>
      <th>30</th>
      <td>Cross-linking BioThings APIs through JSON-LD to facilitate knowledge exploration.</td>
      <td> <a href=./notes/Q49483648.md> Q49483648 </a></td>
    </tr>
    <tr>
      <th>31</th>
      <td>Knowledge-based biomedical Data Science</td>
      <td> <a href=./notes/Q57001498.md> Q57001498 </a></td>
    </tr>
    <tr>
      <th>32</th>
      <td>The Species Concept Among Viruses</td>
      <td> <a href=./notes/Q59063943.md> Q59063943 </a></td>
    </tr>
    <tr>
      <th>33</th>
      <td>The application of computers to taxonomy</td>
      <td> <a href=./notes/Q28181838.md> Q28181838 </a></td>
    </tr>
    <tr>
      <th>34</th>
      <td>Publishing DisGeNET as nanopublications</td>
      <td> <a href=./notes/Q31194033.md> Q31194033 </a></td>
    </tr>
    <tr>
      <th>35</th>
      <td>The moral status of animals</td>
      <td> <a href=./notes/Q79763269.md> Q79763269 </a></td>
    </tr>
    <tr>
      <th>36</th>
      <td>The use of models in bacterial classification</td>
      <td> <a href=./notes/Q78658697.md> Q78658697 </a></td>
    </tr>
    <tr>
      <th>37</th>
      <td>The types of retinal ganglion cells: current status and implications for neuronal classification.</td>
      <td> <a href=./notes/Q38432885.md> Q38432885 </a></td>
    </tr>
    <tr>
      <th>38</th>
      <td>Genuine semantic publishing</td>
      <td> <a href=./notes/Q46412887.md> Q46412887 </a></td>
    </tr>
    <tr>
      <th>39</th>
      <td>The anatomy of a nanopublication</td>
      <td> <a href=./notes/Q57011346.md> Q57011346 </a></td>
    </tr>
    <tr>
      <th>40</th>
      <td>Heritability enrichment of specifically expressed genes identifies disease-relevant tissues and cell types.</td>
      <td> <a href=./notes/Q52720499.md> Q52720499 </a></td>
    </tr>
    <tr>
      <th>41</th>
      <td>A gene expression atlas of the central nervous system based on bacterial artificial chromosomes</td>
      <td> <a href=./notes/Q29617753.md> Q29617753 </a></td>
    </tr>
    <tr>
      <th>42</th>
      <td>SourceData: a semantic platform for curating and searching figures.</td>
      <td> <a href=./notes/Q48094767.md> Q48094767 </a></td>
    </tr>
    <tr>
      <th>43</th>
      <td>Biological knowledge management: the emerging role of the Semantic Web technologies.</td>
      <td> <a href=./notes/Q37490532.md> Q37490532 </a></td>
    </tr>
    <tr>
      <th>44</th>
      <td>Systematic integration of biomedical knowledge prioritizes drugs for repurposing.</td>
      <td> <a href=./notes/Q42255083.md> Q42255083 </a></td>
    </tr>
    <tr>
      <th>45</th>
      <td>Utopia documents: linking scholarly literature with research data</td>
      <td> <a href=./notes/Q28292614.md> Q28292614 </a></td>
    </tr>
    <tr>
      <th>46</th>
      <td>Astrocyte layers in the mammalian cerebral cortex revealed by a single-cell in situ transcriptomic map</td>
      <td> <a href=./notes/Q90591926.md> Q90591926 </a></td>
    </tr>
    <tr>
      <th>47</th>
      <td>Web scraping technologies in an API world</td>
      <td> <a href=./notes/Q86694255.md> Q86694255 </a></td>
    </tr>
    <tr>
      <th>48</th>
      <td>How we learnt to stop worrying and love web scraping</td>
      <td> <a href=./notes/Q99234140.md> Q99234140 </a></td>
    </tr>
    <tr>
      <th>49</th>
      <td>A translational profiling approach for the molecular characterization of CNS cell types</td>
      <td> <a href=./notes/Q29615252.md> Q29615252 </a></td>
    </tr>
    <tr>
      <th>50</th>
      <td>Publishing on the semantic web</td>
      <td> <a href=./notes/Q28185770.md> Q28185770 </a></td>
    </tr>
    <tr>
      <th>51</th>
      <td>A periodic table of cell types</td>
      <td> <a href=./notes/Q93086975.md> Q93086975 </a></td>
    </tr>
    <tr>
      <th>52</th>
      <td>Universal cell type identifier based on number theory.</td>
      <td> <a href=./notes/Q52374237.md> Q52374237 </a></td>
    </tr>
    <tr>
      <th>53</th>
      <td>Neuronal cell types.</td>
      <td> <a href=./notes/Q35830509.md> Q35830509 </a></td>
    </tr>
    <tr>
      <th>54</th>
      <td>Heterogeneity of neutrophils</td>
      <td> <a href=./notes/Q91997728.md> Q91997728 </a></td>
    </tr>
    <tr>
      <th>55</th>
      <td>Natural killer cell immunotypes related to COVID-19 disease severity</td>
      <td> <a href=./notes/Q98622035.md> Q98622035 </a></td>
    </tr>
    <tr>
      <th>56</th>
      <td>Adventures in semantic publishing: exemplar semantic enhancements of a research article</td>
      <td> <a href=./notes/Q21092566.md> Q21092566 </a></td>
    </tr>
    <tr>
      <th>57</th>
      <td>Reproducibility of Flow Cytometry Through Standardization: Opportunities and Challenges</td>
      <td> <a href=./notes/Q90588777.md> Q90588777 </a></td>
    </tr>
    <tr>
      <th>58</th>
      <td>Application of a translational profiling approach for the comparative analysis of CNS cell types</td>
      <td> <a href=./notes/Q42585641.md> Q42585641 </a></td>
    </tr>
    <tr>
      <th>59</th>
      <td>Will a biological database be different from a biological journal?</td>
      <td> <a href=./notes/Q21092584.md> Q21092584 </a></td>
    </tr>
    <tr>
      <th>60</th>
      <td>The future of annotation/biocuration</td>
      <td> <a href=./notes/Q56880438.md> Q56880438 </a></td>
    </tr>
    <tr>
      <th>61</th>
      <td>Swiss-Prot: juggling between evolution and stability.</td>
      <td> <a href=./notes/Q47579749.md> Q47579749 </a></td>
    </tr>
    <tr>
      <th>62</th>
      <td>Retinal ganglion cells: properties, types, genera, pathways and trans-species comparisons.</td>
      <td> <a href=./notes/Q48839876.md> Q48839876 </a></td>
    </tr>
    <tr>
      <th>63</th>
      <td>WikiPathways: connecting communities</td>
      <td> <a href=./notes/Q102205677.md> Q102205677 </a></td>
    </tr>
    <tr>
      <th>64</th>
      <td>The neuron classification problem</td>
      <td> <a href=./notes/Q36854753.md> Q36854753 </a></td>
    </tr>
    <tr>
      <th>65</th>
      <td>Petilla terminology: nomenclature of features of GABAergic interneurons of the cerebral cortex</td>
      <td> <a href=./notes/Q24620160.md> Q24620160 </a></td>
    </tr>
    <tr>
      <th>66</th>
      <td>Protein ontology on the semantic web for knowledge discovery</td>
      <td> <a href=./notes/Q100525924.md> Q100525924 </a></td>
    </tr>
    <tr>
      <th>67</th>
      <td>Towards a classification of stem cells</td>
      <td> <a href=./notes/Q64097544.md> Q64097544 </a></td>
    </tr>
    <tr>
      <th>68</th>
      <td>Reader engagement with medical content on Wikipedia</td>
      <td> <a href=./notes/Q90109096.md> Q90109096 </a></td>
    </tr>
    <tr>
      <th>69</th>
      <td>How Single-Cell Genomics Is Changing Evolutionary and Developmental Biology.</td>
      <td> <a href=./notes/Q38622289.md> Q38622289 </a></td>
    </tr>
    <tr>
      <th>70</th>
      <td>The evolution of cell types in animals: emerging principles from molecular studies.</td>
      <td> <a href=./notes/Q37301036.md> Q37301036 </a></td>
    </tr>
    <tr>
      <th>71</th>
      <td>Key transitions during the evolution of animal phototransduction: novelty, "tree-thinking," co-option, and co-duplication.</td>
      <td> <a href=./notes/Q46221687.md> Q46221687 </a></td>
    </tr>
    <tr>
      <th>72</th>
      <td>Neuronal diversity: too many cell types for comfort?</td>
      <td> <a href=./notes/Q48373178.md> Q48373178 </a></td>
    </tr>
    <tr>
      <th>73</th>
      <td>The BridgeDb framework: standardized access to gene, protein and metabolite identifier mapping services</td>
      <td> <a href=./notes/Q28842753.md> Q28842753 </a></td>
    </tr>
    <tr>
      <th>74</th>
      <td>Definitions</td>
      <td> <a href=./notes/Q102232524.md> Q102232524 </a></td>
    </tr>
    <tr>
      <th>75</th>
      <td>A recipe book for cell types in the human brain</td>
      <td> <a href=./notes/Q93087299.md> Q93087299 </a></td>
    </tr>
    <tr>
      <th>76</th>
      <td>Single-cell sequencing paints diverse pictures of the brain</td>
      <td> <a href=./notes/Q58084012.md> Q58084012 </a></td>
    </tr>
    <tr>
      <th>77</th>
      <td>Neuron Names: A Gene- and Property-Based Name Format, With Special Reference to Cortical Neurons</td>
      <td> <a href=./notes/Q64065346.md> Q64065346 </a></td>
    </tr>
    <tr>
      <th>78</th>
      <td>The BRAIN Initiative Cell Census Consortium: Lessons Learned toward Generating a Comprehensive Brain Cell Atlas.</td>
      <td> <a href=./notes/Q42778383.md> Q42778383 </a></td>
    </tr>
    <tr>
      <th>79</th>
      <td>Transcriptional landscape of the prenatal human brain</td>
      <td> <a href=./notes/Q33926910.md> Q33926910 </a></td>
    </tr>
    <tr>
      <th>80</th>
      <td>Nanopublications: A Growing Resource of Provenance-Centric Scientific Linked Data</td>
      <td> <a href=./notes/Q62924270.md> Q62924270 </a></td>
    </tr>
    <tr>
      <th>81</th>
      <td>Transcriptomic and morphophysiological evidence for a specialized human cortical GABAergic cell type</td>
      <td> <a href=./notes/Q60017992.md> Q60017992 </a></td>
    </tr>
    <tr>
      <th>82</th>
      <td>DisGeNET-RDF: harnessing the innovative power of the Semantic Web to explore the genetic basis of diseases</td>
      <td> <a href=./notes/Q28596617.md> Q28596617 </a></td>
    </tr>
    <tr>
      <th>83</th>
      <td>Cell-type-specific consequences of Reelin deficiency in the mouse neocortex, hippocampus, and amygdala</td>
      <td> <a href=./notes/Q28595082.md> Q28595082 </a></td>
    </tr>
    <tr>
      <th>84</th>
      <td>The Semanticscience Integrated Ontology (SIO) for biomedical research and knowledge discovery</td>
      <td> <a href=./notes/Q19671659.md> Q19671659 </a></td>
    </tr>
    <tr>
      <th>85</th>
      <td>Ontology Design Patterns for Semantic Web Content</td>
      <td> <a href=./notes/Q57774232.md> Q57774232 </a></td>
    </tr>
    <tr>
      <th>86</th>
      <td>Single-nucleus and single-cell transcriptomes compared in matched cortical cell types</td>
      <td> <a href=./notes/Q60933812.md> Q60933812 </a></td>
    </tr>
    <tr>
      <th>87</th>
      <td>How should access to a COVID-19 vaccine be regulated?</td>
      <td> <a href=./notes/Q102542608.md> Q102542608 </a></td>
    </tr>
    <tr>
      <th>88</th>
      <td>Shifting the paradigm: new approaches for characterizing and classifying neurons.</td>
      <td> <a href=./notes/Q37629171.md> Q37629171 </a></td>
    </tr>
    <tr>
      <th>89</th>
      <td>OWL 2 Web Ontology Language Document Overview (Second Edition)</td>
      <td> <a href=./notes/Q102926772.md> Q102926772 </a></td>
    </tr>
    <tr>
      <th>90</th>
      <td>OWL 2 Web Ontology Language Primer (Second Edition)</td>
      <td> <a href=./notes/Q102929379.md> Q102929379 </a></td>
    </tr>
    <tr>
      <th>91</th>
      <td>A genetic and computational approach to structurally classify neuronal types.</td>
      <td> <a href=./notes/Q34182035.md> Q34182035 </a></td>
    </tr>
    <tr>
      <th>92</th>
      <td>Adult mouse cortical cell taxonomy revealed by single cell transcriptomics</td>
      <td> <a href=./notes/Q27311570.md> Q27311570 </a></td>
    </tr>
    <tr>
      <th>93</th>
      <td>Identifiers.org - Compact Identifier Services in the Cloud</td>
      <td> <a href=./notes/Q100500142.md> Q100500142 </a></td>
    </tr>
    <tr>
      <th>94</th>
      <td>Immune-centric network of cytokines and cells in disease context identified by computational mining of PubMed.</td>
      <td> <a href=./notes/Q64962407.md> Q64962407 </a></td>
    </tr>
    <tr>
      <th>95</th>
      <td>Single-nucleus RNA-seq identifies Huntington disease astrocyte states</td>
      <td> <a href=./notes/Q89763874.md> Q89763874 </a></td>
    </tr>
    <tr>
      <th>96</th>
      <td>Converting neXtProt into Linked Data and nanopublications</td>
      <td> <a href=./notes/Q103834013.md> Q103834013 </a></td>
    </tr>
    <tr>
      <th>97</th>
      <td>Innovations present in the primate interneuron repertoire</td>
      <td> <a href=./notes/Q100395427.md> Q100395427 </a></td>
    </tr>
    <tr>
      <th>98</th>
      <td>A curated database reveals trends in single-cell transcriptomics</td>
      <td> <a href=./notes/Q103034964.md> Q103034964 </a></td>
    </tr>
    <tr>
      <th>99</th>
      <td>Species and cell-type properties of classically defined human and rodent neurons and glia</td>
      <td> <a href=./notes/Q57462271.md> Q57462271 </a></td>
    </tr>
    <tr>
      <th>100</th>
      <td>Single cell transcriptomics in neuroscience: cell classification and beyond</td>
      <td> <a href=./notes/Q88605805.md> Q88605805 </a></td>
    </tr>
    <tr>
      <th>101</th>
      <td>Towards the collaborative curation of the registry underlying Identifiers.org</td>
      <td> <a href=./notes/Q28709067.md> Q28709067 </a></td>
    </tr>
    <tr>
      <th>102</th>
      <td>The Covid-19 Conundrum</td>
      <td> <a href=./notes/Q103964322.md> Q103964322 </a></td>
    </tr>
    <tr>
      <th>103</th>
      <td>Towards the automatic classification of neurons</td>
      <td> <a href=./notes/Q27010194.md> Q27010194 </a></td>
    </tr>
    <tr>
      <th>104</th>
      <td>Brain structure. Cell types in the mouse cortex and hippocampus revealed by single-cell RNA-seq.</td>
      <td> <a href=./notes/Q42776227.md> Q42776227 </a></td>
    </tr>
    <tr>
      <th>105</th>
      <td>Adding a little reality to building ontologies for biology</td>
      <td> <a href=./notes/Q21136172.md> Q21136172 </a></td>
    </tr>
    <tr>
      <th>106</th>
      <td>Data curation + process curation=data integration + science</td>
      <td> <a href=./notes/Q28303099.md> Q28303099 </a></td>
    </tr>
    <tr>
      <th>107</th>
      <td>Implementing a "publish, then review" model of publishing</td>
      <td> <a href=./notes/Q103811380.md> Q103811380 </a></td>
    </tr>
    <tr>
      <th>108</th>
      <td>Interneuron cell types are fit to function</td>
      <td> <a href=./notes/Q27021885.md> Q27021885 </a></td>
    </tr>
    <tr>
      <th>109</th>
      <td>Getting started in text mining</td>
      <td> <a href=./notes/Q21145382.md> Q21145382 </a></td>
    </tr>
    <tr>
      <th>110</th>
      <td>Abstract Meaning Representation for Sembanking</td>
      <td> <a href=./notes/Q104092178.md> Q104092178 </a></td>
    </tr>
    <tr>
      <th>111</th>
      <td>Reconstructing lineage hierarchies of the distal lung epithelium using single-cell RNA-seq</td>
      <td> <a href=./notes/Q34094294.md> Q34094294 </a></td>
    </tr>
    <tr>
      <th>112</th>
      <td>The genomic landscapes of individual melanocytes from human skin</td>
      <td> <a href=./notes/Q100491078.md> Q100491078 </a></td>
    </tr>
    <tr>
      <th>113</th>
      <td>Immune-evasive human islet-like organoids ameliorate diabetes</td>
      <td> <a href=./notes/Q98568692.md> Q98568692 </a></td>
    </tr>
    <tr>
      <th>114</th>
      <td>A single-cell transcriptome atlas of marsupial embryogenesis and X inactivation</td>
      <td> <a href=./notes/Q98568691.md> Q98568691 </a></td>
    </tr>
    <tr>
      <th>115</th>
      <td>Nanopublications for exposing experimental data in the life-sciences: a Huntington's Disease case study</td>
      <td> <a href=./notes/Q28606409.md> Q28606409 </a></td>
    </tr>
    <tr>
      <th>116</th>
      <td>Querying neXtProt nanopublications and their value for insights on sequence variants and tissue expression</td>
      <td> <a href=./notes/Q54670808.md> Q54670808 </a></td>
    </tr>
    <tr>
      <th>117</th>
      <td>Northstar enables automatic classification of known and novel cell types from tumor samples</td>
      <td> <a href=./notes/Q99564826.md> Q99564826 </a></td>
    </tr>
    <tr>
      <th>118</th>
      <td>Molecular biology gets wikified</td>
      <td> <a href=./notes/Q104161380.md> Q104161380 </a></td>
    </tr>
    <tr>
      <th>119</th>
      <td>Capillary cell-type specialization in the alveolus</td>
      <td> <a href=./notes/Q100560109.md> Q100560109 </a></td>
    </tr>
    <tr>
      <th>120</th>
      <td>Naming human genes</td>
      <td> <a href=./notes/Q98196903.md> Q98196903 </a></td>
    </tr>
    <tr>
      <th>121</th>
      <td>Guidelines for human gene nomenclature</td>
      <td> <a href=./notes/Q98148417.md> Q98148417 </a></td>
    </tr>
    <tr>
      <th>122</th>
      <td>Knowledge networks in the age of the Semantic Web</td>
      <td> <a href=./notes/Q28842577.md> Q28842577 </a></td>
    </tr>
    <tr>
      <th>123</th>
      <td>Internet encyclopaedias go head to head</td>
      <td> <a href=./notes/Q24239902.md> Q24239902 </a></td>
    </tr>
    <tr>
      <th>124</th>
      <td>snRNA-seq reveals a subpopulation of adipocytes that regulates thermogenesis</td>
      <td> <a href=./notes/Q101051363.md> Q101051363 </a></td>
    </tr>
    <tr>
      <th>125</th>
      <td>Beyond the frontiers of neuronal types.</td>
      <td> <a href=./notes/Q42156351.md> Q42156351 </a></td>
    </tr>
    <tr>
      <th>126</th>
      <td>Ontology Design Patterns for bio-ontologies: a case study on the Cell Cycle Ontology</td>
      <td> <a href=./notes/Q21093639.md> Q21093639 </a></td>
    </tr>
    <tr>
      <th>127</th>
      <td>Name-calling in the hippocampus (and beyond): coming to terms with neuron types and properties.</td>
      <td> <a href=./notes/Q37656899.md> Q37656899 </a></td>
    </tr>
    <tr>
      <th>128</th>
      <td>Biocuration - mapping resources and needs</td>
      <td> <a href=./notes/Q101217428.md> Q101217428 </a></td>
    </tr>
    <tr>
      <th>129</th>
      <td>Computing Machinery and Intelligence</td>
      <td> <a href=./notes/Q772056.md> Q772056 </a></td>
    </tr>
    <tr>
      <th>130</th>
      <td>ScispaCy: Fast and Robust Models for Biomedical Natural Language Processing</td>
      <td> <a href=./notes/Q101248419.md> Q101248419 </a></td>
    </tr>
    <tr>
      <th>131</th>
      <td>Hippocampome.org: a knowledge base of neuron types in the rodent hippocampus</td>
      <td> <a href=./notes/Q26253817.md> Q26253817 </a></td>
    </tr>
    <tr>
      <th>132</th>
      <td>An ontological approach to describing neurons and their relationships</td>
      <td> <a href=./notes/Q28730328.md> Q28730328 </a></td>
    </tr>
    <tr>
      <th>133</th>
      <td>Extracting a cellular hierarchy from high-dimensional cytometry data with SPADE</td>
      <td> <a href=./notes/Q34037159.md> Q34037159 </a></td>
    </tr>
    <tr>
      <th>134</th>
      <td>NeuroLex.org: an online framework for neuroscience knowledge</td>
      <td> <a href=./notes/Q28681367.md> Q28681367 </a></td>
    </tr>
    <tr>
      <th>135</th>
      <td>New insights into the classification and nomenclature of cortical GABAergic interneurons</td>
      <td> <a href=./notes/Q36747163.md> Q36747163 </a></td>
    </tr>
    <tr>
      <th>136</th>
      <td>Revisiting enigmatic cortical calretinin-expressing interneurons</td>
      <td> <a href=./notes/Q33796214.md> Q33796214 </a></td>
    </tr>
    <tr>
      <th>137</th>
      <td>Concept Wikiﬁcation for COVID-19</td>
      <td> <a href=./notes/Q104392113.md> Q104392113 </a></td>
    </tr>
    <tr>
      <th>138</th>
      <td>Local and Global Algorithms for Disambiguation to Wikipedia</td>
      <td> <a href=./notes/Q104415799.md> Q104415799 </a></td>
    </tr>
    <tr>
      <th>139</th>
      <td>Beyond the frontiers of neuronal types: fuzzy classification of interneurons.</td>
      <td> <a href=./notes/Q46137497.md> Q46137497 </a></td>
    </tr>
    <tr>
      <th>140</th>
      <td>Probabilistic cell typing enables fine mapping of closely related cell types in situ</td>
      <td> <a href=./notes/Q91330127.md> Q91330127 </a></td>
    </tr>
    <tr>
      <th>141</th>
      <td>mRNA-Seq whole-transcriptome analysis of a single cell</td>
      <td> <a href=./notes/Q28240611.md> Q28240611 </a></td>
    </tr>
    <tr>
      <th>142</th>
      <td>Entity linking for biomedical literature</td>
      <td> <a href=./notes/Q28642941.md> Q28642941 </a></td>
    </tr>
    <tr>
      <th>143</th>
      <td>SciBERT: A Pretrained Language Model for Scientific Text</td>
      <td> <a href=./notes/Q104433073.md> Q104433073 </a></td>
    </tr>
    <tr>
      <th>144</th>
      <td>Text mining approaches for dealing with the rapidly expanding literature on COVID-19</td>
      <td> <a href=./notes/Q104079663.md> Q104079663 </a></td>
    </tr>
    <tr>
      <th>145</th>
      <td>Structural Scaffolds for Citation Intent Classification in Scientific Publications</td>
      <td> <a href=./notes/Q104450390.md> Q104450390 </a></td>
    </tr>
    <tr>
      <th>146</th>
      <td>Integration of electrophysiological recordings with single-cell RNA-seq data identifies neuronal subtypes</td>
      <td> <a href=./notes/Q31033187.md> Q31033187 </a></td>
    </tr>
    <tr>
      <th>147</th>
      <td>Integrated Single-Cell Atlases Reveal an Oral SARS-CoV-2 Infection and Transmission Axis</td>
      <td> <a href=./notes/Q101210731.md> Q101210731 </a></td>
    </tr>
    <tr>
      <th>148</th>
      <td>Bridging the Gap Between Research and Practice: Predicting What Will Work Locally</td>
      <td> <a href=./notes/Q104475823.md> Q104475823 </a></td>
    </tr>
    <tr>
      <th>149</th>
      <td>Identification and characterization of distinct brown adipocyte subtypes in C57BL/6J mice</td>
      <td> <a href=./notes/Q103800764.md> Q103800764 </a></td>
    </tr>
    <tr>
      <th>150</th>
      <td>Dental cell type atlas reveals stem and differentiated cell types in mouse and human teeth</td>
      <td> <a href=./notes/Q99616616.md> Q99616616 </a></td>
    </tr>
    <tr>
      <th>151</th>
      <td>Construction of the Literature Graph in Semantic Scholar</td>
      <td> <a href=./notes/Q69587220.md> Q69587220 </a></td>
    </tr>
    <tr>
      <th>152</th>
      <td>CellMarker: a manually curated resource of cell markers in human and mouse</td>
      <td> <a href=./notes/Q56984510.md> Q56984510 </a></td>
    </tr>
    <tr>
      <th>153</th>
      <td>A standard nomenclature for referencing and authentication of pluripotent stem cells.</td>
      <td> <a href=./notes/Q48503347.md> Q48503347 </a></td>
    </tr>
    <tr>
      <th>154</th>
      <td>A resource for cell line authentication, annotation and quality control.</td>
      <td> <a href=./notes/Q38886793.md> Q38886793 </a></td>
    </tr>
    <tr>
      <th>155</th>
      <td>GeneDB and Wikidata</td>
      <td> <a href=./notes/Q91273803.md> Q91273803 </a></td>
    </tr>
    <tr>
      <th>156</th>
      <td>The Cellosaurus, a cell-line knowledge resource.</td>
      <td> <a href=./notes/Q54370168.md> Q54370168 </a></td>
    </tr>
    <tr>
      <th>157</th>
      <td>The bacterial species definition in the genomic era.</td>
      <td> <a href=./notes/Q33261434.md> Q33261434 </a></td>
    </tr>
    <tr>
      <th>158</th>
      <td>Theory of cell fate</td>
      <td> <a href=./notes/Q91908361.md> Q91908361 </a></td>
    </tr>
    <tr>
      <th>159</th>
      <td>Building a biomedical semantic network in Wikipedia with Semantic Wiki Links</td>
      <td> <a href=./notes/Q21169186.md> Q21169186 </a></td>
    </tr>
    <tr>
      <th>160</th>
      <td>Userscripts for the life sciences</td>
      <td> <a href=./notes/Q27062487.md> Q27062487 </a></td>
    </tr>
    <tr>
      <th>161</th>
      <td>biocuration</td>
      <td> <a href=./notes/biocuration.md> biocuration </a></td>
    </tr>
    <tr>
      <th>162</th>
      <td>SemMedDB: a PubMed-scale repository of biomedical semantic predications</td>
      <td> <a href=./notes/Q36432186.md> Q36432186 </a></td>
    </tr>
    <tr>
      <th>163</th>
      <td>Wikidata as a linked-data hub for Biodiversity data</td>
      <td> <a href=./notes/Q66058247.md> Q66058247 </a></td>
    </tr>
    <tr>
      <th>164</th>
      <td>Forcing cells to change lineages.</td>
      <td> <a href=./notes/Q34020397.md> Q34020397 </a></td>
    </tr>
    <tr>
      <th>165</th>
      <td>Stem cell states, fates, and the rules of attraction.</td>
      <td> <a href=./notes/Q37475461.md> Q37475461 </a></td>
    </tr>
    <tr>
      <th>166</th>
      <td>Integrated Morphoelectric and Transcriptomic Classification of Cortical GABAergic Cells</td>
      <td> <a href=./notes/Q102056745.md> Q102056745 </a></td>
    </tr>
    <tr>
      <th>167</th>
      <td>Common Cell type Nomenclature for the mammalian brain: A systematic, extensible convention</td>
      <td> <a href=./notes/Q104247451.md> Q104247451 </a></td>
    </tr>
    <tr>
      <th>168</th>
      <td>Neuro-symbolic representation learning on biological knowledge graphs.</td>
      <td> <a href=./notes/Q38377458.md> Q38377458 </a></td>
    </tr>
    <tr>
      <th>169</th>
      <td>Structured reviews for data and knowledge-driven research</td>
      <td> <a href=./notes/Q91866899.md> Q91866899 </a></td>
    </tr>
    <tr>
      <th>170</th>
      <td>Data management made simple</td>
      <td> <a href=./notes/Q56524364.md> Q56524364 </a></td>
    </tr>
    <tr>
      <th>171</th>
      <td>Everyone needs a data-management plan</td>
      <td> <a href=./notes/Q56524391.md> Q56524391 </a></td>
    </tr>
    <tr>
      <th>172</th>
      <td>Neuronal cell-type classification: challenges, opportunities and the path forward.</td>
      <td> <a href=./notes/Q46326031.md> Q46326031 </a></td>
    </tr>
    <tr>
      <th>173</th>
      <td>Naming of neurones. Classification and naming of cat retinal ganglion cells.</td>
      <td> <a href=./notes/Q41052480.md> Q41052480 </a></td>
    </tr>
    <tr>
      <th>174</th>
      <td>A rose by any other name... on 'naming of neurones' by Rowe and Stone.</td>
      <td> <a href=./notes/Q39790383.md> Q39790383 </a></td>
    </tr>
    <tr>
      <th>175</th>
      <td>Describing species: practical taxonomic procedure for biologists</td>
      <td> <a href=./notes/Q104043701.md> Q104043701 </a></td>
    </tr>
    <tr>
      <th>176</th>
      <td>Searching large-scale scRNA-seq databases via unbiased cell embedding with Cell BLAST</td>
      <td> <a href=./notes/Q97529981.md> Q97529981 </a></td>
    </tr>
    <tr>
      <th>177</th>
      <td>CellFishing.jl: an ultrafast and scalable cell search method for single-cell RNA sequencing</td>
      <td> <a href=./notes/Q64237155.md> Q64237155 </a></td>
    </tr>
    <tr>
      <th>178</th>
      <td>Probabilistic gene expression signatures identify cell-types from single cell RNA-seq data</td>
      <td> <a href=./notes/Q104371272.md> Q104371272 </a></td>
    </tr>
    <tr>
      <th>179</th>
      <td>CHETAH: a selective, hierarchical cell type identification method for single-cell RNA sequencing</td>
      <td> <a href=./notes/Q92937241.md> Q92937241 </a></td>
    </tr>
    <tr>
      <th>180</th>
      <td>Single-cell RNAseq cell cluster labelling by reference (celaref)</td>
      <td> <a href=./notes/Q104552099.md> Q104552099 </a></td>
    </tr>
    <tr>
      <th>181</th>
      <td>PanglaoDB: a web server for exploration of mouse and human single-cell RNA sequencing data</td>
      <td> <a href=./notes/Q63664483.md> Q63664483 </a></td>
    </tr>
    <tr>
      <th>182</th>
      <td>Unifying single-cell annotations based on the Cell Ontology</td>
      <td> <a href=./notes/Q104057222.md> Q104057222 </a></td>
    </tr>
    <tr>
      <th>183</th>
      <td>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq</td>
      <td> <a href=./notes/Q83231039.md> Q83231039 </a></td>
    </tr>
    <tr>
      <th>184</th>
      <td>MARS: discovering novel cell types across heterogeneous single-cell experiments</td>
      <td> <a href=./notes/Q100737318.md> Q100737318 </a></td>
    </tr>
    <tr>
      <th>185</th>
      <td>Eleven grand challenges in single-cell data science</td>
      <td> <a href=./notes/Q89599719.md> Q89599719 </a></td>
    </tr>
    <tr>
      <th>186</th>
      <td>A comparison of automatic cell identification methods for single-cell RNA sequencing data</td>
      <td> <a href=./notes/Q89998341.md> Q89998341 </a></td>
    </tr>
    <tr>
      <th>187</th>
      <td>scMontage: Fast and Robust Gene Expression Similarity Search for Massive Single-cell Data</td>
      <td> <a href=./notes/Q104552207.md> Q104552207 </a></td>
    </tr>
    <tr>
      <th>188</th>
      <td>Current best practices in single-cell RNA-seq analysis: a tutorial.</td>
      <td> <a href=./notes/Q64974172.md> Q64974172 </a></td>
    </tr>
    <tr>
      <th>189</th>
      <td>Revealing the vectors of cellular identity with single-cell genomics.</td>
      <td> <a href=./notes/Q39002947.md> Q39002947 </a></td>
    </tr>
    <tr>
      <th>190</th>
      <td>CellO: comprehensive and hierarchical cell type classification of human cells with the Cell Ontology</td>
      <td> <a href=./notes/Q104552259.md> Q104552259 </a></td>
    </tr>
    <tr>
      <th>191</th>
      <td>Single-cell RNA-seq reveals new types of human blood dendritic cells, monocytes, and progenitors.</td>
      <td> <a href=./notes/Q34555562.md> Q34555562 </a></td>
    </tr>
    <tr>
      <th>192</th>
      <td>Single-cell sequencing of the human midbrain reveals glial activation and a neuronal state specific to Parkinson's disease</td>
      <td> <a href=./notes/Q104552303.md> Q104552303 </a></td>
    </tr>
    <tr>
      <th>193</th>
      <td>Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets</td>
      <td> <a href=./notes/Q34044099.md> Q34044099 </a></td>
    </tr>
    <tr>
      <th>194</th>
      <td>Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris</td>
      <td> <a href=./notes/Q59070958.md> Q59070958 </a></td>
    </tr>
    <tr>
      <th>195</th>
      <td>Comprehensive single-cell transcriptional profiling of a multicellular organism.</td>
      <td> <a href=./notes/Q38619221.md> Q38619221 </a></td>
    </tr>
    <tr>
      <th>196</th>
      <td>Broad transcriptional dysregulation of brain and choroid plexus cell types with COVID-19</td>
      <td> <a href=./notes/Q104552382.md> Q104552382 </a></td>
    </tr>
    <tr>
      <th>197</th>
      <td>A Single-Cell Atlas of In Vivo Mammalian Chromatin Accessibility</td>
      <td> <a href=./notes/Q59770872.md> Q59770872 </a></td>
    </tr>
    <tr>
      <th>198</th>
      <td>Cell type discovery using single-cell transcriptomics: implications for ontological representation.</td>
      <td> <a href=./notes/Q52625486.md> Q52625486 </a></td>
    </tr>
    <tr>
      <th>199</th>
      <td>ontoProc: processing of ontologies of anatomy, cell lines, and so on</td>
      <td> <a href=./notes/Q101074371.md> Q101074371 </a></td>
    </tr>
    <tr>
      <th>200</th>
      <td>Cells in experimental life sciences - challenges and solution to the rapid evolution of knowledge.</td>
      <td> <a href=./notes/Q48519854.md> Q48519854 </a></td>
    </tr>
    <tr>
      <th>201</th>
      <td>Cells in ExperimentaL Life Sciences (CELLS-2018): capturing the knowledge of normal and diseased cells with ontologies.</td>
      <td> <a href=./notes/Q64928913.md> Q64928913 </a></td>
    </tr>
    <tr>
      <th>202</th>
      <td>Reporting and connecting cell type names and gating definitions through ontologies</td>
      <td> <a href=./notes/Q91614095.md> Q91614095 </a></td>
    </tr>
    <tr>
      <th>203</th>
      <td>An ontology for cell types</td>
      <td> <a href=./notes/Q21184168.md> Q21184168 </a></td>
    </tr>
    <tr>
      <th>204</th>
      <td>Logical development of the cell ontology.</td>
      <td> <a href=./notes/Q33786317.md> Q33786317 </a></td>
    </tr>
    <tr>
      <th>205</th>
      <td>An improved ontological representation of dendritic cells as a paradigm for all cell types</td>
      <td> <a href=./notes/Q21284365.md> Q21284365 </a></td>
    </tr>
    <tr>
      <th>206</th>
      <td>Hematopoietic cell types: prototype for a revised cell ontology</td>
      <td> <a href=./notes/Q33940690.md> Q33940690 </a></td>
    </tr>
    <tr>
      <th>207</th>
      <td>Usage of cell nomenclature in biomedical literature.</td>
      <td> <a href=./notes/Q47565472.md> Q47565472 </a></td>
    </tr>
    <tr>
      <th>208</th>
      <td>Cell ontology in an age of data-driven cell classification.</td>
      <td> <a href=./notes/Q49192555.md> Q49192555 </a></td>
    </tr>
    <tr>
      <th>209</th>
      <td>flowCL: ontology-based cell population labelling in flow cytometry</td>
      <td> <a href=./notes/Q23809269.md> Q23809269 </a></td>
    </tr>
    <tr>
      <th>210</th>
      <td>CLO: The cell line ontology</td>
      <td> <a href=./notes/Q28650204.md> Q28650204 </a></td>
    </tr>
    <tr>
      <th>211</th>
      <td>Formalization of taxon-based constraints to detect inconsistencies in annotation and ontology development.</td>
      <td> <a href=./notes/Q33727235.md> Q33727235 </a></td>
    </tr>
    <tr>
      <th>212</th>
      <td>Toward principles for the design of ontologies used for knowledge sharing?</td>
      <td> <a href=./notes/Q47303277.md> Q47303277 </a></td>
    </tr>
    <tr>
      <th>213</th>
      <td>What Is Your Conceptual Definition of "Cell Type" in the Context of a Mature Organism?</td>
      <td> <a href=./notes/Q87649649.md> Q87649649 </a></td>
    </tr>
    <tr>
      <th>214</th>
      <td>Exciting times to study the identity and evolution of cell types</td>
      <td> <a href=./notes/Q90182085.md> Q90182085 </a></td>
    </tr>
    <tr>
      <th>215</th>
      <td>Function first: classifying cell types and circuits of the retina</td>
      <td> <a href=./notes/Q93189154.md> Q93189154 </a></td>
    </tr>
    <tr>
      <th>216</th>
      <td>Defining cell types and states with single-cell genomics</td>
      <td> <a href=./notes/Q26782737.md> Q26782737 </a></td>
    </tr>
    <tr>
      <th>217</th>
      <td>Cell type discovery and representation in the era of high-content single cell phenotyping.</td>
      <td> <a href=./notes/Q47870911.md> Q47870911 </a></td>
    </tr>
    <tr>
      <th>218</th>
      <td>Defining cellular identity through network biology.</td>
      <td> <a href=./notes/Q42084918.md> Q42084918 </a></td>
    </tr>
    <tr>
      <th>219</th>
      <td>Mapping the transcriptional diversity of genetically and anatomically defined cell populations in the mouse brain</td>
      <td> <a href=./notes/Q64118290.md> Q64118290 </a></td>
    </tr>
    <tr>
      <th>220</th>
      <td>Terminal Selectors of Neuronal Identity.</td>
      <td> <a href=./notes/Q38771724.md> Q38771724 </a></td>
    </tr>
    <tr>
      <th>221</th>
      <td>Construction of a human cell landscape at single-cell level</td>
      <td> <a href=./notes/Q90665256.md> Q90665256 </a></td>
    </tr>
    <tr>
      <th>222</th>
      <td>The definition of cell type</td>
      <td> <a href=./notes/Q77807915.md> Q77807915 </a></td>
    </tr>
    <tr>
      <th>223</th>
      <td>The Human Cell Atlas: from vision to reality.</td>
      <td> <a href=./notes/Q47565008.md> Q47565008 </a></td>
    </tr>
    <tr>
      <th>224</th>
      <td>CELLPEDIA: a repository for human cell information for cell studies and differentiation analyses.</td>
      <td> <a href=./notes/Q35482351.md> Q35482351 </a></td>
    </tr>
    <tr>
      <th>225</th>
      <td>Human Cell Atlas and cell-type authentication for regenerative medicine</td>
      <td> <a href=./notes/Q99418657.md> Q99418657 </a></td>
    </tr>
    <tr>
      <th>226</th>
      <td>Developing Global Cellular Information Retrieval System with Minimum Reporting Guidelines on Cellular Data for Regenerative Medicine</td>
      <td> <a href=./notes/Q104553080.md> Q104553080 </a></td>
    </tr>
    <tr>
      <th>227</th>
      <td>The human body at cellular resolution: the NIH Human Biomolecular Atlas Program</td>
      <td> <a href=./notes/Q90617765.md> Q90617765 </a></td>
    </tr>
    <tr>
      <th>228</th>
      <td>An estimation of the number of cells in the human body.</td>
      <td> <a href=./notes/Q34037445.md> Q34037445 </a></td>
    </tr>
    <tr>
      <th>229</th>
      <td>Opinion: Why science needs philosophy</td>
      <td> <a href=./notes/Q63479773.md> Q63479773 </a></td>
    </tr>
    <tr>
      <th>230</th>
      <td>The meanings of 'function' in biology and the problematic case of de novo gene emergence</td>
      <td> <a href=./notes/Q83227592.md> Q83227592 </a></td>
    </tr>
    <tr>
      <th>231</th>
      <td>Philosophical bias is the one bias that science cannot avoid</td>
      <td> <a href=./notes/Q64097527.md> Q64097527 </a></td>
    </tr>
    <tr>
      <th>232</th>
      <td>The challenges of big data biology</td>
      <td> <a href=./notes/Q63241652.md> Q63241652 </a></td>
    </tr>
    <tr>
      <th>233</th>
      <td>From primal scenes to synthetic cells</td>
      <td> <a href=./notes/Q64097538.md> Q64097538 </a></td>
    </tr>
    <tr>
      <th>234</th>
      <td>Ontologies in biology: design, applications and future challenges.</td>
      <td> <a href=./notes/Q35670188.md> Q35670188 </a></td>
    </tr>
    <tr>
      <th>235</th>
      <td>Ontologies: Formalising biological knowledge for bioinformatics.</td>
      <td> <a href=./notes/Q38522811.md> Q38522811 </a></td>
    </tr>
    <tr>
      <th>236</th>
      <td>FR-Match: robust matching of cell type clusters from single cell RNA sequencing data using the Friedman-Rafsky non-parametric test</td>
      <td> <a href=./notes/Q103045800.md> Q103045800 </a></td>
    </tr>
    <tr>
      <th>237</th>
      <td>Overview of the Alliance for Cellular Signaling.</td>
      <td> <a href=./notes/Q34164191.md> Q34164191 </a></td>
    </tr>
    <tr>
      <th>238</th>
      <td>CellFinder: a cell data repository</td>
      <td> <a href=./notes/Q28660708.md> Q28660708 </a></td>
    </tr>
    <tr>
      <th>239</th>
      <td>HeLa, a new microbial species</td>
      <td> <a href=./notes/Q104591553.md> Q104591553 </a></td>
    </tr>
    <tr>
      <th>240</th>
      <td>A survey on annotation tools for the biomedical literature.</td>
      <td> <a href=./notes/Q38068801.md> Q38068801 </a></td>
    </tr>
    <tr>
      <th>241</th>
      <td>Keeping raw data in context.</td>
      <td> <a href=./notes/Q43589284.md> Q43589284 </a></td>
    </tr>
    <tr>
      <th>242</th>
      <td>What makes a gene name? Named entity recognition in the biomedical literature.</td>
      <td> <a href=./notes/Q51955678.md> Q51955678 </a></td>
    </tr>
    <tr>
      <th>243</th>
      <td>Zika virus: designate standardized names</td>
      <td> <a href=./notes/Q24247056.md> Q24247056 </a></td>
    </tr>
    <tr>
      <th>244</th>
      <td>Representing Specified Values in OWL: "value partitions" and "value sets"</td>
      <td> <a href=./notes/Q104601506.md> Q104601506 </a></td>
    </tr>
    <tr>
      <th>245</th>
      <td>CELDA -- an ontology for the comprehensive representation of cells in complex systems</td>
      <td> <a href=./notes/Q21284308.md> Q21284308 </a></td>
    </tr>
    <tr>
      <th>246</th>
      <td>Using ontologies to study cell transitions</td>
      <td> <a href=./notes/Q34026802.md> Q34026802 </a></td>
    </tr>
    <tr>
      <th>247</th>
      <td>Genetic identification of brain cell types underlying schizophrenia</td>
      <td> <a href=./notes/Q58462175.md> Q58462175 </a></td>
    </tr>
    <tr>
      <th>248</th>
      <td>Q104626424</td>
      <td> <a href=./notes/Q104626424.md> Q104626424 </a></td>
    </tr>
    <tr>
      <th>249</th>
      <td>Uniform resolution of compact identifiers for biomedical data.</td>
      <td> <a href=./notes/Q55053685.md> Q55053685 </a></td>
    </tr>
    <tr>
      <th>250</th>
      <td>Semantically enabling a genome-wide association study database</td>
      <td> <a href=./notes/Q28707570.md> Q28707570 </a></td>
    </tr>
    <tr>
      <th>251</th>
      <td>The Monarch Initiative in 2019: an integrative data and analytic platform connecting phenotypes to genotypes across species</td>
      <td> <a href=./notes/Q91179320.md> Q91179320 </a></td>
    </tr>
    <tr>
      <th>252</th>
      <td>A Survey of Bioinformatics Database and Software Usage through Mining the Literature</td>
      <td> <a href=./notes/Q28600864.md> Q28600864 </a></td>
    </tr>
    <tr>
      <th>253</th>
      <td>LifeMap Discovery™: the embryonic development, stem cells, and regenerative medicine research portal</td>
      <td> <a href=./notes/Q34846475.md> Q34846475 </a></td>
    </tr>
    <tr>
      <th>254</th>
      <td>Distinction of Neurons, Glia and Endothelial Cells in the Cerebral Cortex: An Algorithm Based on Cytological Features</td>
      <td> <a href=./notes/Q36193595.md> Q36193595 </a></td>
    </tr>
    <tr>
      <th>255</th>
      <td>The Molecular Pages of the mesotelencephalic dopamine consortium (DopaNet)</td>
      <td> <a href=./notes/Q24802073.md> Q24802073 </a></td>
    </tr>
    <tr>
      <th>256</th>
      <td>SC2disease: a manually curated database of single-cell transcriptome for human diseases</td>
      <td> <a href=./notes/Q100429893.md> Q100429893 </a></td>
    </tr>
    <tr>
      <th>257</th>
      <td>AliBaba: PubMed as a graph.</td>
      <td> <a href=./notes/Q43433584.md> Q43433584 </a></td>
    </tr>
    <tr>
      <th>258</th>
      <td>The language of gene ontology: a Zipf's law analysis</td>
      <td> <a href=./notes/Q28268340.md> Q28268340 </a></td>
    </tr>
    <tr>
      <th>259</th>
      <td>Applying Ontology Design Patterns in Bio-ontologies</td>
      <td> <a href=./notes/Q57002925.md> Q57002925 </a></td>
    </tr>
    <tr>
      <th>260</th>
      <td>The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells.</td>
      <td> <a href=./notes/Q41861103.md> Q41861103 </a></td>
    </tr>
    <tr>
      <th>261</th>
      <td>A maximum-likelihood approach for building cell-type trees by lifting</td>
      <td> <a href=./notes/Q35906597.md> Q35906597 </a></td>
    </tr>
    <tr>
      <th>262</th>
      <td>Cell-type phylogenetics and the origin of endometrial stromal cells.</td>
      <td> <a href=./notes/Q38904507.md> Q38904507 </a></td>
    </tr>
    <tr>
      <th>263</th>
      <td>Implications of Epigenetic Variability within a Cell Population for "Cell Type" Classification</td>
      <td> <a href=./notes/Q26770736.md> Q26770736 </a></td>
    </tr>
    <tr>
      <th>264</th>
      <td>Decentralized provenance-aware publishing with nanopublications</td>
      <td> <a href=./notes/Q57339063.md> Q57339063 </a></td>
    </tr>
    <tr>
      <th>265</th>
      <td>Trusty URIs: Verifiable, Immutable, and Permanent Digital Artifacts for Linked Data</td>
      <td> <a href=./notes/Q56915510.md> Q56915510 </a></td>
    </tr>
    <tr>
      <th>266</th>
      <td>Using the brainbow to trace lineages in early mouse development</td>
      <td> <a href=./notes/Q58836377.md> Q58836377 </a></td>
    </tr>
    <tr>
      <th>267</th>
      <td>A transcriptome database for astrocytes, neurons, and oligodendrocytes: a new resource for understanding brain development and function</td>
      <td> <a href=./notes/Q29547320.md> Q29547320 </a></td>
    </tr>
    <tr>
      <th>268</th>
      <td>The value of data</td>
      <td> <a href=./notes/Q22676713.md> Q22676713 </a></td>
    </tr>
    <tr>
      <th>269</th>
      <td>Knowledge.Bio: A Web application for exploring, building and sharing webs of biomedical relationships mined from PubMed</td>
      <td> <a href=./notes/Q57831016.md> Q57831016 </a></td>
    </tr>
    <tr>
      <th>270</th>
      <td>histoCAT: analysis of cell phenotypes and interactions in multiplex image cytometry data.</td>
      <td> <a href=./notes/Q38637165.md> Q38637165 </a></td>
    </tr>
    <tr>
      <th>271</th>
      <td>What is a glial cell?</td>
      <td> <a href=./notes/Q46388366.md> Q46388366 </a></td>
    </tr>
    <tr>
      <th>272</th>
      <td>Div-Seq: Single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons.</td>
      <td> <a href=./notes/Q41068482.md> Q41068482 </a></td>
    </tr>
    <tr>
      <th>273</th>
      <td>Geometry of the Gene Expression Space of Individual Cells</td>
      <td> <a href=./notes/Q35688096.md> Q35688096 </a></td>
    </tr>
    <tr>
      <th>274</th>
      <td>A Survey and Classification of Controlled Natural Languages</td>
      <td> <a href=./notes/Q57402167.md> Q57402167 </a></td>
    </tr>
    <tr>
      <th>275</th>
      <td>Fish oil, Raynaud's syndrome, and undiscovered public knowledge</td>
      <td> <a href=./notes/Q69703977.md> Q69703977 </a></td>
    </tr>
    <tr>
      <th>276</th>
      <td>Effective mapping of biomedical text to the UMLS Metathesaurus: the MetaMap program</td>
      <td> <a href=./notes/Q29614363.md> Q29614363 </a></td>
    </tr>
    <tr>
      <th>277</th>
      <td>BioGPS and MyGene.info: organizing online, gene-centric information</td>
      <td> <a href=./notes/Q27575818.md> Q27575818 </a></td>
    </tr>
    <tr>
      <th>278</th>
      <td>Accurate estimation of cell-type composition from gene expression data</td>
      <td> <a href=./notes/Q91665384.md> Q91665384 </a></td>
    </tr>
    <tr>
      <th>279</th>
      <td>Robust enumeration of cell subsets from tissue expression profiles</td>
      <td> <a href=./notes/Q36539395.md> Q36539395 </a></td>
    </tr>
    <tr>
      <th>280</th>
      <td>Determining cell type abundance and expression from bulk tissues with digital cytometry</td>
      <td> <a href=./notes/Q64109527.md> Q64109527 </a></td>
    </tr>
    <tr>
      <th>281</th>
      <td>Evolution of Cellular Differentiation: From Hypotheses to Models</td>
      <td> <a href=./notes/Q98633613.md> Q98633613 </a></td>
    </tr>
    <tr>
      <th>282</th>
      <td>Utilizing the Wikidata system to improve the quality of medical content in Wikipedia in diverse languages: a pilot study</td>
      <td> <a href=./notes/Q21503276.md> Q21503276 </a></td>
    </tr>
    <tr>
      <th>283</th>
      <td>Cell Identity Codes: Understanding Cell Identity from Gene Expression Profiles using Deep Neural Networks</td>
      <td> <a href=./notes/Q64263662.md> Q64263662 </a></td>
    </tr>
    <tr>
      <th>284</th>
      <td>Genome-wide atlas of gene expression in the adult mouse brain</td>
      <td> <a href=./notes/Q26253934.md> Q26253934 </a></td>
    </tr>
    <tr>
      <th>285</th>
      <td>Group formation, relatedness, and the evolution of multicellularity</td>
      <td> <a href=./notes/Q86916694.md> Q86916694 </a></td>
    </tr>
    <tr>
      <th>286</th>
      <td>GiniClust2: a cluster-aware, weighted ensemble clustering method for cell-type detection.</td>
      <td> <a href=./notes/Q53748361.md> Q53748361 </a></td>
    </tr>
    <tr>
      <th>287</th>
      <td>Situating Wikipedia as a health information resource in various contexts: A scoping review</td>
      <td> <a href=./notes/Q85632863.md> Q85632863 </a></td>
    </tr>
    <tr>
      <th>288</th>
      <td>Crowdsourced assessment of common genetic contribution to predicting anti-TNF treatment response in rheumatoid arthritis.</td>
      <td> <a href=./notes/Q37201814.md> Q37201814 </a></td>
    </tr>
    <tr>
      <th>289</th>
      <td>An RDF/OWL knowledge base for query answering and decision support in clinical pharmacogenetics.</td>
      <td> <a href=./notes/Q38420432.md> Q38420432 </a></td>
    </tr>
    <tr>
      <th>290</th>
      <td>Introduction to Random Boolean Networks</td>
      <td> <a href=./notes/Q104815188.md> Q104815188 </a></td>
    </tr>
    <tr>
      <th>291</th>
      <td>A global network of biomedical relationships derived from text.</td>
      <td> <a href=./notes/Q52681328.md> Q52681328 </a></td>
    </tr>
    <tr>
      <th>292</th>
      <td>Literature Based Discovery: models, methods, and trends.</td>
      <td> <a href=./notes/Q38371706.md> Q38371706 </a></td>
    </tr>
    <tr>
      <th>293</th>
      <td>Identification of region-specific astrocyte subtypes at single cell resolution</td>
      <td> <a href=./notes/Q90091126.md> Q90091126 </a></td>
    </tr>
    <tr>
      <th>294</th>
      <td>Single-cell profiling identifies myeloid cell subsets with distinct fates during neuroinflammation</td>
      <td> <a href=./notes/Q91210236.md> Q91210236 </a></td>
    </tr>
    <tr>
      <th>295</th>
      <td>BioGPS: an extensible and customizable portal for querying and organizing gene annotation resources.</td>
      <td> <a href=./notes/Q21183895.md> Q21183895 </a></td>
    </tr>
    <tr>
      <th>296</th>
      <td>Literature-based discovery of new candidates for drug repurposing.</td>
      <td> <a href=./notes/Q38817541.md> Q38817541 </a></td>
    </tr>
    <tr>
      <th>297</th>
      <td>GiniClust: detecting rare cell types from single-cell gene expression data with Gini index.</td>
      <td> <a href=./notes/Q31111856.md> Q31111856 </a></td>
    </tr>
    <tr>
      <th>298</th>
      <td>Evolutionary trade-offs, Pareto optimality, and the geometry of phenotype space.</td>
      <td> <a href=./notes/Q54337677.md> Q54337677 </a></td>
    </tr>
    <tr>
      <th>299</th>
      <td>CA1 pyramidal cells have diverse biophysical properties, affected by development, experience, and aging</td>
      <td> <a href=./notes/Q41694511.md> Q41694511 </a></td>
    </tr>
    <tr>
      <th>300</th>
      <td>Single-cell messenger RNA sequencing reveals rare intestinal cell types.</td>
      <td> <a href=./notes/Q34490100.md> Q34490100 </a></td>
    </tr>
    <tr>
      <th>301</th>
      <td>Toward an epistemology of Wikipedia</td>
      <td> <a href=./notes/Q101955295.md> Q101955295 </a></td>
    </tr>
    <tr>
      <th>302</th>
      <td>Online tools to support literature-based discovery in the life sciences.</td>
      <td> <a href=./notes/Q36280460.md> Q36280460 </a></td>
    </tr>
    <tr>
      <th>303</th>
      <td>The distribution of cellular turnover in the human body</td>
      <td> <a href=./notes/Q104873201.md> Q104873201 </a></td>
    </tr>
    <tr>
      <th>304</th>
      <td>Revised Estimates for the Number of Human and Bacteria Cells in the Body</td>
      <td> <a href=./notes/Q34537872.md> Q34537872 </a></td>
    </tr>
    <tr>
      <th>305</th>
      <td>Literature mining for the biologist: from information retrieval to biological discovery</td>
      <td> <a href=./notes/Q30002370.md> Q30002370 </a></td>
    </tr>
    <tr>
      <th>306</th>
      <td>Overview of BioCreAtIvE: critical assessment of information extraction for biology</td>
      <td> <a href=./notes/Q21093643.md> Q21093643 </a></td>
    </tr>
    <tr>
      <th>307</th>
      <td>What is the total number of protein molecules per cell volume? A call to rethink some published values</td>
      <td> <a href=./notes/Q26782007.md> Q26782007 </a></td>
    </tr>
    <tr>
      <th>308</th>
      <td>Are We Really Vastly Outnumbered? Revisiting the Ratio of Bacterial to Host Cells in Humans.</td>
      <td> <a href=./notes/Q34511374.md> Q34511374 </a></td>
    </tr>
    <tr>
      <th>309</th>
      <td>UniProt: the universal protein knowledgebase in 2021</td>
      <td> <a href=./notes/Q102383737.md> Q102383737 </a></td>
    </tr>
    <tr>
      <th>310</th>
      <td>Evaluation of text data mining for database curation: lessons learned from the KDD Challenge Cup.</td>
      <td> <a href=./notes/Q30811199.md> Q30811199 </a></td>
    </tr>
    <tr>
      <th>311</th>
      <td>Are Platelets Cells? And if Yes, are They Immune Cells?</td>
      <td> <a href=./notes/Q28083403.md> Q28083403 </a></td>
    </tr>
    <tr>
      <th>312</th>
      <td>Extracellular vesicles or exosomes? On primacy, precision, and popularity influencing a choice of nomenclature</td>
      <td> <a href=./notes/Q93135668.md> Q93135668 </a></td>
    </tr>
    <tr>
      <th>313</th>
      <td>Q104904652</td>
      <td> <a href=./notes/Q104904652.md> Q104904652 </a></td>
    </tr>
    <tr>
      <th>314</th>
      <td>Rutabaga by any other name: extracting biological names.</td>
      <td> <a href=./notes/Q42599537.md> Q42599537 </a></td>
    </tr>
    <tr>
      <th>315</th>
      <td>Body cell mass: model development and validation at the cellular level of body composition.</td>
      <td> <a href=./notes/Q47579411.md> Q47579411 </a></td>
    </tr>
    <tr>
      <th>316</th>
      <td>Mitochondrial DNA molecules and virtual number of mitochondria per cell in mammalian cells.</td>
      <td> <a href=./notes/Q34168557.md> Q34168557 </a></td>
    </tr>
    <tr>
      <th>317</th>
      <td>An interactive system for finding complementary literatures: a stimulus to scientific discovery</td>
      <td> <a href=./notes/Q57372877.md> Q57372877 </a></td>
    </tr>
    <tr>
      <th>318</th>
      <td>Exploiting semantic relations for literature-based discovery.</td>
      <td> <a href=./notes/Q35725188.md> Q35725188 </a></td>
    </tr>
    <tr>
      <th>319</th>
      <td>The quantified cell</td>
      <td> <a href=./notes/Q34504200.md> Q34504200 </a></td>
    </tr>
    <tr>
      <th>320</th>
      <td>SnapShot: key numbers in biology.</td>
      <td> <a href=./notes/Q34124340.md> Q34124340 </a></td>
    </tr>
    <tr>
      <th>321</th>
      <td>BioNumbers--the database of key numbers in molecular and cell biology</td>
      <td> <a href=./notes/Q24643881.md> Q24643881 </a></td>
    </tr>
    <tr>
      <th>322</th>
      <td>One Knowledge Graph to Rule Them All? Analyzing the Differences Between DBpedia, YAGO, Wikidata &amp; co</td>
      <td> <a href=./notes/Q41078402.md> Q41078402 </a></td>
    </tr>
    <tr>
      <th>323</th>
      <td>An era of single-cell genomics consortia</td>
      <td> <a href=./notes/Q99418649.md> Q99418649 </a></td>
    </tr>
    <tr>
      <th>324</th>
      <td>The Human Cell Atlas: Technical approaches and challenges.</td>
      <td> <a href=./notes/Q48563763.md> Q48563763 </a></td>
    </tr>
    <tr>
      <th>325</th>
      <td>The automation of science.</td>
      <td> <a href=./notes/Q27937312.md> Q27937312 </a></td>
    </tr>
    <tr>
      <th>326</th>
      <td>Representation of probabilistic scientific knowledge.</td>
      <td> <a href=./notes/Q36786073.md> Q36786073 </a></td>
    </tr>
    <tr>
      <th>327</th>
      <td>An atlas of active enhancers across human cell types and tissues.</td>
      <td> <a href=./notes/Q34660689.md> Q34660689 </a></td>
    </tr>
    <tr>
      <th>328</th>
      <td>A promoter-level mammalian expression atlas</td>
      <td> <a href=./notes/Q34412203.md> Q34412203 </a></td>
    </tr>
    <tr>
      <th>329</th>
      <td>Cell type-specific genetic regulation of gene expression across human tissues</td>
      <td> <a href=./notes/Q99349840.md> Q99349840 </a></td>
    </tr>
    <tr>
      <th>330</th>
      <td>xCell: digitally portraying the tissue cellular heterogeneity landscape.</td>
      <td> <a href=./notes/Q46377913.md> Q46377913 </a></td>
    </tr>
    <tr>
      <th>331</th>
      <td>Identifying cell types to interpret scRNA-seq data: how, why and more possibilities</td>
      <td> <a href=./notes/Q90784336.md> Q90784336 </a></td>
    </tr>
    <tr>
      <th>332</th>
      <td>Models of global gene expression define major domains of cell type and tissue identity</td>
      <td> <a href=./notes/Q33558314.md> Q33558314 </a></td>
    </tr>
    <tr>
      <th>333</th>
      <td>Building a high-quality Human Cell Atlas</td>
      <td> <a href=./notes/Q105080655.md> Q105080655 </a></td>
    </tr>
    <tr>
      <th>334</th>
      <td>scMatch: a single-cell gene expression profile annotation tool using reference datasets</td>
      <td> <a href=./notes/Q91557912.md> Q91557912 </a></td>
    </tr>
    <tr>
      <th>335</th>
      <td>A protocol for adding knowledge to Wikidata: aligning resources on human coronaviruses</td>
      <td> <a href=./notes/Q105037759.md> Q105037759 </a></td>
    </tr>
    <tr>
      <th>336</th>
      <td>Automating drug discovery.</td>
      <td> <a href=./notes/Q47780833.md> Q47780833 </a></td>
    </tr>
    <tr>
      <th>337</th>
      <td>Conceptions of Good Science in Our Data-Rich World.</td>
      <td> <a href=./notes/Q55214531.md> Q55214531 </a></td>
    </tr>
    <tr>
      <th>338</th>
      <td>On the formalization and reuse of scientific research.</td>
      <td> <a href=./notes/Q34178056.md> Q34178056 </a></td>
    </tr>
    <tr>
      <th>339</th>
      <td>A subcellular map of the human proteome.</td>
      <td> <a href=./notes/Q48155823.md> Q48155823 </a></td>
    </tr>
    <tr>
      <th>340</th>
      <td>Representation, simulation, and hypothesis generation in graph and logical models of biological networks.</td>
      <td> <a href=./notes/Q37921686.md> Q37921686 </a></td>
    </tr>
    <tr>
      <th>341</th>
      <td>The EXACT description of biomedical protocols.</td>
      <td> <a href=./notes/Q38513299.md> Q38513299 </a></td>
    </tr>
    <tr>
      <th>342</th>
      <td>The Bari Manifesto: An interoperability framework for essential biodiversity variables</td>
      <td> <a href=./notes/Q60250756.md> Q60250756 </a></td>
    </tr>
    <tr>
      <th>343</th>
      <td>The Philosophy of Logical Atomism, Lecture 1: Facts and Propositions</td>
      <td> <a href=./notes/Q105105637.md> Q105105637 </a></td>
    </tr>
    <tr>
      <th>344</th>
      <td>Transcriptomes of the major human pancreatic cell types.</td>
      <td> <a href=./notes/Q37430849.md> Q37430849 </a></td>
    </tr>
    <tr>
      <th>345</th>
      <td>Variability and memory of protein levels in human cells.</td>
      <td> <a href=./notes/Q40205119.md> Q40205119 </a></td>
    </tr>
    <tr>
      <th>346</th>
      <td>scConsensus: combining supervised and unsupervised clustering for cell type identification in single-cell RNA sequencing data</td>
      <td> <a href=./notes/Q104450617.md> Q104450617 </a></td>
    </tr>
    <tr>
      <th>347</th>
      <td>Study of cell differentiation by phylogenetic analysis using histone modification data</td>
      <td> <a href=./notes/Q30841360.md> Q30841360 </a></td>
    </tr>
    <tr>
      <th>348</th>
      <td>Evaluation of text-mining systems for biology: overview of the Second BioCreative community challenge.</td>
      <td> <a href=./notes/Q34844839.md> Q34844839 </a></td>
    </tr>
    <tr>
      <th>349</th>
      <td>Q</td>
      <td> <a href=./notes/Q.md> Q </a></td>
    </tr>
    <tr>
      <th>350</th>
      <td>Topic pages: PLoS Computational Biology meets Wikipedia</td>
      <td> <a href=./notes/Q21092570.md> Q21092570 </a></td>
    </tr>
    <tr>
      <th>351</th>
      <td>Submit a Topic Page to PLOS Computational Biology and Wikipedia</td>
      <td> <a href=./notes/Q54655231.md> Q54655231 </a></td>
    </tr>
    <tr>
      <th>352</th>
      <td>Genomic and molecular control of cell type and cell type conversions.</td>
      <td> <a href=./notes/Q49436822.md> Q49436822 </a></td>
    </tr>
    <tr>
      <th>353</th>
      <td>Inferring cell type innovations by phylogenetic methods-concepts, methods, and limitations.</td>
      <td> <a href=./notes/Q40436539.md> Q40436539 </a></td>
    </tr>
    <tr>
      <th>354</th>
      <td>From Freebase to Wikidata: The Great Migration</td>
      <td> <a href=./notes/Q24074986.md> Q24074986 </a></td>
    </tr>
    <tr>
      <th>355</th>
      <td>What Makes a Good Collaborative Knowledge Graph: Group Composition and Quality in Wikidata</td>
      <td> <a href=./notes/Q38918394.md> Q38918394 </a></td>
    </tr>
    <tr>
      <th>356</th>
      <td>Single-cell sequencing-based technologies will revolutionize whole-organism science.</td>
      <td> <a href=./notes/Q34360540.md> Q34360540 </a></td>
    </tr>
    <tr>
      <th>357</th>
      <td>Model-based understanding of single-cell CRISPR screening</td>
      <td> <a href=./notes/Q64052031.md> Q64052031 </a></td>
    </tr>
    <tr>
      <th>358</th>
      <td>Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage</td>
      <td> <a href=./notes/Q91045816.md> Q91045816 </a></td>
    </tr>
    <tr>
      <th>359</th>
      <td>Strategic reading, ontologies, and the future of scientific publishing.</td>
      <td> <a href=./notes/Q33493284.md> Q33493284 </a></td>
    </tr>
    <tr>
      <th>360</th>
      <td>Geospatial data and Scholia</td>
      <td> <a href=./notes/Q50813856.md> Q50813856 </a></td>
    </tr>
    <tr>
      <th>361</th>
      <td>SuperCT: a supervised-learning framework for enhanced characterization of single-cell transcriptomic profiles</td>
      <td> <a href=./notes/Q64106229.md> Q64106229 </a></td>
    </tr>
    <tr>
      <th>362</th>
      <td>Principles of Cell Circuits for Tissue Repair and Fibrosis</td>
      <td> <a href=./notes/Q89717434.md> Q89717434 </a></td>
    </tr>
    <tr>
      <th>363</th>
      <td>WikiGenomes: an open Web application for community consumption and curation of gene annotation data in Wikidata</td>
      <td> <a href=./notes/Q28529449.md> Q28529449 </a></td>
    </tr>
    <tr>
      <th>364</th>
      <td>Towards Robot Scientists for autonomous scientific discovery.</td>
      <td> <a href=./notes/Q21198755.md> Q21198755 </a></td>
    </tr>
    <tr>
      <th>365</th>
      <td>Single-cell RNA sequencing reveals the heterogeneity of liver-resident immune cells in human</td>
      <td> <a href=./notes/Q94464890.md> Q94464890 </a></td>
    </tr>
    <tr>
      <th>366</th>
      <td>Cell Atlas of The Human Fovea and Peripheral Retina</td>
      <td> <a href=./notes/Q96576585.md> Q96576585 </a></td>
    </tr>
    <tr>
      <th>367</th>
      <td>Wikidata: A Free Collaborative Knowledgebase</td>
      <td> <a href=./notes/Q18507561.md> Q18507561 </a></td>
    </tr>
    <tr>
      <th>368</th>
      <td>Biotea: semantics for Pubmed Central.</td>
      <td> <a href=./notes/Q47191940.md> Q47191940 </a></td>
    </tr>
    <tr>
      <th>369</th>
      <td>Making Linked Data SPARQL with the InterMine Biological Data Warehouse</td>
      <td> <a href=./notes/Q58881147.md> Q58881147 </a></td>
    </tr>
    <tr>
      <th>370</th>
      <td>Cell-type deconvolution from DNA methylation: a review of recent applications.</td>
      <td> <a href=./notes/Q47661868.md> Q47661868 </a></td>
    </tr>
    <tr>
      <th>371</th>
      <td>Cell-type-specific programs for activity-regulated gene expression</td>
      <td> <a href=./notes/Q90360171.md> Q90360171 </a></td>
    </tr>
    <tr>
      <th>372</th>
      <td>Learning for single-cell assignment</td>
      <td> <a href=./notes/Q101132241.md> Q101132241 </a></td>
    </tr>
    <tr>
      <th>373</th>
      <td>Droplet scRNA-seq is not zero-inflated</td>
      <td> <a href=./notes/Q92644135.md> Q92644135 </a></td>
    </tr>
    <tr>
      <th>374</th>
      <td>Circular RNA expression in human hematopoietic cells is widespread and cell-type specific</td>
      <td> <a href=./notes/Q58779756.md> Q58779756 </a></td>
    </tr>
    <tr>
      <th>375</th>
      <td>scReClassify: post hoc cell type classification of single-cell rNA-seq data</td>
      <td> <a href=./notes/Q92237112.md> Q92237112 </a></td>
    </tr>
    <tr>
      <th>376</th>
      <td>Microparadigms: chains of collective reasoning in publications about molecular interactions.</td>
      <td> <a href=./notes/Q34451887.md> Q34451887 </a></td>
    </tr>
    <tr>
      <th>377</th>
      <td>Seeking a new biology through text mining.</td>
      <td> <a href=./notes/Q42220103.md> Q42220103 </a></td>
    </tr>
    <tr>
      <th>378</th>
      <td>Extraction of relations between genes and diseases from text and large-scale data analysis: implications for translational research</td>
      <td> <a href=./notes/Q21146739.md> Q21146739 </a></td>
    </tr>
    <tr>
      <th>379</th>
      <td>TOAST: improving reference-free cell composition estimation by cross-cell type differential analysis</td>
      <td> <a href=./notes/Q93103927.md> Q93103927 </a></td>
    </tr>
    <tr>
      <th>380</th>
      <td>Cell-type-resolved alternative splicing patterns in mouse liver.</td>
      <td> <a href=./notes/Q50084595.md> Q50084595 </a></td>
    </tr>
    <tr>
      <th>381</th>
      <td>Modularisation of domain ontologies implemented in description logics and related formalisms including OWL</td>
      <td> <a href=./notes/Q29027414.md> Q29027414 </a></td>
    </tr>
    <tr>
      <th>382</th>
      <td>Categorization of services for seeking information in biomedical literature: a typology for improvement of practice.</td>
      <td> <a href=./notes/Q40065967.md> Q40065967 </a></td>
    </tr>
    <tr>
      <th>383</th>
      <td>Scrutinizing SNOMED CT's Ability to Reconcile Clinical Language Ambiguities with an Ontology Representation</td>
      <td> <a href=./notes/Q95522960.md> Q95522960 </a></td>
    </tr>
    <tr>
      <th>384</th>
      <td>alona: a web server for single cell RNA-seq analysis</td>
      <td> <a href=./notes/Q93179879.md> Q93179879 </a></td>
    </tr>
    <tr>
      <th>385</th>
      <td>The selection and function of cell type-specific enhancers</td>
      <td> <a href=./notes/Q28081960.md> Q28081960 </a></td>
    </tr>
    <tr>
      <th>386</th>
      <td>Scale and context: issues in ontologies to link health- and bio-informatics.</td>
      <td> <a href=./notes/Q38431134.md> Q38431134 </a></td>
    </tr>
    <tr>
      <th>387</th>
      <td>Using the wisdom of the crowds to find critical errors in biomedical ontologies: a study of SNOMED CT.</td>
      <td> <a href=./notes/Q38262429.md> Q38262429 </a></td>
    </tr>
    <tr>
      <th>388</th>
      <td>scSLAM-seq reveals core features of transcription dynamics in single cells</td>
      <td> <a href=./notes/Q91776830.md> Q91776830 </a></td>
    </tr>
    <tr>
      <th>389</th>
      <td>Slide-seq: A scalable technology for measuring genome-wide expression at high spatial resolution</td>
      <td> <a href=./notes/Q92687649.md> Q92687649 </a></td>
    </tr>
    <tr>
      <th>390</th>
      <td>A review of the semantic web field</td>
      <td> <a href=./notes/Q105079960.md> Q105079960 </a></td>
    </tr>
    <tr>
      <th>391</th>
      <td>Towards a Systematic Approach to Sync Factual Data across Wikipedia, Wikidata and External Data Sources</td>
      <td> <a href=./notes/Q105428190.md> Q105428190 </a></td>
    </tr>
    <tr>
      <th>392</th>
      <td>Discovering pathway and cell-type signatures in transcriptomic compendia with machine learning</td>
      <td> <a href=./notes/Q57897434.md> Q57897434 </a></td>
    </tr>
    <tr>
      <th>393</th>
      <td>Computational methods for the integrative analysis of single-cell data</td>
      <td> <a href=./notes/Q94503935.md> Q94503935 </a></td>
    </tr>
    <tr>
      <th>394</th>
      <td>DBpedia FlexiFusion the Best of Wikipedia > Wikidata > Your Data</td>
      <td> <a href=./notes/Q87813149.md> Q87813149 </a></td>
    </tr>
    <tr>
      <th>395</th>
      <td>OSCAR4: a flexible architecture for chemical text-mining</td>
      <td> <a href=./notes/Q27062470.md> Q27062470 </a></td>
    </tr>
    <tr>
      <th>396</th>
      <td>Moana: A robust and scalable cell type classification framework for single-cell RNA-Seq data</td>
      <td> <a href=./notes/Q105394114.md> Q105394114 </a></td>
    </tr>
    <tr>
      <th>397</th>
      <td>Bias, robustness and scalability in single-cell differential expression analysis</td>
      <td> <a href=./notes/Q56879991.md> Q56879991 </a></td>
    </tr>
    <tr>
      <th>398</th>
      <td>Cell-type deconvolution in epigenome-wide association studies: a review and recommendations.</td>
      <td> <a href=./notes/Q39314677.md> Q39314677 </a></td>
    </tr>
    <tr>
      <th>399</th>
      <td>Text processing through Web services: calling Whatizit.</td>
      <td> <a href=./notes/Q38514993.md> Q38514993 </a></td>
    </tr>
    <tr>
      <th>400</th>
      <td>Why do it the hard way? The case for an expressive description logic for SNOMED</td>
      <td> <a href=./notes/Q28292343.md> Q28292343 </a></td>
    </tr>
    <tr>
      <th>401</th>
      <td>Comparison of concept recognizers for building the Open Biomedical Annotator.</td>
      <td> <a href=./notes/Q33504367.md> Q33504367 </a></td>
    </tr>
    <tr>
      <th>402</th>
      <td>Wikidata through the eyes of DBpedia</td>
      <td> <a href=./notes/Q87813191.md> Q87813191 </a></td>
    </tr>
    <tr>
      <th>403</th>
      <td>Wikidata through the Eyes of DBpedia</td>
      <td> <a href=./notes/Q27036443.md> Q27036443 </a></td>
    </tr>
    <tr>
      <th>404</th>
      <td>A comprehensive, cell specific microRNA catalogue of human peripheral blood.</td>
      <td> <a href=./notes/Q47678818.md> Q47678818 </a></td>
    </tr>
    <tr>
      <th>405</th>
      <td>CNS microRNA profiles: a database for cell type enriched microRNA expression across the mouse central nervous system</td>
      <td> <a href=./notes/Q90438547.md> Q90438547 </a></td>
    </tr>
    <tr>
      <th>406</th>
      <td>Regulation of cell-type-specific transcriptomes by microRNA networks during human brain development</td>
      <td> <a href=./notes/Q93223327.md> Q93223327 </a></td>
    </tr>
    <tr>
      <th>407</th>
      <td>[What is a pancreatic stem cell?]</td>
      <td> <a href=./notes/Q73867340.md> Q73867340 </a></td>
    </tr>
    <tr>
      <th>408</th>
      <td>BioPortal: ontologies and integrated data resources at the click of a mouse</td>
      <td> <a href=./notes/Q29617519.md> Q29617519 </a></td>
    </tr>
    <tr>
      <th>409</th>
      <td>Identification of SARS-CoV-2 Inhibitors using Lung and Colonic Organoids</td>
      <td> <a href=./notes/Q101051356.md> Q101051356 </a></td>
    </tr>
    <tr>
      <th>410</th>
      <td>Decoding myofibroblast origins in human kidney fibrosis</td>
      <td> <a href=./notes/Q101569793.md> Q101569793 </a></td>
    </tr>
    <tr>
      <th>411</th>
      <td>Distinct hypothalamic control of same- and opposite-sex mounting behaviour in mice</td>
      <td> <a href=./notes/Q103836863.md> Q103836863 </a></td>
    </tr>
    <tr>
      <th>412</th>
      <td>Machine learning for rediscovering revolutionary ideas of the past</td>
      <td> <a href=./notes/Q105302822.md> Q105302822 </a></td>
    </tr>
    <tr>
      <th>413</th>
      <td>Scientific intuition inspired by machine learning generated hypotheses</td>
      <td> <a href=./notes/Q105516701.md> Q105516701 </a></td>
    </tr>
    <tr>
      <th>414</th>
      <td>Tacit Knowledge, Trust and the Q of Sapphire</td>
      <td> <a href=./notes/Q105516720.md> Q105516720 </a></td>
    </tr>
    <tr>
      <th>415</th>
      <td>Towards Operational Abduction from a Cognitive Perspective</td>
      <td> <a href=./notes/Q105516744.md> Q105516744 </a></td>
    </tr>
    <tr>
      <th>416</th>
      <td>Undiscovered Public Knowledge</td>
      <td> <a href=./notes/Q105516839.md> Q105516839 </a></td>
    </tr>
    <tr>
      <th>417</th>
      <td>Neurotoxic microglia promote TDP-43 proteinopathy in progranulin deficiency</td>
      <td> <a href=./notes/Q99634613.md> Q99634613 </a></td>
    </tr>
    <tr>
      <th>418</th>
      <td>Cells of the adult human heart</td>
      <td> <a href=./notes/Q99708572.md> Q99708572 </a></td>
    </tr>
    <tr>
      <th>419</th>
      <td>Systems biological assessment of immunity to mild versus severe COVID-19 infection in humans</td>
      <td> <a href=./notes/Q98394258.md> Q98394258 </a></td>
    </tr>
    <tr>
      <th>420</th>
      <td>Simultaneous epitope and transcriptome measurement in single cells.</td>
      <td> <a href=./notes/Q42595570.md> Q42595570 </a></td>
    </tr>
    <tr>
      <th>421</th>
      <td>Google Scholar to overshadow them all? Comparing the sizes of 12 academic search engines and bibliographic databases</td>
      <td> <a href=./notes/Q58898255.md> Q58898255 </a></td>
    </tr>
    <tr>
      <th>422</th>
      <td>Academic Search Engine Optimization (ASEO)</td>
      <td> <a href=./notes/Q56506009.md> Q56506009 </a></td>
    </tr>
    <tr>
      <th>423</th>
      <td>Thalia: Semantic search engine for biomedical abstracts</td>
      <td> <a href=./notes/Q57452795.md> Q57452795 </a></td>
    </tr>
    <tr>
      <th>424</th>
      <td>GoWeb: A Semantic Search Engine for the Life Science Web</td>
      <td> <a href=./notes/Q58914760.md> Q58914760 </a></td>
    </tr>
    <tr>
      <th>425</th>
      <td>BioSearch: a semantic search engine for Bio2RDF.</td>
      <td> <a href=./notes/Q47268720.md> Q47268720 </a></td>
    </tr>
    <tr>
      <th>426</th>
      <td>Mapping anatomical related entities to human body parts based on wikipedia in discharge summaries</td>
      <td> <a href=./notes/Q87818346.md> Q87818346 </a></td>
    </tr>
    <tr>
      <th>427</th>
      <td>Hypothesis, analysis and synthesis, it's all Greek to me</td>
      <td> <a href=./notes/Q64261830.md> Q64261830 </a></td>
    </tr>
    <tr>
      <th>428</th>
      <td>Preliminary evaluation of the CellFinder literature curation pipeline for gene expression in kidney cells and anatomical parts.</td>
      <td> <a href=./notes/Q36776925.md> Q36776925 </a></td>
    </tr>
    <tr>
      <th>429</th>
      <td>Ontology based molecular signatures for immune cell types via gene expression analysis</td>
      <td> <a href=./notes/Q34978215.md> Q34978215 </a></td>
    </tr>
    <tr>
      <th>430</th>
      <td>Identification of research hypotheses and new knowledge from scientific literature.</td>
      <td> <a href=./notes/Q55440752.md> Q55440752 </a></td>
    </tr>
    <tr>
      <th>431</th>
      <td>Enriching a biomedical event corpus with meta-knowledge annotation.</td>
      <td> <a href=./notes/Q31033181.md> Q31033181 </a></td>
    </tr>
    <tr>
      <th>432</th>
      <td>SciLite: a platform for displaying text-mined annotations as a means to link research articles with biological data.</td>
      <td> <a href=./notes/Q41695166.md> Q41695166 </a></td>
    </tr>
    <tr>
      <th>433</th>
      <td>EXTRACT: interactive extraction of environment metadata and term suggestion for metagenomic sample annotation</td>
      <td> <a href=./notes/Q29169641.md> Q29169641 </a></td>
    </tr>
    <tr>
      <th>434</th>
      <td>In defence of classification</td>
      <td> <a href=./notes/Q57945379.md> Q57945379 </a></td>
    </tr>
    <tr>
      <th>435</th>
      <td>Regulatory logic of neuronal diversity: terminal selector genes and selector motifs.</td>
      <td> <a href=./notes/Q37068535.md> Q37068535 </a></td>
    </tr>
    <tr>
      <th>436</th>
      <td>Molecular taxonomy of major neuronal classes in the adult mouse forebrain</td>
      <td> <a href=./notes/Q28288068.md> Q28288068 </a></td>
    </tr>
    <tr>
      <th>437</th>
      <td>Europe PMC in 2017.</td>
      <td> <a href=./notes/Q47121112.md> Q47121112 </a></td>
    </tr>
    <tr>
      <th>438</th>
      <td>Reflect: A practical approach to web semantics</td>
      <td> <a href=./notes/Q30002409.md> Q30002409 </a></td>
    </tr>
    <tr>
      <th>439</th>
      <td>Aggregating transcript-level analyses for single-cell differential gene expression</td>
      <td> <a href=./notes/Q96127495.md> Q96127495 </a></td>
    </tr>
    <tr>
      <th>440</th>
      <td>Single-cell landscape of bronchoalveolar immune cells in patients with COVID-19</td>
      <td> <a href=./notes/Q94600823.md> Q94600823 </a></td>
    </tr>
    <tr>
      <th>441</th>
      <td>EXTRACT 2.0: text-mining-assisted interactive annotation of biomedical named entities and ontology terms</td>
      <td> <a href=./notes/Q30002333.md> Q30002333 </a></td>
    </tr>
    <tr>
      <th>442</th>
      <td>biochem4j: Integrated and extensible biochemical knowledge through graph databases.</td>
      <td> <a href=./notes/Q38678861.md> Q38678861 </a></td>
    </tr>
    <tr>
      <th>443</th>
      <td>Comprehensive Integration of Single-Cell Data</td>
      <td> <a href=./notes/Q92614138.md> Q92614138 </a></td>
    </tr>
    <tr>
      <th>444</th>
      <td>Text mining and its potential applications in systems biology.</td>
      <td> <a href=./notes/Q36625511.md> Q36625511 </a></td>
    </tr>
    <tr>
      <th>445</th>
      <td>Improving reference prioritisation with PICO recognition</td>
      <td> <a href=./notes/Q91728644.md> Q91728644 </a></td>
    </tr>
    <tr>
      <th>446</th>
      <td>Toward systematic review automation: a practical guide to using machine learning tools in research synthesis</td>
      <td> <a href=./notes/Q91804991.md> Q91804991 </a></td>
    </tr>
    <tr>
      <th>447</th>
      <td>Genes and homology in nervous system evolution: comparing gene functions, expression patterns, and cell type molecular fingerprints.</td>
      <td> <a href=./notes/Q36625916.md> Q36625916 </a></td>
    </tr>
    <tr>
      <th>448</th>
      <td>Character trees from transcriptome data: Origin and individuation of morphological characters and the so-called "species signal".</td>
      <td> <a href=./notes/Q30980445.md> Q30980445 </a></td>
    </tr>
    <tr>
      <th>449</th>
      <td>Master transcription factors determine cell-type-specific responses to TGF-β signaling</td>
      <td> <a href=./notes/Q28586183.md> Q28586183 </a></td>
    </tr>
    <tr>
      <th>450</th>
      <td>Regulatory Logic of Pan-Neuronal Gene Expression in C. elegans.</td>
      <td> <a href=./notes/Q35984619.md> Q35984619 </a></td>
    </tr>
    <tr>
      <th>451</th>
      <td>Menagerie: A text-mining tool to support animal-human translation in neurodegeneration research</td>
      <td> <a href=./notes/Q92043520.md> Q92043520 </a></td>
    </tr>
    <tr>
      <th>452</th>
      <td>Automatic annotation of protein residues in published papers</td>
      <td> <a href=./notes/Q91186622.md> Q91186622 </a></td>
    </tr>
    <tr>
      <th>453</th>
      <td>Cell Types Promoting Goosebumps Form a Niche to Regulate Hair Follicle Stem Cells</td>
      <td> <a href=./notes/Q97557008.md> Q97557008 </a></td>
    </tr>
    <tr>
      <th>454</th>
      <td>High-Dimensional Single-Cell Mapping of Central Nervous System Immune Cells Reveals Distinct Myeloid Subsets in Health, Aging, and Disease.</td>
      <td> <a href=./notes/Q51760951.md> Q51760951 </a></td>
    </tr>
    <tr>
      <th>455</th>
      <td>High-Dimensional Single-Cell Mapping of Central Nervous System Immune Cells Reveals Distinct Myeloid Subsets in Health, Aging, and Disease.</td>
      <td> <a href=./notes/Q50092192.md> Q50092192 </a></td>
    </tr>
    <tr>
      <th>456</th>
      <td>The Zebrafish Information Network: new support for non-coding genes, richer Gene Ontology annotations and the Alliance of Genome Resources</td>
      <td> <a href=./notes/Q58587083.md> Q58587083 </a></td>
    </tr>
    <tr>
      <th>457</th>
      <td>The Sequence Ontology: a tool for the unification of genome annotations</td>
      <td> <a href=./notes/Q21092875.md> Q21092875 </a></td>
    </tr>
    <tr>
      <th>458</th>
      <td>Parts of Animals</td>
      <td> <a href=./notes/Q1180725.md> Q1180725 </a></td>
    </tr>
    <tr>
      <th>459</th>
      <td>History of Animals</td>
      <td> <a href=./notes/Q1620799.md> Q1620799 </a></td>
    </tr>
    <tr>
      <th>460</th>
      <td>Perspectives on defining cell types in the brain</td>
      <td> <a href=./notes/Q90361677.md> Q90361677 </a></td>
    </tr>
    <tr>
      <th>461</th>
      <td>Tools and Concepts for Interrogating and Defining Cellular Identity</td>
      <td> <a href=./notes/Q94585630.md> Q94585630 </a></td>
    </tr>
    <tr>
      <th>462</th>
      <td>A method for increasing expressivity of Gene Ontology annotations using a compositional approach.</td>
      <td> <a href=./notes/Q33690250.md> Q33690250 </a></td>
    </tr>
    <tr>
      <th>463</th>
      <td>Funding knowledgebases: Towards a sustainable funding model for the UniProt use case.</td>
      <td> <a href=./notes/Q47732789.md> Q47732789 </a></td>
    </tr>
    <tr>
      <th>464</th>
      <td>A single-cell atlas of the airway epithelium reveals the CFTR-rich pulmonary ionocyte</td>
      <td> <a href=./notes/Q57318689.md> Q57318689 </a></td>
    </tr>
    <tr>
      <th>465</th>
      <td>Measuring the quality of scientific references in Wikipedia: an analysis of more than 115M citations to over 800,000 scientific articles</td>
      <td> <a href=./notes/Q100766127.md> Q100766127 </a></td>
    </tr>
    <tr>
      <th>466</th>
      <td>Allergic inflammatory memory in human respiratory epithelial progenitor cells</td>
      <td> <a href=./notes/Q59050242.md> Q59050242 </a></td>
    </tr>
    <tr>
      <th>467</th>
      <td>Big data: Wikiomics</td>
      <td> <a href=./notes/Q28292893.md> Q28292893 </a></td>
    </tr>
    <tr>
      <th>468</th>
      <td>BioDash: a Semantic Web dashboard for drug development.</td>
      <td> <a href=./notes/Q38402355.md> Q38402355 </a></td>
    </tr>
    <tr>
      <th>469</th>
      <td>Bioinformatics Data Skills</td>
      <td> <a href=./notes/Q105756183.md> Q105756183 </a></td>
    </tr>
    <tr>
      <th>470</th>
      <td>Neuronal cell types and connectivity: lessons from the retina</td>
      <td> <a href=./notes/Q27021497.md> Q27021497 </a></td>
    </tr>
    <tr>
      <th>471</th>
      <td>Classifying Drosophila Olfactory Projection Neuron Subtypes by Single-Cell RNA Sequencing.</td>
      <td> <a href=./notes/Q47391265.md> Q47391265 </a></td>
    </tr>
    <tr>
      <th>472</th>
      <td>Relations in biomedical ontologies</td>
      <td> <a href=./notes/Q19671539.md> Q19671539 </a></td>
    </tr>
    <tr>
      <th>473</th>
      <td>Ensembles, dynamics, and cell types: Revisiting the statistical mechanics perspective on cellular regulation</td>
      <td> <a href=./notes/Q91316993.md> Q91316993 </a></td>
    </tr>
    <tr>
      <th>474</th>
      <td>A Taxonomy of Part-Whole Relations</td>
      <td> <a href=./notes/Q54248976.md> Q54248976 </a></td>
    </tr>
    <tr>
      <th>475</th>
      <td>Determining Relative Dynamic Stability of Cell States Using Boolean Network Model</td>
      <td> <a href=./notes/Q58787036.md> Q58787036 </a></td>
    </tr>
    <tr>
      <th>476</th>
      <td>A guide to best practices for Gene Ontology (GO) manual annotation.</td>
      <td> <a href=./notes/Q36997786.md> Q36997786 </a></td>
    </tr>
    <tr>
      <th>477</th>
      <td>Cross-product extensions of the Gene Ontology.</td>
      <td> <a href=./notes/Q34020884.md> Q34020884 </a></td>
    </tr>
    <tr>
      <th>478</th>
      <td>KG-COVID-19: a framework to produce customized knowledge graphs for COVID-19 response</td>
      <td> <a href=./notes/Q102043840.md> Q102043840 </a></td>
    </tr>
    <tr>
      <th>479</th>
      <td>Classification, Ontology, and Precision Medicine</td>
      <td> <a href=./notes/Q57281359.md> Q57281359 </a></td>
    </tr>
    <tr>
      <th>480</th>
      <td>Supervised clustering for single-cell analysis</td>
      <td> <a href=./notes/Q90003298.md> Q90003298 </a></td>
    </tr>
    <tr>
      <th>481</th>
      <td>Supervised classification enables rapid annotation of cell atlases</td>
      <td> <a href=./notes/Q90003307.md> Q90003307 </a></td>
    </tr>
    <tr>
      <th>482</th>
      <td>Probabilistic cell-type assignment of single-cell RNA-seq for tumor microenvironment profiling</td>
      <td> <a href=./notes/Q90003401.md> Q90003401 </a></td>
    </tr>
    <tr>
      <th>483</th>
      <td>Q392754: Criação de perfis académicos com Wikidata e Scholia – projecto-piloto</td>
      <td> <a href=./notes/Q79377124.md> Q79377124 </a></td>
    </tr>
    <tr>
      <th>484</th>
      <td>The emergence of Semantic Systems Biology.</td>
      <td> <a href=./notes/Q38060960.md> Q38060960 </a></td>
    </tr>
    <tr>
      <th>485</th>
      <td>RDFScape: Semantic Web meets systems biology.</td>
      <td> <a href=./notes/Q33331997.md> Q33331997 </a></td>
    </tr>
    <tr>
      <th>486</th>
      <td>The application of the open pharmacological concepts triple store (Open PHACTS) to support drug discovery research</td>
      <td> <a href=./notes/Q19845588.md> Q19845588 </a></td>
    </tr>
    <tr>
      <th>487</th>
      <td>Representing and querying disease networks using graph databases</td>
      <td> <a href=./notes/Q26741378.md> Q26741378 </a></td>
    </tr>
    <tr>
      <th>488</th>
      <td>Cell-type-based analysis of microRNA profiles in the mouse brain.</td>
      <td> <a href=./notes/Q35725749.md> Q35725749 </a></td>
    </tr>
    <tr>
      <th>489</th>
      <td>A flagellate-to-amoeboid switch in the closest living relatives of animals</td>
      <td> <a href=./notes/Q105870433.md> Q105870433 </a></td>
    </tr>
    <tr>
      <th>490</th>
      <td>A behind the scenes tour of the IEDB curation process: an optimized process empirically integrating automation and human curation efforts</td>
      <td> <a href=./notes/Q96947357.md> Q96947357 </a></td>
    </tr>
    <tr>
      <th>491</th>
      <td>Manual curation is not sufficient for annotation of genomic databases</td>
      <td> <a href=./notes/Q34655326.md> Q34655326 </a></td>
    </tr>
    <tr>
      <th>492</th>
      <td>Evolution of cellular diversity in primary motor cortex of human, marmoset monkey, and mouse</td>
      <td> <a href=./notes/Q105870478.md> Q105870478 </a></td>
    </tr>
    <tr>
      <th>493</th>
      <td>A taxonomy of transcriptomic cell types across the isocortex and hippocampal formation</td>
      <td> <a href=./notes/Q105870479.md> Q105870479 </a></td>
    </tr>
    <tr>
      <th>494</th>
      <td>Capturing scientific knowledge in computable form</td>
      <td> <a href=./notes/Q105969725.md> Q105969725 </a></td>
    </tr>
    <tr>
      <th>495</th>
      <td>Text mining meets community curation: a newly designed curation platform to improve author experience and participation at WormBase</td>
      <td> <a href=./notes/Q90415774.md> Q90415774 </a></td>
    </tr>
    <tr>
      <th>496</th>
      <td>Replacement of an Essentialistic Perspective on Taxonomic Definitions as Exemplified by the Definition of "Mammalia"</td>
      <td> <a href=./notes/Q105870481.md> Q105870481 </a></td>
    </tr>
    <tr>
      <th>497</th>
      <td>Metabolic stability and epigenesis in randomly constructed genetic nets</td>
      <td> <a href=./notes/Q28256189.md> Q28256189 </a></td>
    </tr>
    <tr>
      <th>498</th>
      <td>Biodynamic ontology: applying BFO in the biomedical domain.</td>
      <td> <a href=./notes/Q38520561.md> Q38520561 </a></td>
    </tr>
    <tr>
      <th>499</th>
      <td>Tecnologias da web semântica para a recuperação da informação no wikidata</td>
      <td> <a href=./notes/Q97090036.md> Q97090036 </a></td>
    </tr>
    <tr>
      <th>500</th>
      <td>Q106114584</td>
      <td> <a href=./notes/Q106114584.md> Q106114584 </a></td>
    </tr>
    <tr>
      <th>501</th>
      <td>Biomedical articles share annotations with their citation neighbors</td>
      <td> <a href=./notes/Q105870538.md> Q105870538 </a></td>
    </tr>
    <tr>
      <th>502</th>
      <td>WikiUMLS: Aligning UMLS to Wikipedia via Cross-lingual Neural Ranking</td>
      <td> <a href=./notes/Q105870539.md> Q105870539 </a></td>
    </tr>
    <tr>
      <th>503</th>
      <td>ABDUCTIVE REASONING: A CONTRIBUTION TO KNOWLEDGE CREATION IN EDUCATION</td>
      <td> <a href=./notes/Q105870540.md> Q105870540 </a></td>
    </tr>
    <tr>
      <th>504</th>
      <td>Comunicação entre sistemas de informação médicos</td>
      <td> <a href=./notes/Q106121070.md> Q106121070 </a></td>
    </tr>
    <tr>
      <th>505</th>
      <td>Single-cell proteo-genomic reference maps of the hematopoietic system enable the purification and massive profiling of precisely defined cell states</td>
      <td> <a href=./notes/Q106143534.md> Q106143534 </a></td>
    </tr>
    <tr>
      <th>506</th>
      <td>DisGeNET-RDF: harnessing the innovative power of the Semantic Web to explore the genetic basis of diseases</td>
      <td> <a href=./notes/Q57002691.md> Q57002691 </a></td>
    </tr>
    <tr>
      <th>507</th>
      <td>The markup is the model: reasoning about systems biology models in the Semantic Web era</td>
      <td> <a href=./notes/Q28259568.md> Q28259568 </a></td>
    </tr>
    <tr>
      <th>508</th>
      <td>Automated cell-type classification in intact tissues by single-cell molecular profiling.</td>
      <td> <a href=./notes/Q48262342.md> Q48262342 </a></td>
    </tr>
    <tr>
      <th>509</th>
      <td>Single-cell expression profiling reveals dynamic flux of cardiac stromal, vascular and immune cells in health and injury</td>
      <td> <a href=./notes/Q64117530.md> Q64117530 </a></td>
    </tr>
    <tr>
      <th>510</th>
      <td>SBOL-OWL: An Ontological Approach for Formal and Semantic Representation of Synthetic Biology Information</td>
      <td> <a href=./notes/Q91791013.md> Q91791013 </a></td>
    </tr>
    <tr>
      <th>511</th>
      <td>SynBioHub: A Standards-Enabled Design Repository for Synthetic Biology.</td>
      <td> <a href=./notes/Q47566455.md> Q47566455 </a></td>
    </tr>
    <tr>
      <th>512</th>
      <td>Identification of an inhibitory neuron subtype, the L-stellate cell of the cochlear nucleus</td>
      <td> <a href=./notes/Q101212121.md> Q101212121 </a></td>
    </tr>
    <tr>
      <th>513</th>
      <td>Single cell transcriptional signatures of the human placenta in term and preterm parturition</td>
      <td> <a href=./notes/Q83224730.md> Q83224730 </a></td>
    </tr>
    <tr>
      <th>514</th>
      <td>The SBOL Stack: A Platform for Storing, Publishing, and Sharing Synthetic Biology Designs.</td>
      <td> <a href=./notes/Q36042745.md> Q36042745 </a></td>
    </tr>
    <tr>
      <th>515</th>
      <td>The biocurator: connecting and enhancing scientific data</td>
      <td> <a href=./notes/Q21092581.md> Q21092581 </a></td>
    </tr>
    <tr>
      <th>516</th>
      <td>EKLF/KLF1 expression defines a unique macrophage subset during mouse erythropoiesis</td>
      <td> <a href=./notes/Q106234628.md> Q106234628 </a></td>
    </tr>
    <tr>
      <th>517</th>
      <td>Decoding human fetal liver haematopoiesis</td>
      <td> <a href=./notes/Q90617724.md> Q90617724 </a></td>
    </tr>
    <tr>
      <th>518</th>
      <td>Community annotation and bioinformatics workforce development in concert--Little Skate Genome Annotation Workshops and Jamborees.</td>
      <td> <a href=./notes/Q30427789.md> Q30427789 </a></td>
    </tr>
    <tr>
      <th>519</th>
      <td>BioCreative-IV virtual issue.</td>
      <td> <a href=./notes/Q42584832.md> Q42584832 </a></td>
    </tr>
    <tr>
      <th>520</th>
      <td>Why data citation isn't working, and what to do about it</td>
      <td> <a href=./notes/Q94522308.md> Q94522308 </a></td>
    </tr>
    <tr>
      <th>521</th>
      <td>On expert curation and scalability: UniProtKB/Swiss-Prot as a case study.</td>
      <td> <a href=./notes/Q47313936.md> Q47313936 </a></td>
    </tr>
    <tr>
      <th>522</th>
      <td>Community next steps for making globally unique identifiers work for biocollections data</td>
      <td> <a href=./notes/Q22676264.md> Q22676264 </a></td>
    </tr>
    <tr>
      <th>523</th>
      <td>Q106248504</td>
      <td> <a href=./notes/Q106248504.md> Q106248504 </a></td>
    </tr>
    <tr>
      <th>524</th>
      <td>Single-cell transcriptomic evidence for dense intracortical neuropeptide networks</td>
      <td> <a href=./notes/Q83226392.md> Q83226392 </a></td>
    </tr>
    <tr>
      <th>525</th>
      <td>Painel de informação sobre a COVID-19: consultas SPARQL na Wikidata</td>
      <td> <a href=./notes/Q106249454.md> Q106249454 </a></td>
    </tr>
    <tr>
      <th>526</th>
      <td>FAIR and open multilingual clinical trials in Wikidata and Wikipedia</td>
      <td> <a href=./notes/Q106235601.md> Q106235601 </a></td>
    </tr>
    <tr>
      <th>527</th>
      <td>Introducing meta-services for biomedical information extraction</td>
      <td> <a href=./notes/Q36920197.md> Q36920197 </a></td>
    </tr>
    <tr>
      <th>528</th>
      <td>Improving Text Mining with Controlled Natural Language: A Case Study for Protein Interactions</td>
      <td> <a href=./notes/Q57402195.md> Q57402195 </a></td>
    </tr>
    <tr>
      <th>529</th>
      <td>An atlas of cell types in the mammalian epididymis and vas deferens</td>
      <td> <a href=./notes/Q98166912.md> Q98166912 </a></td>
    </tr>
    <tr>
      <th>530</th>
      <td>Spatiotemporal analysis of human intestinal development at single-cell resolution</td>
      <td> <a href=./notes/Q104754414.md> Q104754414 </a></td>
    </tr>
    <tr>
      <th>531</th>
      <td>On the role of knowledge graphs in explainable AI</td>
      <td> <a href=./notes/Q98792615.md> Q98792615 </a></td>
    </tr>
    <tr>
      <th>532</th>
      <td>The impact of focused Gene Ontology curation of specific mammalian systems</td>
      <td> <a href=./notes/Q34102856.md> Q34102856 </a></td>
    </tr>
    <tr>
      <th>533</th>
      <td>Single cell RNA sequencing of human liver reveals distinct intrahepatic macrophage populations</td>
      <td> <a href=./notes/Q57804774.md> Q57804774 </a></td>
    </tr>
    <tr>
      <th>534</th>
      <td>An overview of the intrathymic intricacies of T cell development</td>
      <td> <a href=./notes/Q26862512.md> Q26862512 </a></td>
    </tr>
    <tr>
      <th>535</th>
      <td>Use of Gene Ontology Annotation to understand the peroxisome proteome in humans</td>
      <td> <a href=./notes/Q36543290.md> Q36543290 </a></td>
    </tr>
    <tr>
      <th>536</th>
      <td>Editorial: biological ontologies and semantic biology</td>
      <td> <a href=./notes/Q38435922.md> Q38435922 </a></td>
    </tr>
    <tr>
      <th>537</th>
      <td>Semantic systems biology</td>
      <td> <a href=./notes/Q57002913.md> Q57002913 </a></td>
    </tr>
    <tr>
      <th>538</th>
      <td>Fate mapping analysis reveals a novel murine dermal migratory Langerhans-like cell population</td>
      <td> <a href=./notes/Q106354969.md> Q106354969 </a></td>
    </tr>
    <tr>
      <th>539</th>
      <td>Specification of diverse cell types during early neurogenesis of the mouse cerebellum</td>
      <td> <a href=./notes/Q64261872.md> Q64261872 </a></td>
    </tr>
    <tr>
      <th>540</th>
      <td>A cell atlas of human thymic development defines T cell repertoire formation</td>
      <td> <a href=./notes/Q89808480.md> Q89808480 </a></td>
    </tr>
    <tr>
      <th>541</th>
      <td>Single-cell transcriptomics of the mouse kidney reveals potential cellular targets of kidney disease.</td>
      <td> <a href=./notes/Q52603808.md> Q52603808 </a></td>
    </tr>
    <tr>
      <th>542</th>
      <td>Molecular, spatial and functional single-cell profiling of the hypothalamic preoptic region</td>
      <td> <a href=./notes/Q58577965.md> Q58577965 </a></td>
    </tr>
    <tr>
      <th>543</th>
      <td>Semantic Systems Biology: Formal Knowledge Representation in Systems Biology for Model Construction, Retrieval, Validation and Discovery</td>
      <td> <a href=./notes/Q56915551.md> Q56915551 </a></td>
    </tr>
    <tr>
      <th>544</th>
      <td>Querying biomedical Linked Data with natural language questions</td>
      <td> <a href=./notes/Q105870677.md> Q105870677 </a></td>
    </tr>
    <tr>
      <th>545</th>
      <td>Q106369750</td>
      <td> <a href=./notes/Q106369750.md> Q106369750 </a></td>
    </tr>
    <tr>
      <th>546</th>
      <td>A competency question-oriented approach for the transformation of semi-structured bioinformatics data into linked open data</td>
      <td> <a href=./notes/Q105870679.md> Q105870679 </a></td>
    </tr>
    <tr>
      <th>547</th>
      <td>Ontologies for the life sciences</td>
      <td> <a href=./notes/Q105870680.md> Q105870680 </a></td>
    </tr>
    <tr>
      <th>548</th>
      <td>Single-Cell Transcriptomics Reveals that Differentiation and Spatial Signatures Shape Epidermal and Hair Follicle Heterogeneity.</td>
      <td> <a href=./notes/Q37314462.md> Q37314462 </a></td>
    </tr>
    <tr>
      <th>549</th>
      <td>Distinct developmental pathways from blood monocytes generate human lung macrophage diversity</td>
      <td> <a href=./notes/Q104684648.md> Q104684648 </a></td>
    </tr>
    <tr>
      <th>550</th>
      <td>A single-cell survey of the small intestinal epithelium.</td>
      <td> <a href=./notes/Q47564808.md> Q47564808 </a></td>
    </tr>
    <tr>
      <th>551</th>
      <td>The Ontology for Parasite Lifecycle (OPL): towards a consistent vocabulary of lifecycle stages in parasitic organisms</td>
      <td> <a href=./notes/Q36368053.md> Q36368053 </a></td>
    </tr>
    <tr>
      <th>552</th>
      <td>A single-cell transcriptomic atlas characterizes ageing tissues in the mouse</td>
      <td> <a href=./notes/Q97569253.md> Q97569253 </a></td>
    </tr>
    <tr>
      <th>553</th>
      <td>On the Application of Formal Principles to Life Science Data: a Case Study in the Gene Ontology</td>
      <td> <a href=./notes/Q105870682.md> Q105870682 </a></td>
    </tr>
    <tr>
      <th>554</th>
      <td>Crowdsourcing biomedical research: leveraging communities as innovation engines</td>
      <td> <a href=./notes/Q27887654.md> Q27887654 </a></td>
    </tr>
    <tr>
      <th>555</th>
      <td>Textpresso: an ontology-based information retrieval and extraction system for biological literature</td>
      <td> <a href=./notes/Q24793009.md> Q24793009 </a></td>
    </tr>
    <tr>
      <th>556</th>
      <td>openSNP--a crowdsourced web resource for personal genomics</td>
      <td> <a href=./notes/Q28657448.md> Q28657448 </a></td>
    </tr>
    <tr>
      <th>557</th>
      <td>Much more than a mere technology: A systematic review of Wikidata in libraries</td>
      <td> <a href=./notes/Q105429450.md> Q105429450 </a></td>
    </tr>
    <tr>
      <th>558</th>
      <td>Open Natural Products Research: Curation and Dissemination of Biological Occurrences of Chemical Structures through Wikidata</td>
      <td> <a href=./notes/Q105742243.md> Q105742243 </a></td>
    </tr>
    <tr>
      <th>559</th>
      <td>Ten Simple Rules for Developing Public Biological Databases</td>
      <td> <a href=./notes/Q28595967.md> Q28595967 </a></td>
    </tr>
    <tr>
      <th>560</th>
      <td>Syntactic analyses and named entity recognition for PubMed and PubMed Central — up-to-the-minute</td>
      <td> <a href=./notes/Q105870783.md> Q105870783 </a></td>
    </tr>
    <tr>
      <th>561</th>
      <td>COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas</td>
      <td> <a href=./notes/Q106315467.md> Q106315467 </a></td>
    </tr>
    <tr>
      <th>562</th>
      <td>The Human Cell Atlas.</td>
      <td> <a href=./notes/Q46368626.md> Q46368626 </a></td>
    </tr>
    <tr>
      <th>563</th>
      <td>Molecular Architecture of the Mouse Nervous System</td>
      <td> <a href=./notes/Q58793343.md> Q58793343 </a></td>
    </tr>
    <tr>
      <th>564</th>
      <td>Two physically, functionally, and developmentally distinct peritoneal macrophage subsets.</td>
      <td> <a href=./notes/Q33664675.md> Q33664675 </a></td>
    </tr>
    <tr>
      <th>565</th>
      <td>BioBERT: a pre-trained biomedical language representation model for biomedical text mining</td>
      <td> <a href=./notes/Q90006632.md> Q90006632 </a></td>
    </tr>
    <tr>
      <th>566</th>
      <td>BioGPS: building your own mash-up of gene annotations and expression profiles.</td>
      <td> <a href=./notes/Q29169145.md> Q29169145 </a></td>
    </tr>
    <tr>
      <th>567</th>
      <td>Single-cell reconstruction of the early maternal–fetal interface in humans</td>
      <td> <a href=./notes/Q59068540.md> Q59068540 </a></td>
    </tr>
    <tr>
      <th>568</th>
      <td>High-performance web services for querying gene and variant annotation</td>
      <td> <a href=./notes/Q27943213.md> Q27943213 </a></td>
    </tr>
    <tr>
      <th>569</th>
      <td>Text mining resources for the life sciences.</td>
      <td> <a href=./notes/Q28076577.md> Q28076577 </a></td>
    </tr>
    <tr>
      <th>570</th>
      <td>Conserved cell types with divergent features in human versus mouse cortex</td>
      <td> <a href=./notes/Q71306466.md> Q71306466 </a></td>
    </tr>
    <tr>
      <th>571</th>
      <td>Systematic Comparison of High-throughput Single-Cell and Single-Nucleus Transcriptomes during Cardiomyocyte Differentiation</td>
      <td> <a href=./notes/Q89454854.md> Q89454854 </a></td>
    </tr>
    <tr>
      <th>572</th>
      <td>Distinct microbial and immune niches of the human colon</td>
      <td> <a href=./notes/Q89748559.md> Q89748559 </a></td>
    </tr>
    <tr>
      <th>573</th>
      <td>Cochlear hair cell regeneration after noise-induced hearing loss: Does regeneration follow development?</td>
      <td> <a href=./notes/Q39062528.md> Q39062528 </a></td>
    </tr>
    <tr>
      <th>574</th>
      <td>APICURON: a database to credit and acknowledge the work of biocurators</td>
      <td> <a href=./notes/Q106486134.md> Q106486134 </a></td>
    </tr>
    <tr>
      <th>575</th>
      <td>CIViC is a community knowledgebase for expert crowdsourcing the clinical interpretation of variants in cancer</td>
      <td> <a href=./notes/Q29178984.md> Q29178984 </a></td>
    </tr>
    <tr>
      <th>576</th>
      <td>Single cell dissection of plasma cell heterogeneity in symptomatic and asymptomatic myeloma</td>
      <td> <a href=./notes/Q90318029.md> Q90318029 </a></td>
    </tr>
    <tr>
      <th>577</th>
      <td>Single-Cell Transcriptomic Analysis of Human Lung Provides Insights into the Pathobiology of Pulmonary Fibrosis</td>
      <td> <a href=./notes/Q90565414.md> Q90565414 </a></td>
    </tr>
    <tr>
      <th>578</th>
      <td>Mining data and metadata from the gene expression omnibus</td>
      <td> <a href=./notes/Q64119088.md> Q64119088 </a></td>
    </tr>
    <tr>
      <th>579</th>
      <td>Sharing biological data: why, when, and how</td>
      <td> <a href=./notes/Q106498418.md> Q106498418 </a></td>
    </tr>
    <tr>
      <th>580</th>
      <td>Resolving the fibrotic niche of human liver cirrhosis at single-cell level</td>
      <td> <a href=./notes/Q90612750.md> Q90612750 </a></td>
    </tr>
    <tr>
      <th>581</th>
      <td>SARS-CoV-2 receptor ACE2 and TMPRSS2 are primarily expressed in bronchial transient secretory cells</td>
      <td> <a href=./notes/Q91561671.md> Q91561671 </a></td>
    </tr>
    <tr>
      <th>582</th>
      <td>Nuclei multiplexing with barcoded antibodies for single-nucleus genomics</td>
      <td> <a href=./notes/Q91564696.md> Q91564696 </a></td>
    </tr>
    <tr>
      <th>583</th>
      <td>A human liver cell atlas reveals heterogeneity and epithelial progenitors</td>
      <td> <a href=./notes/Q91776824.md> Q91776824 </a></td>
    </tr>
    <tr>
      <th>584</th>
      <td>Pressing needs of biomedical text mining in biocuration and beyond: opportunities and challenges</td>
      <td> <a href=./notes/Q36233671.md> Q36233671 </a></td>
    </tr>
    <tr>
      <th>585</th>
      <td>Integrating text mining into the MGI biocuration workflow</td>
      <td> <a href=./notes/Q33554526.md> Q33554526 </a></td>
    </tr>
    <tr>
      <th>586</th>
      <td>Accelerating literature curation with text-mining tools: a case study of using PubTator to curate genes in PubMed abstracts.</td>
      <td> <a href=./notes/Q34480402.md> Q34480402 </a></td>
    </tr>
    <tr>
      <th>587</th>
      <td>Collaborative biocuration--text-mining development task for document prioritization for curation</td>
      <td> <a href=./notes/Q36418100.md> Q36418100 </a></td>
    </tr>
    <tr>
      <th>588</th>
      <td>Discovering biomedical semantic relations in PubMed queries for information retrieval and database curation</td>
      <td> <a href=./notes/Q35970872.md> Q35970872 </a></td>
    </tr>
    <tr>
      <th>589</th>
      <td>TeamTat: a collaborative text annotation tool</td>
      <td> <a href=./notes/Q94561368.md> Q94561368 </a></td>
    </tr>
    <tr>
      <th>590</th>
      <td>Automated assessment of biological database assertions using the scientific literature.</td>
      <td> <a href=./notes/Q65002314.md> Q65002314 </a></td>
    </tr>
    <tr>
      <th>591</th>
      <td>LitSense: making sense of biomedical literature at sentence level</td>
      <td> <a href=./notes/Q90290716.md> Q90290716 </a></td>
    </tr>
    <tr>
      <th>592</th>
      <td>Semi-automated ontology generation within OBO-Edit</td>
      <td> <a href=./notes/Q28751720.md> Q28751720 </a></td>
    </tr>
    <tr>
      <th>593</th>
      <td>Toward a Common Coordinate Framework for the Human Body</td>
      <td> <a href=./notes/Q91960464.md> Q91960464 </a></td>
    </tr>
    <tr>
      <th>594</th>
      <td>scRNA-seq assessment of the human lung, spleen, and esophagus tissue stability after cold preservation</td>
      <td> <a href=./notes/Q92354852.md> Q92354852 </a></td>
    </tr>
    <tr>
      <th>595</th>
      <td>Characterization of cell fate probabilities in single-cell data with Palantir</td>
      <td> <a href=./notes/Q92539085.md> Q92539085 </a></td>
    </tr>
    <tr>
      <th>596</th>
      <td>A single-cell transcriptome atlas of the adult human retina</td>
      <td> <a href=./notes/Q92766837.md> Q92766837 </a></td>
    </tr>
    <tr>
      <th>597</th>
      <td>A cellular census of human lungs identifies novel cell states in health and in asthma</td>
      <td> <a href=./notes/Q92826478.md> Q92826478 </a></td>
    </tr>
    <tr>
      <th>598</th>
      <td>The Confidence Information Ontology: a step towards a standard for asserting confidence in annotations</td>
      <td> <a href=./notes/Q28647782.md> Q28647782 </a></td>
    </tr>
    <tr>
      <th>599</th>
      <td>Knowledge engineering tools for reasoning with scientific observations and interpretations: a neural connectivity use case.</td>
      <td> <a href=./notes/Q35221928.md> Q35221928 </a></td>
    </tr>
    <tr>
      <th>600</th>
      <td>A functional map for diverse forelimb actions within brainstem circuitry</td>
      <td> <a href=./notes/Q106534301.md> Q106534301 </a></td>
    </tr>
    <tr>
      <th>601</th>
      <td>Cell-type-specific asynchronous modulation of PKA by dopamine in learning</td>
      <td> <a href=./notes/Q106534414.md> Q106534414 </a></td>
    </tr>
    <tr>
      <th>602</th>
      <td>In situ mapping identifies distinct vascular niches for myelopoiesis</td>
      <td> <a href=./notes/Q106534530.md> Q106534530 </a></td>
    </tr>
    <tr>
      <th>603</th>
      <td>Gut-licensed IFNγ NK cells drive LAMP1TRAIL anti-inflammatory astrocytes</td>
      <td> <a href=./notes/Q106535191.md> Q106535191 </a></td>
    </tr>
    <tr>
      <th>604</th>
      <td>SARS-CoV-2 entry factors are highly expressed in nasal epithelial cells together with innate immune genes</td>
      <td> <a href=./notes/Q93196405.md> Q93196405 </a></td>
    </tr>
    <tr>
      <th>605</th>
      <td>Systematic comparison of single-cell and single-nucleus RNA-sequencing methods</td>
      <td> <a href=./notes/Q93270907.md> Q93270907 </a></td>
    </tr>
    <tr>
      <th>606</th>
      <td>MyMiner: a web application for computer-assisted biocuration and text annotation.</td>
      <td> <a href=./notes/Q47391302.md> Q47391302 </a></td>
    </tr>
    <tr>
      <th>607</th>
      <td>GEOMetaCuration: a web-based application for accurate manual curation of Gene Expression Omnibus metadata.</td>
      <td> <a href=./notes/Q55407135.md> Q55407135 </a></td>
    </tr>
    <tr>
      <th>608</th>
      <td>Accelerating annotation of articles via automated approaches: evaluation of the neXtA5 curation-support tool by neXtProt</td>
      <td> <a href=./notes/Q60302044.md> Q60302044 </a></td>
    </tr>
    <tr>
      <th>609</th>
      <td>BioCreative III interactive task: an overview</td>
      <td> <a href=./notes/Q34094645.md> Q34094645 </a></td>
    </tr>
    <tr>
      <th>610</th>
      <td>Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq</td>
      <td> <a href=./notes/Q37094024.md> Q37094024 </a></td>
    </tr>
    <tr>
      <th>611</th>
      <td>Single-cell analysis reveals new evolutionary complexity in uveal melanoma</td>
      <td> <a href=./notes/Q92932258.md> Q92932258 </a></td>
    </tr>
    <tr>
      <th>612</th>
      <td>Biocuration workflows and text mining: overview of the BioCreative 2012 Workshop Track II</td>
      <td> <a href=./notes/Q28711453.md> Q28711453 </a></td>
    </tr>
    <tr>
      <th>613</th>
      <td>Wikipedia citations: A comprehensive data set of citations with identifiers extracted from English Wikipedia</td>
      <td> <a href=./notes/Q106570089.md> Q106570089 </a></td>
    </tr>
    <tr>
      <th>614</th>
      <td>Why and how medical schools, peer-reviewed journals, and research funders should promote Wikipedia editing</td>
      <td> <a href=./notes/Q90535817.md> Q90535817 </a></td>
    </tr>
    <tr>
      <th>615</th>
      <td>Medical journals and Wikipedia: a global health matter.</td>
      <td> <a href=./notes/Q27480279.md> Q27480279 </a></td>
    </tr>
    <tr>
      <th>616</th>
      <td>A single-cell and single-nucleus RNA-Seq toolbox for fresh and frozen human tumors</td>
      <td> <a href=./notes/Q94671894.md> Q94671894 </a></td>
    </tr>
    <tr>
      <th>617</th>
      <td>SARS-CoV-2 Receptor ACE2 Is an Interferon-Stimulated Gene in Human Airway Epithelial Cells and Is Detected in Specific Cell Subsets across Tissues</td>
      <td> <a href=./notes/Q94938805.md> Q94938805 </a></td>
    </tr>
    <tr>
      <th>618</th>
      <td>An overview of the BioCreative 2012 Workshop Track III: interactive text mining task.</td>
      <td> <a href=./notes/Q36761979.md> Q36761979 </a></td>
    </tr>
    <tr>
      <th>619</th>
      <td>Overview of the interactive task in BioCreative V</td>
      <td> <a href=./notes/Q30002351.md> Q30002351 </a></td>
    </tr>
    <tr>
      <th>620</th>
      <td>MIFlowCyt: the minimum information about a Flow Cytometry Experiment</td>
      <td> <a href=./notes/Q24644748.md> Q24644748 </a></td>
    </tr>
    <tr>
      <th>621</th>
      <td>Promoting coherent minimum reporting guidelines for biological and biomedical investigations: the MIBBI project</td>
      <td> <a href=./notes/Q29301289.md> Q29301289 </a></td>
    </tr>
    <tr>
      <th>622</th>
      <td>Standardizing the standards</td>
      <td> <a href=./notes/Q95815991.md> Q95815991 </a></td>
    </tr>
    <tr>
      <th>623</th>
      <td>Cells and gene expression programs in the adult human heart</td>
      <td> <a href=./notes/Q95602396.md> Q95602396 </a></td>
    </tr>
    <tr>
      <th>624</th>
      <td>Integrated analyses of single-cell atlases reveal age, gender, and smoking status associations with cell type-specific expression of mediators of SARS-CoV-2 viral entry and highlights inflammatory programs in putative target cells</td>
      <td> <a href=./notes/Q95602580.md> Q95602580 </a></td>
    </tr>
    <tr>
      <th>625</th>
      <td>Single-cell atlas of a non-human primate reveals new pathogenic mechanisms of COVID-19</td>
      <td> <a href=./notes/Q95607654.md> Q95607654 </a></td>
    </tr>
    <tr>
      <th>626</th>
      <td>Benchmarking single-cell RNA-sequencing protocols for cell atlas projects</td>
      <td> <a href=./notes/Q96230190.md> Q96230190 </a></td>
    </tr>
    <tr>
      <th>627</th>
      <td>A Single-cell Atlas of the Human Healthy Airways</td>
      <td> <a href=./notes/Q98162195.md> Q98162195 </a></td>
    </tr>
    <tr>
      <th>628</th>
      <td>BioCreative V BioC track overview: collaborative biocurator assistant task for BioGRID.</td>
      <td> <a href=./notes/Q36121943.md> Q36121943 </a></td>
    </tr>
    <tr>
      <th>629</th>
      <td>The BEL information extraction workflow (BELIEF): evaluation in the BioCreative V BEL and IAT track.</td>
      <td> <a href=./notes/Q41265068.md> Q41265068 </a></td>
    </tr>
    <tr>
      <th>630</th>
      <td>STRING v11: protein–protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets</td>
      <td> <a href=./notes/Q60469724.md> Q60469724 </a></td>
    </tr>
    <tr>
      <th>631</th>
      <td>Unsupervised word embeddings capture latent knowledge from materials science literature</td>
      <td> <a href=./notes/Q91595456.md> Q91595456 </a></td>
    </tr>
    <tr>
      <th>632</th>
      <td>Artificial intelligence in neurodegenerative disease research: use of IBM Watson to identify additional RNA-binding proteins altered in amyotrophic lateral sclerosis.</td>
      <td> <a href=./notes/Q47406275.md> Q47406275 </a></td>
    </tr>
    <tr>
      <th>633</th>
      <td>The STRING database in 2021: customizable protein-protein networks, and functional characterization of user-uploaded gene/measurement sets</td>
      <td> <a href=./notes/Q102383784.md> Q102383784 </a></td>
    </tr>
    <tr>
      <th>634</th>
      <td>High throughput error corrected Nanopore single cell transcriptome sequencing</td>
      <td> <a href=./notes/Q98394753.md> Q98394753 </a></td>
    </tr>
    <tr>
      <th>635</th>
      <td>Myocyte Specific Upregulation of ACE2 in Cardiovascular Disease: Implications for SARS-CoV-2 Mediated Myocarditis</td>
      <td> <a href=./notes/Q98468904.md> Q98468904 </a></td>
    </tr>
    <tr>
      <th>636</th>
      <td>Crowdsourcing in biomedicine: challenges and opportunities</td>
      <td> <a href=./notes/Q19857267.md> Q19857267 </a></td>
    </tr>
    <tr>
      <th>637</th>
      <td>Crowdsourcing and curation: perspectives from biology and natural language processing</td>
      <td> <a href=./notes/Q28598176.md> Q28598176 </a></td>
    </tr>
    <tr>
      <th>638</th>
      <td>The Human and Mouse Enteric Nervous System at Single-Cell Resolution</td>
      <td> <a href=./notes/Q98952972.md> Q98952972 </a></td>
    </tr>
    <tr>
      <th>639</th>
      <td>The Human Cell Atlas and equity: lessons learned</td>
      <td> <a href=./notes/Q100491106.md> Q100491106 </a></td>
    </tr>
    <tr>
      <th>640</th>
      <td>SARS-CoV-2 receptor networks in diabetic and COVID-19 associated kidney disease</td>
      <td> <a href=./notes/Q100513994.md> Q100513994 </a></td>
    </tr>
    <tr>
      <th>641</th>
      <td>Wikidata and Libraries: Facilitating Open Knowledge</td>
      <td> <a href=./notes/Q52235186.md> Q52235186 </a></td>
    </tr>
    <tr>
      <th>642</th>
      <td>OpenTapioca: Lightweight Entity Linking for Wikidata</td>
      <td> <a href=./notes/Q63278360.md> Q63278360 </a></td>
    </tr>
    <tr>
      <th>643</th>
      <td>Single nucleus and in situ RNA sequencing reveals cell topographies in the human pancreas</td>
      <td> <a href=./notes/Q102206138.md> Q102206138 </a></td>
    </tr>
    <tr>
      <th>644</th>
      <td>Single-Cell Sequencing of Developing Human Gut Reveals Transcriptional Links to Childhood Crohn's Disease</td>
      <td> <a href=./notes/Q104130336.md> Q104130336 </a></td>
    </tr>
    <tr>
      <th>645</th>
      <td>The COVID-19 Ontology</td>
      <td> <a href=./notes/Q104513823.md> Q104513823 </a></td>
    </tr>
    <tr>
      <th>646</th>
      <td>How to describe a new fungal species</td>
      <td> <a href=./notes/Q28730060.md> Q28730060 </a></td>
    </tr>
    <tr>
      <th>647</th>
      <td>Uberon: towards a comprehensive multi-species anatomy ontology</td>
      <td> <a href=./notes/Q57006348.md> Q57006348 </a></td>
    </tr>
    <tr>
      <th>648</th>
      <td>TELOCYTES - a case of serendipity: the winding way from Interstitial Cells of Cajal (ICC), via Interstitial Cajal-Like Cells (ICLC) to TELOCYTES.</td>
      <td> <a href=./notes/Q42914236.md> Q42914236 </a></td>
    </tr>
    <tr>
      <th>649</th>
      <td>Gramene 2021: harnessing the power of comparative genomics and pathways for plant research</td>
      <td> <a href=./notes/Q101556384.md> Q101556384 </a></td>
    </tr>
    <tr>
      <th>650</th>
      <td>From ArrayExpress to BioStudies</td>
      <td> <a href=./notes/Q102205732.md> Q102205732 </a></td>
    </tr>
    <tr>
      <th>651</th>
      <td>Longitudinal Multi-omics Analyses Identify Responses of Megakaryocytes, Erythroid Cells, and Plasmablasts as Hallmarks of Severe COVID-19</td>
      <td> <a href=./notes/Q104139713.md> Q104139713 </a></td>
    </tr>
    <tr>
      <th>652</th>
      <td>The Organoid Cell Atlas: A Rosetta Stone for Biomedical Discovery and Regenerative Therapy</td>
      <td> <a href=./notes/Q104450592.md> Q104450592 </a></td>
    </tr>
    <tr>
      <th>653</th>
      <td>Tumor to normal single-cell mRNA comparisons reveal a pan-neuroblastoma cancer cell</td>
      <td> <a href=./notes/Q106642644.md> Q106642644 </a></td>
    </tr>
    <tr>
      <th>654</th>
      <td>The fasciacytes: A new cell devoted to fascial gliding regulation.</td>
      <td> <a href=./notes/Q52724376.md> Q52724376 </a></td>
    </tr>
    <tr>
      <th>655</th>
      <td>Openness and trust in data-intensive science: the case of biocuration</td>
      <td> <a href=./notes/Q96307475.md> Q96307475 </a></td>
    </tr>
    <tr>
      <th>656</th>
      <td>UniBioDicts: Unified access to biological dictionaries</td>
      <td> <a href=./notes/Q104582202.md> Q104582202 </a></td>
    </tr>
    <tr>
      <th>657</th>
      <td>IMGT <sup>® </sup> Biocuration and Comparative Analysis of  <i>Bos taurus </i> and  <i>Ovis aries </i> TRA/TRD Loci</td>
      <td> <a href=./notes/Q104619923.md> Q104619923 </a></td>
    </tr>
    <tr>
      <th>658</th>
      <td>Setting the basis of best practices and standards for curation and annotation of logical models in biology-highlights of the [BC]2 2019 CoLoMoTo/SysMod Workshop</td>
      <td> <a href=./notes/Q92104055.md> Q92104055 </a></td>
    </tr>
    <tr>
      <th>659</th>
      <td>Pancreatic ductal cells in development, regeneration, and neoplasia</td>
      <td> <a href=./notes/Q27006729.md> Q27006729 </a></td>
    </tr>
    <tr>
      <th>660</th>
      <td>Creating a Metabolic Syndrome Research Resource using the National Health and Nutrition Examination Survey</td>
      <td> <a href=./notes/Q104684400.md> Q104684400 </a></td>
    </tr>
    <tr>
      <th>661</th>
      <td>Quality Matters: Biocuration Experts on the Impact of Duplication and Other Data Quality Issues in Biological Databases</td>
      <td> <a href=./notes/Q97537095.md> Q97537095 </a></td>
    </tr>
    <tr>
      <th>662</th>
      <td>Retinoic acid synthesis by a population of choroidal stromal cells</td>
      <td> <a href=./notes/Q99604056.md> Q99604056 </a></td>
    </tr>
    <tr>
      <th>663</th>
      <td>The Functionally Unannotated Proteome of Human Male Tissues: A Shared Resource to Uncover New Protein Functions Associated with Reproductive Biology</td>
      <td> <a href=./notes/Q100676469.md> Q100676469 </a></td>
    </tr>
    <tr>
      <th>664</th>
      <td>CARD 2020: antibiotic resistome surveillance with the comprehensive antibiotic resistance database</td>
      <td> <a href=./notes/Q91006744.md> Q91006744 </a></td>
    </tr>
    <tr>
      <th>665</th>
      <td>Integrating image caption information into biomedical document classification in support of biocuration</td>
      <td> <a href=./notes/Q91956285.md> Q91956285 </a></td>
    </tr>
    <tr>
      <th>666</th>
      <td>Single-cell RNA-seq analysis of human coronary arteries using an enhanced workflow reveals SMC transitions and candidate drug targets</td>
      <td> <a href=./notes/Q106659158.md> Q106659158 </a></td>
    </tr>
    <tr>
      <th>667</th>
      <td>Atheroprotective roles of smooth muscle cell phenotypic modulation and the TCF21 disease gene as revealed by single-cell analysis</td>
      <td> <a href=./notes/Q92267490.md> Q92267490 </a></td>
    </tr>
    <tr>
      <th>668</th>
      <td>Deciphering cell-cell interactions and communication from gene expression</td>
      <td> <a href=./notes/Q101476452.md> Q101476452 </a></td>
    </tr>
    <tr>
      <th>669</th>
      <td>Besca, a single-cell transcriptomics analysis toolkit to accelerate translational research | bioRxiv</td>
      <td> <a href=./notes/Q104450593.md> Q104450593 </a></td>
    </tr>
    <tr>
      <th>670</th>
      <td>Plant Reactome: a knowledgebase and resource for comparative pathway analysis</td>
      <td> <a href=./notes/Q91078214.md> Q91078214 </a></td>
    </tr>
    <tr>
      <th>671</th>
      <td>Deep learning and alignment of spatially-resolved whole transcriptomes of single cells in the mouse brain with Tangram | bioRxiv</td>
      <td> <a href=./notes/Q104450595.md> Q104450595 </a></td>
    </tr>
    <tr>
      <th>672</th>
      <td>Delayed viral clearance and exacerbated airway hyperinflammation in hypertensive COVID-19 patients</td>
      <td> <a href=./notes/Q104450596.md> Q104450596 </a></td>
    </tr>
    <tr>
      <th>673</th>
      <td>Hypertension delays viral clearance and exacerbates airway hyperinflammation in patients with COVID-19</td>
      <td> <a href=./notes/Q106686666.md> Q106686666 </a></td>
    </tr>
    <tr>
      <th>674</th>
      <td>Using Reactome to build an autophagy mechanism knowledgebase</td>
      <td> <a href=./notes/Q96120967.md> Q96120967 </a></td>
    </tr>
    <tr>
      <th>675</th>
      <td>Involving community in genes and pathway curation</td>
      <td> <a href=./notes/Q61447691.md> Q61447691 </a></td>
    </tr>
    <tr>
      <th>676</th>
      <td>Open data, open curation</td>
      <td> <a href=./notes/Q56885165.md> Q56885165 </a></td>
    </tr>
    <tr>
      <th>677</th>
      <td>A mechanosensitive peri-arteriolar niche for osteogenesis and lymphopoiesis</td>
      <td> <a href=./notes/Q106692362.md> Q106692362 </a></td>
    </tr>
    <tr>
      <th>678</th>
      <td>DUBStepR: correlation-based feature selection for clustering single-cell RNA sequencing data</td>
      <td> <a href=./notes/Q104450599.md> Q104450599 </a></td>
    </tr>
    <tr>
      <th>679</th>
      <td>Ten quick tips for biocuration</td>
      <td> <a href=./notes/Q64071693.md> Q64071693 </a></td>
    </tr>
    <tr>
      <th>680</th>
      <td>Interpretable Clinical Genomics with a Likelihood Ratio Paradigm</td>
      <td> <a href=./notes/Q98208987.md> Q98208987 </a></td>
    </tr>
    <tr>
      <th>681</th>
      <td>Building deep learning models for evidence classification from the open access biomedical literature</td>
      <td> <a href=./notes/Q64084067.md> Q64084067 </a></td>
    </tr>
    <tr>
      <th>682</th>
      <td>Aristóteles e o estudo dos seres vivos</td>
      <td> <a href=./notes/Q106693249.md> Q106693249 </a></td>
    </tr>
    <tr>
      <th>683</th>
      <td>Applying citizen science to gene, drug and disease relationship extraction from biomedical abstracts</td>
      <td> <a href=./notes/Q90025338.md> Q90025338 </a></td>
    </tr>
    <tr>
      <th>684</th>
      <td>Consensus transcriptional regulatory networks of coronavirus-infected human cells</td>
      <td> <a href=./notes/Q99608377.md> Q99608377 </a></td>
    </tr>
    <tr>
      <th>685</th>
      <td>Worming into the Uncharacterized Human Proteome</td>
      <td> <a href=./notes/Q90091195.md> Q90091195 </a></td>
    </tr>
    <tr>
      <th>686</th>
      <td>IMGT® Biocuration and Comparative Study of the T Cell Receptor Beta Locus of Veterinary Species Based on Homo sapiens TRB</td>
      <td> <a href=./notes/Q95293653.md> Q95293653 </a></td>
    </tr>
    <tr>
      <th>687</th>
      <td>UPCLASS: a deep learning-based classifier for UniProtKB entry publications</td>
      <td> <a href=./notes/Q94522302.md> Q94522302 </a></td>
    </tr>
    <tr>
      <th>688</th>
      <td>Integrated single cell analysis of human lung fibrosis resolves cellular origins of predictive protein signatures in body fluids</td>
      <td> <a href=./notes/Q104450603.md> Q104450603 </a></td>
    </tr>
    <tr>
      <th>689</th>
      <td>Poised cell circuits in human skin are activated in disease</td>
      <td> <a href=./notes/Q104450608.md> Q104450608 </a></td>
    </tr>
    <tr>
      <th>690</th>
      <td>An autonomous debating system</td>
      <td> <a href=./notes/Q106097217.md> Q106097217 </a></td>
    </tr>
    <tr>
      <th>691</th>
      <td>Protein Data Bank: the single global archive for 3D macromolecular structure data</td>
      <td> <a href=./notes/Q57793805.md> Q57793805 </a></td>
    </tr>
    <tr>
      <th>692</th>
      <td>PubTator central: automated concept annotation for biomedical full text articles</td>
      <td> <a href=./notes/Q92202965.md> Q92202965 </a></td>
    </tr>
    <tr>
      <th>693</th>
      <td>Sampling artifacts in single-cell genomics cohort studies</td>
      <td> <a href=./notes/Q104450612.md> Q104450612 </a></td>
    </tr>
    <tr>
      <th>694</th>
      <td>Single cell profiling of immature human postnatal thymocytes resolves the complexity of intra-thymic lineage differentiation and thymus seeding precursors</td>
      <td> <a href=./notes/Q104450621.md> Q104450621 </a></td>
    </tr>
    <tr>
      <th>695</th>
      <td>Spatial and single-cell transcriptional landscape of human cerebellar development</td>
      <td> <a href=./notes/Q104450624.md> Q104450624 </a></td>
    </tr>
    <tr>
      <th>696</th>
      <td>The unipolar brush cell: a remarkable neuron finally receiving deserved attention</td>
      <td> <a href=./notes/Q30471584.md> Q30471584 </a></td>
    </tr>
    <tr>
      <th>697</th>
      <td>The Terminologia Histologica after 10 years: Inconsistencies, mistakes, and new proposals</td>
      <td> <a href=./notes/Q58049549.md> Q58049549 </a></td>
    </tr>
    <tr>
      <th>698</th>
      <td>Towards a Terminologia Neuroanatomica.</td>
      <td> <a href=./notes/Q36210969.md> Q36210969 </a></td>
    </tr>
    <tr>
      <th>699</th>
      <td>Cell segmentation-free inference of cell types from in situ transcriptomics data</td>
      <td> <a href=./notes/Q104450627.md> Q104450627 </a></td>
    </tr>
    <tr>
      <th>700</th>
      <td>Deep generative model embedding of single-cell RNA-Seq profiles on hyperspheres and hyperbolic spaces</td>
      <td> <a href=./notes/Q104450629.md> Q104450629 </a></td>
    </tr>
    <tr>
      <th>701</th>
      <td>Q106777072</td>
      <td> <a href=./notes/Q106777072.md> Q106777072 </a></td>
    </tr>
    <tr>
      <th>702</th>
      <td>interacCircos: an R package based on JavaScript libraries for the generation of interactive circos plots</td>
      <td> <a href=./notes/Q106777072.md> Q106777072 </a></td>
    </tr>
    <tr>
      <th>703</th>
      <td>Q106777073</td>
      <td> <a href=./notes/Q106777073.md> Q106777073 </a></td>
    </tr>
    <tr>
      <th>704</th>
      <td>Q10677707</td>
      <td> <a href=./notes/Q10677707.md> Q10677707 </a></td>
    </tr>
    <tr>
      <th>705</th>
      <td>Quickomics: exploring omics data in an intuitive, interactive and informative manner</td>
      <td> <a href=./notes/Q106777073.md> Q106777073 </a></td>
    </tr>
    <tr>
      <th>706</th>
      <td>PrInCE: an R/Bioconductor package for protein–protein interaction network inference from co-fractionation mass spectrometry data</td>
      <td> <a href=./notes/Q106777074.md> Q106777074 </a></td>
    </tr>
    <tr>
      <th>707</th>
      <td>IRIS-FGM: an integrative single-cell RNA-Seq interpretation system for functional gene module analysis</td>
      <td> <a href=./notes/Q106777075.md> Q106777075 </a></td>
    </tr>
    <tr>
      <th>708</th>
      <td>Gene Tracer: A smart, interactive, voice-controlled Alexa skill for gene information retrieval and browsing, mutation annotation, and network visualization</td>
      <td> <a href=./notes/Q106777076.md> Q106777076 </a></td>
    </tr>
    <tr>
      <th>709</th>
      <td>Gene Teller: An Extensible Alexa Skill for Gene-Relevant Databases</td>
      <td> <a href=./notes/Q98613961.md> Q98613961 </a></td>
    </tr>
    <tr>
      <th>710</th>
      <td>Gene set inference from single-cell sequencing data using a hybrid of matrix factorization and variational autoencoders</td>
      <td> <a href=./notes/Q104450631.md> Q104450631 </a></td>
    </tr>
    <tr>
      <th>711</th>
      <td>Single cell transcriptome atlas of immune cells in human small intestine and in celiac disease</td>
      <td> <a href=./notes/Q104450637.md> Q104450637 </a></td>
    </tr>
    <tr>
      <th>712</th>
      <td>Terminologia Histologica 10 years on: some disputable terms in need of discussion and recent developments</td>
      <td> <a href=./notes/Q92078001.md> Q92078001 </a></td>
    </tr>
    <tr>
      <th>713</th>
      <td>Terminologia Histologica: International Terms for Human Cytology and Histology</td>
      <td> <a href=./notes/Q106787216.md> Q106787216 </a></td>
    </tr>
    <tr>
      <th>714</th>
      <td>The Human Cell Atlas White Paper</td>
      <td> <a href=./notes/Q104450645.md> Q104450645 </a></td>
    </tr>
    <tr>
      <th>715</th>
      <td>The cellular immune response to COVID-19 deciphered by single cell multi-omics across three UK centres</td>
      <td> <a href=./notes/Q106272047.md> Q106272047 </a></td>
    </tr>
    <tr>
      <th>716</th>
      <td>Machine Learning to Support the Presentation of Complex Pathway Graphs</td>
      <td> <a href=./notes/Q93101366.md> Q93101366 </a></td>
    </tr>
    <tr>
      <th>717</th>
      <td>Ten years of CAZypedia: a living encyclopedia of carbohydrate-active enzymes.</td>
      <td> <a href=./notes/Q47639378.md> Q47639378 </a></td>
    </tr>
    <tr>
      <th>718</th>
      <td>The local and systemic response to SARS-CoV-2 infection in children and adults</td>
      <td> <a href=./notes/Q106272048.md> Q106272048 </a></td>
    </tr>
    <tr>
      <th>719</th>
      <td>Mapping the musculoskeletal system one cell at a time</td>
      <td> <a href=./notes/Q106272049.md> Q106272049 </a></td>
    </tr>
    <tr>
      <th>720</th>
      <td>Modeling the pasture-associated severe equine asthma bronchoalveolar lavage fluid proteome identifies molecular events mediating neutrophilic airway inflammation</td>
      <td> <a href=./notes/Q64072479.md> Q64072479 </a></td>
    </tr>
    <tr>
      <th>721</th>
      <td>Biocuration: Distilling data into knowledge.</td>
      <td> <a href=./notes/Q52586099.md> Q52586099 </a></td>
    </tr>
    <tr>
      <th>722</th>
      <td>Future of Evidence Synthesis in Precision Oncology: Between Systematic Reviews and Biocuration</td>
      <td> <a href=./notes/Q92594838.md> Q92594838 </a></td>
    </tr>
    <tr>
      <th>723</th>
      <td>Robust clustering and interpretation of scRNA-seq data using reference component analysis</td>
      <td> <a href=./notes/Q106272050.md> Q106272050 </a></td>
    </tr>
    <tr>
      <th>724</th>
      <td>Shotgun transcriptome, spatial omics, and isothermal profiling of SARS-CoV-2 infection reveals unique host responses, viral diversification, and drug interactions</td>
      <td> <a href=./notes/Q106272052.md> Q106272052 </a></td>
    </tr>
    <tr>
      <th>725</th>
      <td>Improved annotation of the insect vector of citrus greening disease: biocuration by a diverse genomics community</td>
      <td> <a href=./notes/Q64075749.md> Q64075749 </a></td>
    </tr>
    <tr>
      <th>726</th>
      <td>RCSB Protein Data Bank: biological macromolecular structures enabling research and education in fundamental biology, biomedicine, biotechnology and energy</td>
      <td> <a href=./notes/Q57793773.md> Q57793773 </a></td>
    </tr>
    <tr>
      <th>727</th>
      <td>Systemic Tissue and Cellular Disruption from SARS-CoV-2 Infection revealed in COVID-19 Autopsies and Spatial Omics Tissue Maps</td>
      <td> <a href=./notes/Q106272053.md> Q106272053 </a></td>
    </tr>
    <tr>
      <th>728</th>
      <td>Science not art: statistically sound methods for identifying subsets in multi-dimensional flow and mass cytometry data sets.</td>
      <td> <a href=./notes/Q47263573.md> Q47263573 </a></td>
    </tr>
    <tr>
      <th>729</th>
      <td>Karl Touton and his "xanthelasmatic giant cell." A selective review of multinucleated giant cells.</td>
      <td> <a href=./notes/Q34164972.md> Q34164972 </a></td>
    </tr>
    <tr>
      <th>730</th>
      <td>High-Precision Extraction of Emerging Concepts from Scientific Literature</td>
      <td> <a href=./notes/Q106831968.md> Q106831968 </a></td>
    </tr>
    <tr>
      <th>731</th>
      <td>RCSB Protein Data Bank tools for 3D structure-guided cancer research: human papillomavirus (HPV) case study</td>
      <td> <a href=./notes/Q99557412.md> Q99557412 </a></td>
    </tr>
    <tr>
      <th>732</th>
      <td>Systematic tissue annotations of –omics samples by modeling unstructured metadata</td>
      <td> <a href=./notes/Q106833747.md> Q106833747 </a></td>
    </tr>
    <tr>
      <th>733</th>
      <td>Large field of view-spatially resolved transcriptomics at nanoscale resolution</td>
      <td> <a href=./notes/Q106648171.md> Q106648171 </a></td>
    </tr>
    <tr>
      <th>734</th>
      <td>Putative cell type discovery from single-cell gene expression data</td>
      <td> <a href=./notes/Q95328107.md> Q95328107 </a></td>
    </tr>
    <tr>
      <th>735</th>
      <td>Worldwide Protein Data Bank biocuration supporting open access to high-quality 3D structural biology data.</td>
      <td> <a href=./notes/Q50290407.md> Q50290407 </a></td>
    </tr>
    <tr>
      <th>736</th>
      <td>Applying Citizen Science to Gene, Drug, Disease Relationship Extraction from Biomedical Abstracts</td>
      <td> <a href=./notes/Q63640419.md> Q63640419 </a></td>
    </tr>
    <tr>
      <th>737</th>
      <td>Negative Knowledge for Open-world Wikidata</td>
      <td> <a href=./notes/Q106630248.md> Q106630248 </a></td>
    </tr>
    <tr>
      <th>738</th>
      <td>Wikidata and the Bibliography of Life</td>
      <td> <a href=./notes/Q106727511.md> Q106727511 </a></td>
    </tr>
    <tr>
      <th>739</th>
      <td>The Organoid Cell Atlas</td>
      <td> <a href=./notes/Q106686809.md> Q106686809 </a></td>
    </tr>
    <tr>
      <th>740</th>
      <td>The antral clear cell--a new cell type discovered in the pyloric-antral mucosa of the ferret</td>
      <td> <a href=./notes/Q71273935.md> Q71273935 </a></td>
    </tr>
    <tr>
      <th>741</th>
      <td>A resource to explore the discovery of rare diseases and their causative genes</td>
      <td> <a href=./notes/Q106708178.md> Q106708178 </a></td>
    </tr>
    <tr>
      <th>742</th>
      <td>Community curation in PomBase: enabling fission yeast experts to provide detailed, standardized, sharable annotation from research publications</td>
      <td> <a href=./notes/Q94470608.md> Q94470608 </a></td>
    </tr>
    <tr>
      <th>743</th>
      <td>Single-cell RNA-seq analysis reveals the progression of human osteoarthritis</td>
      <td> <a href=./notes/Q90348985.md> Q90348985 </a></td>
    </tr>
    <tr>
      <th>744</th>
      <td>Cell-type-specific gene expression patterns in the knee cartilage in an osteoarthritis rat model.</td>
      <td> <a href=./notes/Q49169602.md> Q49169602 </a></td>
    </tr>
    <tr>
      <th>745</th>
      <td>Defining life: synthesis and conclusions.</td>
      <td> <a href=./notes/Q47407732.md> Q47407732 </a></td>
    </tr>
  </tbody>
</table>